



%\chapter{The Ribosomal City}
%\chapter{The Ribosome, a puzzling protein-rna complex}
\chapter{Ribosomal dynamics, a puzzling affair}
\label{chapter:ribosomes}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Picture of the ribosome
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering    
    %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
    \includegraphics[width=0.8\textwidth]{ribosomes_allproteinscolored.png}
    \caption{ 
        \textbf{Picture of the ribosome.}
        The ribosome consist of a small and a large subunit. These two come together on messenger RNA templates to perform protein synthesis.
        The complete ribosome is made of 3 ribosomal RNA molecules and more than 50 proteins, shown in grey and color in the picture above, respectively \cite{Chen2013}.
        The 16S rRNA and 23S rRNA molecules function as backbone. % for the small and large subunits, respectively.
        Proteins labeled S1 up to S22 
        % (S22 being sub-stoichiometric) 
        bind to the 16S rRNA to form the small subunit, and proteins labeled L1-L36 and the 5S rRNA bind to the 23S rRNA to form the large subunit \cite{Keseler2017}.
        The Ecocyc database lists 58 ribsomal proteins, though also different numbers of ribosomal proteins are listed (ref. \cite{Chen2013} e.g. talks about 54 ribosomal proteins).
        % The 5S rRNA also binds to the large subunit.
        Interestingly, the ribosomal RNA constitutes 73-80\% of the total RNA found in an \textit{E. coli} cell (mRNA constitutes 3-4.5\% and tRNA 15-20\%) \cite{Norris1972}.
        Assembly of the ribosome also requires many co-factors \cite{Chen2013}. % note also DnaK is required.
        %
        This image is based on x-ray crystallography (PDB ID: 4v4q, \cite{Schuwirth2005}). Pdb files downloaded from \texttt{www.rcsb.org} \cite{Berman2000} 
        visualized with UCSF Chimera (version 1.11.2, build 41376, \cite{pettersen2004}).
        % Previously, I seemed to have used 4v4a, which is from 2003 http://www.rcsb.org/pdb/explore.do?structureId=4v4a.        
        %
        %        
    }
    \label{fig:ribo:pictureofribo}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Introduction}

\subsection*{The central role of the ribosome}

Ribosomes are life.
%
In 1958, Francis Crick, who discovered the structure of DNA with James Watson and Rosalind Franklin,
formulated what is called the central dogma of molecular biology.
This states 
% declares 
that DNA holds the amino acid sequence information required to build a protein, and that this sequence information is first translated to RNA, which is then transcribed into a protein \cite{Crick1958}.
% 
This dogma relates immediately to some of the features that are said to define life: 
the capability to store information and 
the regeneration of components from scratch \cite{Lawrence2005, Koshland2002}.
%
Also around that time, 
Slightly earlier, in 1955, "new cytoplasmic components" were already discovered under an electron microscope \cite{Palade1955}.
These components turned out to be the components in the cell that perform the last step of the central dogma: 
the synthesis of proteins using mRNA molecules as templates; also known as transcription.
The components became known as ribosomes.
%
% Given the central place of the ribosome, in the circle of life, and 
Given the idea that early life consisted only of RNA molecules that could catalyze other reactions (the "RNA world" \cite{Campbell2002}), it is not surprising that the ribosomes 
% these components of the cell
consist of both catalytic RNA and proteins,
as shown in figure \ref{fig:ribo:pictureofribo}.
%
%As such, it is not only an embodiment of the central dogma, but also the exception to the rule.
As such, we can say ribosomes are a key element of life.

% life
% Lawrence: information storage, catalysts, energy relation environment, grow and reproduce, respond stimuli
% Koshland: program, improvisation (MW: long term adaptation), compartmentalization, energy, regeneration, adaptabilty (short term adaptation), seclusion (being able to separate different processes, e.g. enzyme's specifity).

\subsection*{Understanding the role of ribosomes in protein fluctuations}

In this thesis, 
we have been trying to further 
% When we want to understand the cell, ribosomes are a good starting point.
%It is our aim to better 
understand % cellular dynamics.
temporal fluctuations in the concentrations of cellular components.%, which occur due to the intrinsic stochastic nature of chemical processes that underlay cellular 
%
As mentioned in earlier chapters (\ref{chapter:literaturereview} and  \ref{chapter:CRP}),
these fluctuations are a result of the intrinsic stochastic nature of chemical processes that occur in the cell.
% (see also previous chapters \ref{chapter:literaturereview} and  \ref{chapter:CRP}).
%
Given the central role of the ribosome in the cell that was just discussed, ribosomes might play a big role in such fluctuations.
%Since ribosomes are key components in the cell, 
Indeed, it is often suggested that fluctuations in ribosomal concentration could result in cell-wide protein concentration fluctuations \cite{Davidson2008, Raj2008, Chalancon2012, Bruggeman2018}.
%
Such concerted concentration fluctuations are also referred to as extrinsic noise (as opposed to intrinsic noise, fluctuations that only occur in one cellular species) \cite{Elowitz2002}.

\subsection*{Quantifying ribosomal dynamics}

In a previous thesis from the Tans lab, Noreen Walker 
% In previous work from our lab, Walker et al. 
quantified the dynamic relationship between single cell ribosomal expression fluctuations and growth in steady state conditions \cite{Walker2016t}.
%
The experiments involved presented many challenges, 
of which a number is described in her thesis.
%
% The results from these experiments were rather inconclusive.
%
Given previously described central role of the ribosome, 
it was hypothesized that the stochastically fluctuating concentration of ribosomes might be limiting, 
meaning that ribosomal fluctuations might result in growth rate fluctuations.
%
In single cell time lapse experiments, no clear indications were found to support this hypothesis.
%
Instead, 
ribosomal proteins L19 and L31 that were labeled with mCherry (L31-R and L19-R)
% mCherry-labeled L31 and L19 subunits (L31-R and L19-R) 
showed expression-growth cross-correlations (CCs) indicative of dilution-scenario behaviour in minimal medium, or none at all in rich medium (see chapters \ref{chapter:literaturereview} and \ref{chapter:CRP} for discussions about the use of cross-correlations to interpret dynamics). 
mCerulean labeled L31 ribosomal protein (L31-C) showed expression-growth CCs with small correlations around zero delay 
% at positive delays 
%between expression and growth 
in minimal medium.
This difference between the L31-C and L31-R experiments was unexpected, as only the label was different.
%
In rich medium, the L31-C cross-correlation was consistent with the L31-R cross-correlation, and showed almost no correlation.
%
Since the ribosome mainly consists of RNA, 
a GFP reporter under the control of one of the ribosomal RNA promoters was also used (rrna-G).
Like the L31-C reporter, the dynamics of this rrna-G reporter showed positive correlations in minimal medium and only very small correlations in rich medium.
% an additional reporter was used which consisted of the ribosomal RNA promoter 
%
Also growth-expression scatter plots were used, to investigate possible interesting shapes of growth-expression relationships, but this yielded no noteworthy shapes.

In an attempt to force the cell in a scenario where ribosomes are limiting, experiments were conducted where cells were grown in the presence of sub-inhibitory concentrations of tetracycline, an antibiotic.
%
Though also interesting in itself, the outcome of such an experiment could serve as a reference to interpret other experiments.
%
In rich medium, both for the L31-R and the rrn-G reporter, the antibiotic did not result in more limiting behaviour, i.e. correlations did not become more positive.
%
Given the disparate observations, both between reporters and conditions, 
and the absence of a point of reference,
the nature of the ribosomal dynamics remained fairly elusive; it was concluded that ribosomal fluctuations perhaps do not have a pivotal role in steady state cellular growth dynamics.

\subsection*{New work}

In this chapter, we will describe a few additional experiments that were aimed at gaining further insights in the ribosomal dynamics.
Specifically, we tried to understand whether fluctuations in ribosomal concentration could have cell-wide implications.
%
To answer this question, we tried both new experiments as well increase the throughput of existing experiments.
%
Since this is an ongoing project, this chapter will be more succinct than previous ones.

\section*{Results}
% > looked at dual reporters with pn25 reporter to assess effect from ribosomal concentration > protein production
    % ^> (outlook) Note that also R_CR-pq would be interesting in future with minor technical adjustment
% > looked at shift experiment to look at a (dynamic) environment where we knew cells with more ribosomes have an advantage
% > Additional semi-steady state measurements w/ and w/o antibiotics (incl. the pn25 reporter)
% > (outlook) we also looked into adding a plasmid that would introduce a burden unto the ribosomes 
%	% ^> Note that cells will adjust, so in a sense this might not increase the effect of fluctuations (perhaps put in sketch of two graphs with two optimums)
% > (outlook) we also looked into rrna reporter in delta(rna) strains
% > (outlook) maybe r-proteins are not as stochastically equivalent as sometimes assumed, ratio Y_m9:Y_ab and R_m9:R_ab not equal (2.4 and 2.3)
    % ^> what about production though, that's ratio_Y 3.18 vs. 2.15, should these ratios not be equal? --> not to C-ratio as the growth rates differ.
% > (outlook) ppgpp titration
% > (outlook) perhpas the fluctuations in total ribosome activity are a difficult function of fluctuations in all of the separate ribosomal components, which all might have their own (partially) independent contributions, and therefor it is hard to see single correlations. (the C

\subsection*{Additional strains}

\begin{table}[h]
    \begin{tabularx}{\textwidth}{llXl}
        %
        \textbf{ASC number}	& \textbf{Shorthand} & \textbf{Description} & \textbf{Source}		\\
        \hline
        ASC976  &	Prrna-C, pn25-Y	&	Δphp::pn25-mVenus-cmR, Δche::Prrsa-mCerulean-kanR. (Kanamycin and chloramphenicol resistance.)  & VS \\
        %     
        ASC968	& L19-C, pn25-Y	& L19-gc-mCerulean-kanR (GC linker), Δ(…)::pn25-mVenus-cmR.	(Kanamycin and chloramphenicol resistance.)	& VS \\
        %
        ASC1058	& L9-R, S2-Y	& Also known as JE202. rplI-mCherry-KanR (L9), rpsB-venus-CmR (S2). (Kanamycin and chloramphenicol resistance.) Gift from Johan Elf lab. & \cite{Wallden2016} \\
        \hline
        %
        % ASC631 & pn25-R, lacA-G & $\Delta$lacA::gfp-cat, $\Delta$php::pn25-mCherry-kanR & \cite{Kiviet2014} \\
        ASC666 & L31-R, gltA-G  & L31::mCherry-kanR, gltA::gfpA206K-cat & \cite{Kiviet2014} \\
        ASC810	& L31-C & L31-gc-mCerulean-kanR (GC linker). (Kanamycin resistant.) & NW, VS \\
        \hline
    \end{tabularx}
    \caption{\textbf{Strains used in this work.} Strain ASC1058 was a kind gift from the Johan Elf lab. VS indicates these strains were created by Vanda Sunderlikova, technician in the Tans lab. ASC stands for AMOLF strain collection. Note that Y, C, G and R indicate yellow, cyan, green and red fluorescent reporters, respectively.}
    \label{table:ribostrains1}
\end{table}

To be able to further explore the interaction of ribosomal expression with cellular dynamics, 
we produced additional strains.
%
Importantly, we were interested in the effect fluctuations in ribosomal expression would have on single cell protein production.
%
We therefore created two dual-label strains that both carried ribosomal reporter constructs, as well as constitutively expressed fluorescent reporters.
%
In the first strain, we fused the mCerulean fluorescent reporter sequence to the ribosomal RNA promoter (rrsa), and additionally inserted an mVenus sequence to the constitutive pn25 promoter. Both were chromosomally inserted.
%
The pn25-mVenus reporter serves as a readout for fluctuations in protein production that likely affect all proteins that are produced in the cell. 
% a proxy for protein production in general.
%
In the second strain, we introduced an mCerulean sequence behind the chromosomal L19 ribosomal protein sequence, thus creating a strain that produces L19 ribosomal proteins that are fused to mCerulean fluorescent labels.
We also introduced the constitutive pn25-mVenus reporter to this strain.
%
These strains, the Prrna-C, pn25-Y strain and the L19-C, pn25-Y strain are listed in table \ref{table:ribostrains1}.

Furthermore, because the ribosome consists of so many ribosomal proteins, we wanted to be able to confirm 
potentially observed dynamics for one labeled ribosomal protein, also for other labeled ribosomal proteins.
%
% potential observations in a strain that had other ribosomal proteins labeled.
For this purpose, we requested a strain from the Elf lab which had both the L9 and S2 ribosomal proteins labeled by the mCherry and Venus fluorescent proteins, respectively \cite{Wallden2016}.
This strain was kindly supplied by the Elf lab, and is also listed in table \ref{table:ribostrains1} with the shorthand notation L9-R, S2-Y.

%\red{
%pn25 both L and rna, refer to table;
%elf strain}

%\subsection*{An attempt to create a limiting situation}
\subsection*{Antibiotic shift experiments to create limiting situations}

%\subsubsection{The shift experiment}

To understand the dynamics of a process, it is often convenient to grasp what happens in extreme cases.
%
We therefore attempted to device an experiment which would lead to a situation where single cells that expressed more ribosomes than the population average would have a growth advantage, thus forcing a "limiting" situation.
% 
We did this by growing cells in minimal medium in our microfluidic device 1 for a few hours, and then switching to minimal medium supplemented with antibiotics (see chapter \ref{chapter:methods} and \ref{chapter:filarecovery} for a description of the microfluidics device).
%We did this by growing cells in minimal medium in our microfluidic device 1 (see chapters \ref{chapter:methods} and \ref{chapter:filarecovery} for a description of the microfluidics device).
%
We hypothesized that right after the medium has switched, cells in the population --- which at that point are calibrated for growth in minimal medium --- that happen to express more ribosomes have a growth advantage.
%
To analyse whether this was indeed the case, we made growth-concentration scatter plots for multiple points in time during the experiment.
%
We further analysed these measurements by calculating the slope and correlation of these scatter plots for each point in time.

Some data involving an antibiotic switch was already gathered
%One experiment like this was already conducted 
by Tans lab alumni Sarah Boulinea for the L31-R, gltA-G strain (see table \ref{table:ribostrains1}); 
we also ran our analysis on this dataset.
Also, two additional experiments were conducted, one with the L31-C strain where one microcolony was analyzed, 
and one with the rrsa-C, pn25-Y strain, where three microcolonies were analyzed.
%
Before looking at the scatter plots, we first look at the 
trends in fluor concentration for all these five datasets, which are shown in figure \ref{fig:ribo:scatter1}.
This figure shows that for the Prrna-C strain datasets, the signal goes up after the switch to medium with antibiotics, as is expected.
However, for both for the L31-R and L31-C datasets, the signal appears to be going down after the shift (though pre-shift fluctuations on the population average level of the L31-R data make it unclear what is exactly going on).
It is unclear why this happens, since ribosome inhibition should increase relative ribosomal demand \cite{You2013}.
%
We now turn to the scatter plots.
%As mentioned, we created growth-expression scatter plots for each time point within the dataset, for all of these datasets, and f
For brevity we only show the representative series of scatter plots that relates to the L31-C dataset, see figure \ref{fig:ribo:scatter1}. 
This figure shows that there is no clear change in correlation visible by eye after the switch to medium with antibiotics.
%
As mentioned however, we further quantified the data by calculating both the slope (least square fit by Matlab's \texttt{polyfit} function) and correlation coefficient for each of the scatter plots.
%
This is shown in figures \ref{fig:ribo:switch1}-\ref{fig:ribo:switch5}.
%
The L31-R dataset (figure \ref{fig:ribo:switch1}) arguably shows a small increase in correlation, but the correlation also decreases swiftly after that.
The L31-Y dataset (figure \ref{fig:ribo:switch2}) is difficult to interpret, 
likely the positive correlations before the switch are caused by chance as there are only a few cells in the microcolony at those points in time.
%
The first Prrsa-Y dataset (figure \ref{fig:ribo:switch3}) shows a change in signal right after the switch, but this is not seen in the two other Prrs-Y datasets (\ref{fig:ribo:switch4}-\ref{fig:ribo:switch5}).
%
%These figures show that there is no clear change in correlation observed after the switch.
In conclusion, it is difficult to interpret these datasets, as the different datasets show different trends.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Switch experiment figures
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%% General trend of signals
\begin{figure}
    \centering    
    \includegraphics[width=0.3\textwidth]{pdf_2012-11-15_pos5_L31-mCherry_fluorTrend.pdf}
    \includegraphics[width=0.3\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_fluorTrend.pdf} \\
    \includegraphics[width=0.3\textwidth]{pdf_2016-09-20_pos1_prrsa-mCerulean_pn25-yfp_fluorTrend} % added
    \includegraphics[width=0.3\textwidth]{pdf_2016-09-20_pos2_prrsa-mCerulean_pn25-yfp_fluorTrend}
    \includegraphics[width=0.3\textwidth]{pdf_2016-09-20_pos3_prrsa-mCerulean_pn25-yfp_fluorTrend}            
    \caption{ 
        \textbf{Fluorescence intensity for switch from clean medium to medium supplemental with antibiotics.}
        Red dots show single cell obsvervations, the black line indicates the colony average.
        (Top left) L31-R strain. 
        (Top right) L31-C strain.
        (Bottom three) All are rrsa-C strains.      
    }
    \label{fig:ribo:fluorsignals}
\end{figure}


%%%%%%%%%%%%%%%%%%%% Scatter plots (only one shown for brevity).
\begin{figure}
    \centering    
    \includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
    %\includegraphics[width=0.8\textwidth]{pdf_2012-11-15_pos5_L31-mCherry_summaryPlot.pdf}
    \caption{ 
        \textbf{Scatter plots for switch from clean medium to medium supplemental with antibiotics.}
        %
        %        
    }
    \label{fig:ribo:scatter1}
\end{figure}

%%%%%%%%%%%%%%%%%%%% Summary plots for shift experiments
\begin{figure}
    \centering    
    %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
    \includegraphics[width=0.8\textwidth]{pdf_2012-11-15_pos5_L31-mCherry_summaryPlot.pdf}
    \caption{ 
        \textbf{Scatter plots for switch from clean medium to medium supplemental with antibiotics.}
        %
        %        
    }
    \label{fig:ribo:switch1}
\end{figure}
\begin{figure}
    \centering    
    %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
    \includegraphics[width=0.8\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_summaryPlot.pdf}
    \caption{ 
        \textbf{Scatter plots for switch from clean medium to medium supplemental with antibiotics.}
        %
        %        
    }
    \label{fig:ribo:switch2}
\end{figure}
\begin{figure}
    \centering    
    %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
    \includegraphics[width=0.8\textwidth]{pdf_2016-09-20_pos1_prrsa-mCerulean_pn25-yfp_summaryPlot.pdf}
    \caption{ 
        \textbf{Scatter plots for switch from clean medium to medium supplemental with antibiotics.}
        %
        %        
    }
    \label{fig:ribo:switch3}
\end{figure}
\begin{figure}
    \centering    
    %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
    \includegraphics[width=0.8\textwidth]{pdf_2016-09-20_pos2_prrsa-mCerulean_pn25-yfp_summaryPlot.pdf}
    \caption{ 
        \textbf{Scatter plots for switch from clean medium to medium supplemental with antibiotics.}
        %
        %        
    }
    \label{fig:ribo:switch4}
\end{figure}
\begin{figure}
    \centering    
    %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
    \includegraphics[width=0.8\textwidth]{pdf_2016-09-20_pos3_prrsa-mCerulean_pn25-yfp_summaryPlot.pdf}
    \caption{ 
        \textbf{Scatter plots for switch from clean medium to medium supplemental with antibiotics.}
        %
        %        
    }
    \label{fig:ribo:switch5}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection*{Higher throughput and additional cross-correlations}

\subsubsection*{Many experiments in one}

One reason that it is hard to analyse the data from the antibiotic switch experiments using microfluidic device 1, 
is
that the size of the microcolony starts out small, which leads to a 
high variability of the observables during the first part of the experiment.
%
To produce a dataset with a constant large amount of cells, we performed additional antibiotic switch experiments in microfluidic device 2.
%
Conveniently, microfluidic device 2 allows for multi-day experiments, which allowed us 
%to increase the throughput of the experiments, and allowed us 
to also measure additional steady state expression-growth cross-correlation curves. % by prolonged multi-hour experiments.
%
Importantly, we performed these additional experiments with the new Prrna-C, pn25-Y and L19-C, pn25-Y strains, to investigate the effect of ribosomal fluctuations on protein production rates with the pn25-mVenus reporter.
%
%For example, table \ref{table:ribosomes:typicaldevice2experiment} shows the setup of a typical experiment.
An example of a typical 
%For example, the 
experimental design is given by the measurements we performed on the L19-R pn25-Y strain, which involved the following sequence of supplied medium: \textit{6 hours TY, 6 hours TY + TET, 2 hours TY, 2 hours TY + TET, 2 hours TY, 2 hours TY + TET, 2 hours TY, 6 hours M9, 6 hours M9 + TET, 2 hours M9, 2 hours M9 + TET, 2 hours M9, 2 hours M9 + TET}. (M9 indicates M9 minimal medium here, supplemented with lactose, uracil and tween20; see chapter \ref{chapter:filarecovery} for further description of used media). 
%
%By analyzing the last two hours of the 6hr sequences, we could investigate the steady state behavior of the population.
%The many switches would allow us to investigate the behavior in the case of switches.


\subsubsection*{Results of the experiments}

The large amount of data generated by microfluidic device 2 is both an advantage and a disadvantage.
%
We will not go into technical details here, but both the amount of data and also the nature of the growth in the wells present the computer analysis with new challenges.
%
We therefore only analysed parts of the data that was produced by these experiments at this point.
%
We chose to first analyse the data where we assumed cells had reached steady state; and where we can perform cross-correlation analyses.
Additional data recorded at the times of the switches exists but has not been fully analysed yet.
%This selection of the data did not include the parts were switches were performed, 
%but only the data were we assumed cells had reached steady state growth.
%We therefore here only present cross-correlation analysis.
%
We also note that 
additional manual corrections 
% (on top of those already performed) 
could perhaps further refine the analyses that we show here. 
%
For example, some of the tracked lineages show unrealistic fluctuations which might be due to artefacts in the computer analysis and might be removed or corrected (see supplemental figures \ref{fig:ribo:branchesXXXX}), and also the analysis now only takes into account a selection of the total amount of imaged cells, this selection could be extended to obtain more data.

\subsubsection*{\red{Expression-growth CCs of constitutive reporter are consistent with dilution mode}}

At any rate, figures \ref{fig:ribo:CCsEmuYpn25}-\ref{fig:ribo:CCsPmuYpn25Ribo} show data obtained from the Prrna-C, pn25-Y and L19-C, pn25-Y strains.
%
Most of these CCs show large error bars, and often fall inside the range of the negative control.
This indicates that more data is required to draw definitive conclusions.
%
Nevertheless, figures \ref{fig:ribo:CCsEmuYpn25} and \ref{fig:ribo:CCsPmuYpn25} show the CCs calculated for the pn25 constitutive expression-growth relationship for growth in minimal medium, TY medium, and TY medium supplemented with sub-inhibitory concentration of the antibiotic tetracycline (0.5 $\upmu$M).
%
Most of the correlation-growth curves show negative correlation values for negative delays.
The production rate-growth curves do not show clear trends.
We saw earlier (see chapter \ref{chapter:CRP}) that constitutive reporters often show dilution mode dynamics.
The CCs we observe here could be consistent with that transmission mode.
%
Furthermore, figures \ref{fig:ribo:CCsEmuYRibo} and \ref{fig:ribo:CCsPmuYRibo} show data from the same strains, but show the CCs for the ribosomal reporters (rrna and L19).
The CC in panel A does not show a clear trend.
The CCs in panels B-D however, which show the behaviour of the rrna reporter, also show behaviour that also might be consistent with dilution mode transmission of noise; negative concentration-growth at negative delays correlations are seen both for M9 and TY medium, and also for the condition where antibiotic was added to the medium.
%
Note here that the rrna reporter might not capture all ribosomal RNA fluctuations, since it is by its nature not a translational fusion reporter such as the other ribosomal reporters.
%



\section{First some material}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Dataset w/ pn25 reporters
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Constitutive reporters 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_Y6_mean_cycCor_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between concentration of constitutive reporter and growth.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsEmuYpn25}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between production rate of constitutive reporter and growth.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsPmuYpn25}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Ribosomal reporters
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_C6_mean_cycCor_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between concentration of ribosomal reporter and growth.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsEmuYRibo}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_dC5_divAreaPx_cycCor_muP9_fitNew_atdC5_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between production rate of ribosomal reporter and growth.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsPmuYRibo}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Correlations between ribosomal expression and pn25 expression
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_C6_mean_cycCor_Y6_mean_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between concentration of ribosomal reporter and concentration of pn25 reporter.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsEmuYpn25Ribo}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_dC5_divAreaPx_cycCor_dY5_divAreaPx_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between production rate of ribosomal reporter and production rate of pn25 reporter.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsPmuYpn25Ribo}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Dataset with Elf dual reporter
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% S2-Y ribosomal reporter
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_Y6_mean_cycCor_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between the concentration of the S2 ribosomal protein and growth.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
        %
%        (A) A: giu_asc1058_M9_steady, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
%        (B) B: giu_asc1058_M9_steady_antibiotics, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
    }
    \label{fig:ribo:CCsEmuYS2}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between the production rate of the S2 ribosomal protein and growth.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
%
%        (A) A: giu_asc1058_M9_steady, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
%        (B) B: giu_asc1058_M9_steady_antibiotics, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
        %        
    }
    \label{fig:ribo:CCsPmuYS2}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% L9-R ribosomal reporter
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_R6_mean_cycCor_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between the concentration of the L9 ribosomal protein and growth.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
%
%        (A) A: giu_asc1058_M9_steady, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
%        (B) B: giu_asc1058_M9_steady_antibiotics, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
        %        
    }
    \label{fig:ribo:CCsEmuYL9}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between the production rate of the L9 ribosomal protein and growth.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
%
%        (A) A: giu_asc1058_M9_steady, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
%        (B) B: giu_asc1058_M9_steady_antibiotics, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
        %        
    }
    \label{fig:ribo:CCsPmuYL9}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Correlations between ribosomal expression r-protein1 and r-protein2
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_Y6_mean_cycCor_R6_mean_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between concentrations of two ribosomal proteins.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics.
        %        
    }
    \label{fig:ribo:CCsEmuYRiboribo}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_dY5_divAreaPx_cycCor_dR5_divAreaPx_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between production rates of two ribosomal proteins.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics.
        %        
    }
    \label{fig:ribo:CCsPmuYRiboribo}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Overview figures of branches and means for all datasets
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% Means for ASC968andASC976 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}
    \centering
        \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_overview_means.pdf}
    \caption{ 
        \textbf{Population mean values of different parameters measured in different strains and conditions.}
%        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
%        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
%        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
%        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
%        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
%        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %       A: giu_asc968_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        B: giu_asc1058_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        C: giu_asc1058_M9_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        D: giu_asc976_TY_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        E: giu_asc976_TY_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        F: giu_asc976_M9_steadymw_asc976_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        
    }
    \label{fig:ribo:CCsPmuY}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_overview_means.pdf}
    \caption{ 
        \textbf{Population mean values of different parameters measured in different strains and conditions.}
%        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
%        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
%        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
%        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
%        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
%        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %       A: giu_asc968_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        B: giu_asc1058_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        C: giu_asc1058_M9_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        D: giu_asc976_TY_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        E: giu_asc976_TY_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        F: giu_asc976_M9_steadymw_asc976_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        
    }
    \label{fig:ribo:CCsPmuY}
\end{figure}



    






			

\begin{table}                       
\begin{tabular}{ l l l l }
    \centering
    r-protein & Gene name & Operon & Fluorescent label \\ 
    \hline
    L31 & rpmE & none & Cerulean, mCherry \\ 
    L19 & rplS & rpsP-rimM-trmD-rplS & Cerulean, mCherry \\ 
    L9 & rplI & rpsF-priB-rpsR-rplI & mCherry \\ 
    S2 & rpsB & ttf-rpsB-tsf & Venus\\ 

\end{tabular}
    \caption{Ribosomal proteins that we have labeled.} \label{tab:ribolabeledprots}
\end{table}



\subsection{rpmE (L31)}
\begin{description}
    \item[General description.] lore ipsum
    \item[Operon.] lore ipsum
\end{description}



\subsection{rplS (L19)}


\subsection{rplI (L9)}


\subsection{rpsB (S2)}










\subsection*{Discussion and conclusion}

(a) 
(b) would be nice to re-run some of noreen's analyses with the script that creates the controls.

(z) mention the order in which ribosomes are formed \cite{Chen2013}


\section*{Acknowledgements}

%I am indebted to Noreen Walker and Giulia Bergamaschi regarding this chapter.
%
I am thankful to Noreen Walker, who fostered the ribosome project for 4 years, for her collaboration and input regarding these additional experiments.
%
Much of the data shown in this chapter were taken by Giulia Bergamaschi, master student in the Tans lab at that time. I am thankful for 
the mountains of work she performed in only three months time.
%her hard work. 

Molecular graphics and analyses 
% addition MW
of crystal structures
%
were performed with the UCSF Chimera package (production version 1.11.2, build 41376). Chimera is developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco (supported by NIGMS P41-GM103311). 




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Supplementary stuff
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\FloatBarrier
\clearpage

\section*{Supplemental figures and tables}

\begin{table}[h]
    \begin{tabularx}{\textwidth}{llXl}
        %
        \textbf{ASC number}	& \textbf{Shorthand} & \textbf{Description} & \textbf{Source}		\\
        \hline
        %      									
        asc656	& L31-R & L31-mCherry-kanR (no linker).	(Kanamycin resistant.) & NW, VS \\
        asc657	& L19-R & L19-mCherry-kanR (no linker).	(Kanamycin resistant.) & NW, VS \\
        asc680	& L31-R, Prrn-G & L31-mCherry-kanR (no linker), Δ(cheZ)::Prrn-GFP-catR. (Kanamycin and chloramphenicol resistant.)	& NW, VS \\
        asc779	& Prrn-G& Δ(cheZ)::Prrn-GFP, rrsa promoter. (No resistance.) &	NW, VS \\
        
        \hline										
        & L19-C & 	L19-mCerulean L19-gc-mCerulean-kanR (GC linker)	(Kanamycin resistance.)	& VS \\
        & Prrn-C & 	Δ(cheZ)::Prrn-mCerulean-kanR	(Kanamycin resistance)	& VS \\
        \hline
        asc1088 & mCherry+
        % L19-C, pn25-Y, p-pn25-R 
        &	asc968 + high copy plasmid w. pn25-mCherry (Resistances: kanamycin, chloramphenicol, ampicilin.) & VS \\
        \hline
    \end{tabularx}
    \caption{\textbf{Strains used in previous work and miscellaneous strains.} NW indicates these strains were used in Noreen Walker's thesis \cite{Walker2016t}. VS indicates these strains are created by Vanda Sunderlikova.}
\end{table}



% Branches for ASC968andASC976 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_branches_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Growth of single cells of the different strain populations in the different conditions.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %       A: giu_asc968_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        B: giu_asc1058_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        C: giu_asc1058_M9_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        D: giu_asc976_TY_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        E: giu_asc976_TY_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        F: giu_asc976_M9_steadymw_asc976_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        
    }
    \label{fig:ribo:branchesASC968andASC976}
\end{figure}

% Branches for ASC1058 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_branches_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Growth of single cells of the different strain populations in the different conditions.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %       A: giu_asc968_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        B: giu_asc1058_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        C: giu_asc1058_M9_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        D: giu_asc976_TY_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        E: giu_asc976_TY_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        F: giu_asc976_M9_steadymw_asc976_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        
    }
    \label{fig:ribo:branchesASC1058}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Notes to self
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\clearpage

{\color{red}
    \section{Misc. todo's etc}
    
    References that might be convenient:
    \begin{itemize}
        \item Bosdriesz E, Molenaar, D., Teusink, B., and Bruggeman, F.J. CHAPTER 6 How fast-growing bacteria robustly tune their ribosome concentration to approximate growth rate maximisation. Manuscr. Accept. Publ. FEBS J., 119–153.
        \cite{BosdrieszE}
        \item Maeda, M., Shimada, T., and Ishihama, A. (2015). Strength and Regulation of Seven rRNA Promoters in Escherichia coli. PLoS One 10, 1–19. \cite{Maeda2015}
        \item Gyorfy, Z., Draskovits, G., Vernyik, V., Blattner, F.F., Gaal, T., and Posfai, G. (2015). Engineered ribosomal RNA operon copy-number variants of E. coli reveal the evolutionary trade-offs shaping rRNA operon number. Nucleic Acids Res. 43, 1783–1794. \cite{Gyorfy2015}
        \item \textit{(The following two articles talk about the ribosomal RNA deletion strains, also surprising interactions with antibiotics. Of main importance is the observation that mainly tRNA seems to be limiting. See also unpublished \cite{Quan2013}.)} Bollenbach, T., Quan, S., Chait, R., and Kishony, R. (2009). Nonoptimal Microbial Response to Antibiotics Underlies Suppressive Drug Interactions. Cell 139, 707–718. \cite{Bollenbach2009} AND 1. Quan, S., Skovgaard, O., McLaughlin, R.E., Buurman, E.T., and Squires, C.L. (2015). Markerless Escherichia coli rrn Deletion Strains for Genetic Determination of Ribosomal Binding Sites. G3 Genes|Genomes|Genetics 5, 2555–2557. \cite{Quan2015} \textit{Quan et al. also refer to Orelle et al. 2013; Polikanov et al. 2014b; Orelle et al. 2015.}
        \item \textit{(This article details the rRNA regulation.)} Schneider, D.A., Ross, W., and Gourse, R.L. (2003). Control of rRNA expression in Escherichia coli. Curr. Opin. Microbiol. 6, 151–156. \cite{Schneider2003} 
        \item \textit{(These two article with description of the structures generated from rRNA genes, also mention the cleavage of the RNA genes.)}  Kaczanowska, M., and Ryden-Aulin, M. (2007). Ribosome Biogenesis and the Translation Process in Escherichia coli. Microbiol. Mol. Biol. Rev. 71, 477–494. \cite{Kaczanowska2007} AND Shajani, Z., Sykes, M.T., and Williamson, J.R. (2011). Assembly of bacterial ribosomes. Annu. Rev. Biochem. 80, 501–26. \cite{Shajani2011}
        NOTE that \cite{Chen2013} has also a picture of biogenesis and assembly.
        \item \textit{(Following article measures burden on ribosome and effect on growth in a dynamic environment.)} Shachrai, I., Zaslaver, A., Alon, U., and Dekel, E. (2010). Cost of Unneeded Proteins in E. coli Is Reduced after Several Generations in Exponential Growth. Mol. Cell 38, 758–767.        
        \cite{Shachrai2010}
        \item Maaloe1979 and Scott2010 (see Hui2015) which claims that ribosomes operate at "saturated translational capacity".
    \end{itemize} 
    
\clearpage    
}
















