



\chapter{The Ribosomal City}
\label{chapter:ribosomes}

{\color{red}
\section{Misc. todo's etc}

References that might be convenient:
\begin{itemize}
    \item Bosdriesz E, Molenaar, D., Teusink, B., and Bruggeman, F.J. CHAPTER 6 How fast-growing bacteria robustly tune their ribosome concentration to approximate growth rate maximisation. Manuscr. Accept. Publ. FEBS J., 119–153.
    \cite{BosdrieszE}
    \item Maeda, M., Shimada, T., and Ishihama, A. (2015). Strength and Regulation of Seven rRNA Promoters in Escherichia coli. PLoS One 10, 1–19. \cite{Maeda2015}
    \item Gyorfy, Z., Draskovits, G., Vernyik, V., Blattner, F.F., Gaal, T., and Posfai, G. (2015). Engineered ribosomal RNA operon copy-number variants of E. coli reveal the evolutionary trade-offs shaping rRNA operon number. Nucleic Acids Res. 43, 1783–1794. \cite{Gyorfy2015}
    \item \textit{(The following two articles talk about the ribosomal RNA deletion strains, also surprising interactions with antibiotics. Of main importance is the observation that mainly tRNA seems to be limiting. See also unpublished \cite{Quan2013}.)} Bollenbach, T., Quan, S., Chait, R., and Kishony, R. (2009). Nonoptimal Microbial Response to Antibiotics Underlies Suppressive Drug Interactions. Cell 139, 707–718. \cite{Bollenbach2009} AND 1. Quan, S., Skovgaard, O., McLaughlin, R.E., Buurman, E.T., and Squires, C.L. (2015). Markerless Escherichia coli rrn Deletion Strains for Genetic Determination of Ribosomal Binding Sites. G3 Genes|Genomes|Genetics 5, 2555–2557. \cite{Quan2015} \textit{Quan et al. also refer to Orelle et al. 2013; Polikanov et al. 2014b; Orelle et al. 2015.}
    \item \textit{(This article details the rRNA regulation.)} Schneider, D.A., Ross, W., and Gourse, R.L. (2003). Control of rRNA expression in Escherichia coli. Curr. Opin. Microbiol. 6, 151–156. \cite{Schneider2003} 
    \item \textit{(These two article with description of the structures generated from rRNA genes, also mention the cleavage of the RNA genes.)}  Kaczanowska, M., and Ryden-Aulin, M. (2007). Ribosome Biogenesis and the Translation Process in Escherichia coli. Microbiol. Mol. Biol. Rev. 71, 477–494. \cite{Kaczanowska2007} AND Shajani, Z., Sykes, M.T., and Williamson, J.R. (2011). Assembly of bacterial ribosomes. Annu. Rev. Biochem. 80, 501–26. \cite{Shajani2011}
    \item \textit{(Following article measures burden on ribosome and effect on growth in a dynamic environment.)} Shachrai, I., Zaslaver, A., Alon, U., and Dekel, E. (2010). Cost of Unneeded Proteins in E. coli Is Reduced after Several Generations in Exponential Growth. Mol. Cell 38, 758–767.
    \cite{Shachrai2010}
\item Maaloe1979 and Scott2010 (see Hui2015) which claims that ribosomes operate at "saturated translational capacity".
\end{itemize} 


}

\section{Introduction}


{\color{red}[This text needs a storyline:]}
%
%Ribosomes play a central role in biology.
%
In 1958, Francis Crick, who discovered the structure of DNA with James Watson and Rosalind Franklin,
formulated what is called the central dogma of molecular biology.
This declares that DNA holds the sequence information on how to build a protein, the sequence information is first translated to RNA, which is then transcribed into a protein \cite{Crick1958}.
% 
It relates immediately to some of the features that are said to define life: 
the capability to store information and 
the regeneration of components from scratch \cite{Lawrence2005, Koshland2002}.
%
Also in 1958, "new cytoplasmic components" were discovered under an electron microscope \cite{Palade1955}.
These components turned out to be the component in the cell that performs the last step of the central dogma: 
the synthesis of proteins using mRNA molecules as templates; also known as transcription.
The components became known as ribosomes.

Given the central place of the ribosome in the circle of life, and the idea that early life consisted only of RNA molecules that could catalyze other reactions (the "RNA world" \cite{Campbell2002}), it is not surprising that this component of the cell consists of both catalytic RNA and proteins.
As such, it is not only an embodiment of the central dogma, but also the exception to the rule.

% life
% Lawrence: information storage, catalysts, energy relation environment, grow and reproduce, respond stimuli
% Koshland: program, improvisation (MW: long term adaptation), compartmentalization, energy, regeneration, adaptabilty (short term adaptation), seclusion (being able to separate different processes, e.g. enzyme's specifity).

\section{Structure of the ribosome}

Nowadays, the structure of the ribosome has been unraveled.

It is known that the ribosome consists of 3 RNA molecules and 54 proteins \cite{Chen2012}.%, Lodish2003}.

% about the structure
% AA, B0 and 
structure retrieved from
http://www.rcsb.org/pdb/explore.do?structureId=4v4a
structure visualized with UCSF Chimera \cite{pettersen2004}.



% effects on ribosomes 
% > regulation by? 
% > find out if there are global regulators
% 
% effects of ribosomes
% > they build themselves
% > they build anything in the cell

\begin{figure}
    \centering
    \includegraphics[width=0.9\textwidth]{riboFig1.pdf}
    \caption{ 
        (A) \red{XXX ribosome} 
    }
    \label{fig:modeldrawing}
\end{figure}


\section{The Ribosome}





The ribosome is a vast RNA-protein complex. 
This complex is responsible for translating messenger RNA (mRNA) to proteins.


When expressed, any of \textit{E. coli}'s approximately 4288 genes \cite{Blattner1997} is first converted in messenger RNA (mrNA), and then into protein.
Despite this fact, only 3-4.5\% of the cell's RNA is mRNA.
%(organized into 2584 operons)


3-4.5\% mRNA
73-80\% rRNA
15-20\% tRNA

\cite{Norris1972}

%(For 55 minutes: 80\% and 15\%)

\section{Labeling proteins of the ribosome}

Table \ref{tab:ribosubunits} shows a list of the subunits we have currently labeled.

Given that the ribosome
			

\begin{table}                       
\begin{tabular}{ l l l l }
    \centering
    subunit & gene name & operon & fluorescent label \\ 
    \hline
    L31 & rpmE & none & Cerulean, mCherry \\ 
    L19 & rplS & rpsP-rimM-trmD-rplS & Cerulean, mCherry \\ 
    L9 & rplI & rpsF-priB-rpsR-rplI & mCherry \\ 
    S2 & rpsB & ttf-rpsB-tsf & Venus\\ 

\end{tabular}
    \caption{Ribosomal subunits that we have labeled.} \label{tab:ribosubunits}
\end{table}



\subsection{rpmE (L31)}
\begin{description}
    \item[General description.] lore ipsum
    \item[Operon.] lore ipsum
\end{description}



\subsection{rplS (L19)}


\subsection{rplI (L9)}


\subsection{rpsB (S2)}




\subsection{Acknowledgements}
Molecular graphics and analyses 
% addition MW
of crystal structures
%
were performed with the UCSF Chimera package. Chimera is developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco (supported by NIGMS P41-GM103311). 





























