



%\chapter{The Ribosomal City}
%\chapter{The Ribosome, a puzzling protein-rna complex}
\chapter{Ribosomal dynamics, a puzzling affair}
\label{chapter:ribosomes}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Picture of the ribosome
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering    
    %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
    \includegraphics[width=0.8\textwidth]{ribosomes_allproteinscolored.png}
    \caption{ 
        \textbf{Picture of the ribosome.}
        The ribosome consist of a small and a large subunit. These two come together on messenger RNA templates to perform protein synthesis.
        The complete ribosome is made of 3 ribosomal RNA molecules and more than 50 proteins, shown in grey and color in the picture above, respectively \cite{Chen2013}.
        The 16S rRNA and 23S rRNA molecules function as backbone. % for the small and large subunits, respectively.
        Proteins labeled S1 up to S22 
        % (S22 being sub-stoichiometric) 
        bind to the 16S rRNA to form the small subunit, and proteins labeled L1-L36 and the 5S rRNA bind to the 23S rRNA to form the large subunit \cite{Keseler2017}.
        The Ecocyc database lists 58 ribsomal proteins, though also different numbers of ribosomal proteins are listed (ref. \cite{Chen2013} e.g. talks about 54 ribosomal proteins).
        % The 5S rRNA also binds to the large subunit.
        Interestingly, the ribosomal RNA constitutes 73-80\% of the total RNA found in an \textit{E. coli} cell (mRNA constitutes 3-4.5\% and tRNA 15-20\%) \cite{Norris1972}.
        Assembly of the ribosome requires many co-factors \cite{Chen2013}. % note also DnaK is required.
        %
        This image is based on x-ray crystallography (PDB ID: 4v4q, \cite{Schuwirth2005}). Pdb files downloaded from \texttt{www.rcsb.org} \cite{Berman2000} 
        visualized with UCSF Chimera (version 1.11.2, build 41376, \cite{pettersen2004}).
        % Previously, I seemed to have used 4v4a, which is from 2003 http://www.rcsb.org/pdb/explore.do?structureId=4v4a.        
        %
        %        
    }
    \label{fig:ribo:pictureofribo}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Introduction}

\subsection{The central role of the ribosome}

Ribosomes are life.
%
In 1958, Francis Crick, who discovered the structure of DNA with James Watson and Rosalind Franklin,
formulated what is called the central dogma of molecular biology.
This states 
% declares 
that DNA holds the amino acid sequence information required to build a protein, and that this sequence information is first translated to RNA, which is then transcribed into a protein \cite{Crick1958}.
% 
This dogma relates immediately to some of the features that are said to define life: 
the capability to store information and 
the regeneration of components from scratch \cite{Lawrence2005, Koshland2002}.
%
%Also around that time, 
Slightly earlier, in 1955, "new cytoplasmic components" were already discovered under an electron microscope \cite{Palade1955}.
These components turned out to be the components in the cell that perform the last step of the central dogma: 
the synthesis of proteins using mRNA molecules as templates; also known as transcription.
The components became known as ribosomes.
%
% Given the central place of the ribosome, in the circle of life, and 
Given the idea that early life consisted only of RNA molecules that could catalyze other reactions (the "RNA world" \cite{Campbell2002}), it is not surprising that the ribosomes 
% these components of the cell
consist of both catalytic RNA molecules and (many) proteins,
as shown in figure \ref{fig:ribo:pictureofribo}.
%
%As such, it is not only an embodiment of the central dogma, but also the exception to the rule.
Given these facts, we can say ribosomes are (a key element of) life.

% life
% Lawrence: information storage, catalysts, energy relation environment, grow and reproduce, respond stimuli
% Koshland: program, improvisation (MW: long term adaptation), compartmentalization, energy, regeneration, adaptabilty (short term adaptation), seclusion (being able to separate different processes, e.g. enzyme's specifity).

\subsection{Understanding the role of ribosomes in protein fluctuations}

In this thesis, 
we have been trying to 
% When we want to understand the cell, ribosomes are a good starting point.
%It is our aim to better 
understand % cellular dynamics.
temporal fluctuations in the concentrations of cellular components.%, which occur due to the intrinsic stochastic nature of chemical processes that underlay cellular 
%
As mentioned in earlier chapters (\ref{chapter:literaturereview} and  \ref{chapter:CRP}),
these fluctuations are a result of the intrinsic stochastic nature of chemical processes that occur in the cell.
% (see also previous chapters \ref{chapter:literaturereview} and  \ref{chapter:CRP}).
%
Given the central role of the ribosome in the cell that was just discussed, ribosomes might play a big role in such fluctuations.
%Since ribosomes are key components in the cell, 
Indeed, it is often suggested that fluctuations in ribosomal concentration could lead to in cell-wide protein concentration fluctuations \cite{Davidson2008, Raj2008, Chalancon2012, Bruggeman2018}.
%
Such concerted concentration fluctuations are also referred to as extrinsic noise (as opposed to intrinsic noise, which refers to fluctuations that only occur in one cellular species) \cite{Elowitz2002}.

\subsection{Quantifying ribosomal dynamics}
\label{section:ribos:walker}

In a previous thesis from the Tans lab, Noreen Walker 
% In previous work from our lab, Walker et al. 
quantified the dynamic relationship between single cell ribosomal expression fluctuations and growth in steady state conditions \cite{Walker2016t}.
%
The experiments involved presented many challenges, 
of which a number are described in her thesis.
%
% The results from these experiments were rather inconclusive.
%
Given previously described central role of the ribosome, 
it was hypothesized that the stochastically fluctuating concentration of ribosomes might be limiting, 
meaning that ribosomal fluctuations might result in growth rate fluctuations.
%
In single cell time lapse experiments, no clear indications were found to support this hypothesis.
%
Instead, 
ribosomal proteins L19 and L31 that were labeled with mCherry (L31-R and L19-R)
% mCherry-labeled L31 and L19 subunits (L31-R and L19-R) 
showed expression-growth cross-correlations (CCs) indicative of dilution-scenario behaviour in minimal medium, or none at all in rich medium (see chapters \ref{chapter:literaturereview} and \ref{chapter:CRP} for discussions about the use of cross-correlations to interpret dynamics, and also Kiviet et al. \cite{Kiviet2014}). 
mCerulean labeled L31 ribosomal protein (L31-C) showed expression-growth CCs with small correlations around zero delay 
% at positive delays 
%between expression and growth 
in minimal medium.
This difference between the L31-C and L31-R experiments was unexpected, as only the label was different.
%
In rich medium, the L31-C cross-correlation was consistent with the L31-R cross-correlation, and showed almost no correlation.
%
Since the ribosome mainly consists of RNA, 
a GFP reporter under the control of one of the ribosomal RNA promoters was also used (rrna-G).
Like the L31-C reporter, the dynamics of this rrna-G reporter showed positive correlations in minimal medium and only very small correlations in rich medium.
% an additional reporter was used which consisted of the ribosomal RNA promoter 
%
Also growth-expression scatter plots were used, to investigate possible interesting shapes of growth-expression relationships, but this yielded no noteworthy shapes.

In an attempt to force the cell in a scenario where ribosomes are limiting, experiments were conducted where cells were grown in the presence of sub-inhibitory concentrations of tetracycline, an antibiotic.
%
Though also interesting in itself, the outcome of such an experiment could serve as a reference to interpret other experiments.
%
In rich medium, both for the L31-R and the rrn-G reporter, the antibiotic did not result in more limiting behaviour, i.e. correlations did not become more positive.
%
Given the disparate observations, both between reporters and conditions, 
and the absence of a point of reference,
the nature of the ribosomal dynamics remained fairly elusive; it was concluded that ribosomal fluctuations perhaps do not have a pivotal role in steady state cellular growth dynamics.

\subsection{New work}

In this chapter, we will describe additional experiments that were aimed at gaining further insights in the ribosomal dynamics.
Specifically, we tried to understand whether fluctuations in ribosomal concentration could have cell-wide implications.
%
To answer this question, we tried both new experiments as well increase the throughput of existing experiments.
%
Since this is an ongoing project, some sections in this chapter will be more succinct.

\section{Results}
% > looked at dual reporters with pn25 reporter to assess effect from ribosomal concentration > protein production
    % ^> (outlook) Note that also R_CR-pq would be interesting in future with minor technical adjustment
% > looked at shift experiment to look at a (dynamic) environment where we knew cells with more ribosomes have an advantage
% > Additional semi-steady state measurements w/ and w/o antibiotics (incl. the pn25 reporter)
% > (outlook) we also looked into adding a plasmid that would introduce a burden unto the ribosomes 
%	% ^> Note that cells will adjust, so in a sense this might not increase the effect of fluctuations (perhaps put in sketch of two graphs with two optimums)
% > (outlook) we also looked into rrna reporter in delta(rna) strains
% > (outlook) maybe r-proteins are not as stochastically equivalent as sometimes assumed, ratio Y_m9:Y_ab and R_m9:R_ab not equal (2.4 and 2.3)
    % ^> what about production though, that's ratio_Y 3.18 vs. 2.15, should these ratios not be equal? --> not to C-ratio as the growth rates differ.
% > (outlook) ppgpp titration
% > (outlook) perhpas the fluctuations in total ribosome activity are a difficult function of fluctuations in all of the separate ribosomal components, which all might have their own (partially) independent contributions, and therefor it is hard to see single correlations. (the C

\subsection{Additional strains}

\begin{table}[h]
    \begin{tabularx}{\textwidth}{llXl}
        %
        \textbf{ASC number}	& \textbf{Shorthand} & \textbf{Description} & \textbf{Source}		\\
        \hline
        ASC976  &	Prrna-C, pn25-Y	&	Δphp::pn25-mVenus-cmR, Δche::Prrsa-mCerulean-kanR. (Kanamycin and chloramphenicol resistance.)  & VS \\
        %     
        ASC968	& L19-C, pn25-Y	& L19-gc-mCerulean-kanR (GC linker), Δ(…)::pn25-mVenus-cmR.	(Kanamycin and chloramphenicol resistance.)	& VS \\
        %
        ASC1058	& L9-R, S2-Y	& Also known as JE202. rplI-mCherry-KanR (L9), rpsB-venus-CmR (S2). (Kanamycin and chloramphenicol resistance.) Gift from Johan Elf lab. & \cite{Wallden2016} \\
        \hline
        %
        % ASC631 & pn25-R, lacA-G & $\Delta$lacA::gfp-cat, $\Delta$php::pn25-mCherry-kanR & \cite{Kiviet2014} \\
        ASC666 & L31-R, gltA-G  & L31::mCherry-kanR, gltA::gfpA206K-cat & \cite{Kiviet2014} \\
        ASC810	& L31-C & L31-gc-mCerulean-kanR (GC linker). (Kanamycin resistant.) & NW, VS \\
        \hline
    \end{tabularx}
    \caption{\textbf{Strains used in this chapter.} Strain ASC1058 was a kind gift from the Johan Elf lab. VS indicates these strains were created by Vanda Sunderlikova, technician in the Tans lab. ASC stands for AMOLF strain collection. Note that Y, C, G and R indicate yellow, cyan, green and red fluorescent reporters, respectively. The first two strains in this table were specifically created for experiments described in this chapter, and the third strain was requested specifically for work described in this chapter. The last two strains in the table were created earlier.}
    \label{table:ribostrains1}
\end{table}

%To be able to further explore the interaction of ribosomal expression with cellular dynamics, 
To be able to further explore the implications of ribosomal fluctuations, 
we produced additional strains.
%
Importantly, we were interested in the effect fluctuations in ribosomal expression would have on 
the single cell capacity to produce proteins.
%single cell protein production.
%
We therefore created two dual-label strains that both carried ribosomal reporter constructs, as well as constitutively expressed fluorescent reporters.
%
The constitutive reporter served as a readout for fluctuations in protein production that likely affect all proteins that are produced in the cell. 
% a proxy for protein production in general.
%
In the first strain, we fused the mCerulean fluorescent reporter sequence to the ribosomal RNA promoter (rrsa), and additionally inserted an mVenus sequence to the constitutive pn25 promoter. Both were chromosomally inserted.
%
In the second strain, we introduced an mCerulean sequence behind the chromosomal L19 ribosomal protein sequence, thus creating a strain that produces L19 ribosomal proteins that are fused to mCerulean fluorescent labels.
We also introduced the constitutive pn25-mVenus reporter to this strain.
%
These strains, the Prrna-C, pn25-Y strain and the L19-C, pn25-Y strain are listed in table \ref{table:ribostrains1}.
Additionally, supplementary table \ref{tab:ribolabeledprots} gives an overview of the operons in which the ribosomal proteins are encoded.

Furthermore, because the ribosome consists of so many ribosomal proteins, we wanted to be able to confirm 
dynamics observed for one labeled ribosomal protein, also for other labeled ribosomal proteins.
%
% potential observations in a strain that had other ribosomal proteins labeled.
For this purpose, we requested a strain from the Elf lab which had both the L9 and S2 ribosomal proteins labeled by the mCherry and Venus fluorescent proteins, respectively \cite{Wallden2016}.
This strain was kindly supplied by the Elf lab, and is also listed in table \ref{table:ribostrains1} with the shorthand notation L9-R, S2-Y.

Thus, we expanded our ability to track ribosomal dynamics by 
expanding our collection of ribosomal reporters, 
%obtaining more strains with ribosomal reporters, and additionally 
and introduced a way to probe the effect of ribosomal fluctuations on protein expression by the introduction of the pn25 reporter.

%\red{
%pn25 both L and rna, refer to table;
%elf strain}

%\subsection{An attempt to create a limiting situation}
\subsection{Antibiotic shift experiments to create limiting situations}
\label{section:ribo:shiftexperiments}

%\subsubsection{The shift experiment}

To understand the dynamics of a process, it is often convenient to grasp what happens in extreme cases.
%
We therefore attempted to device an experiment which would lead to a situation where single cells that expressed more ribosomes than the population average would have a growth advantage, thus forcing a "limiting" situation.
% 
We did this by growing cells in minimal medium in our microfluidic device 1 for a few hours, and then switching to minimal medium supplemented with antibiotics (see chapters \ref{chapter:methods} and \ref{chapter:filarecovery} for a description of the microfluidics device).
%We did this by growing cells in minimal medium in our microfluidic device 1 (see chapters \ref{chapter:methods} and \ref{chapter:filarecovery} for a description of the microfluidics device).
%
We hypothesized that right after the medium has switched, cells in the population --- which at that point are calibrated for growth in minimal medium --- that happen to express more ribosomes have a growth advantage.
%
To analyse whether this was indeed the case, we made growth-concentration scatter plots for multiple points in time during the experiment.
%
We further analysed these measurements by calculating the slope and correlation of these scatter plots for each point in time.

Some data involving an antibiotic switch was already gathered
%One experiment like this was already conducted 
by Tans lab alumni Sarah Boulinea for the L31-R, gltA-G strain (see table \ref{table:ribostrains1}); 
we also ran our analysis on this dataset.
Also, two additional experiments were conducted, one with the L31-C strain where one microcolony was analyzed, 
and one with the rrsa-C, pn25-Y strain, where three microcolonies were analyzed.
%
Before looking at the scatter plots, we first look at the 
trends in fluor concentration for all these five datasets, which are shown in figure \ref{fig:ribo:fluorsignals}. 
This figure shows that for the Prrna-C strain datasets, the signal goes up after the switch to antibiotics supplemented medium.
However, for both for the L31-R and L31-C datasets, the signal appears to be going down after the shift (though pre-shift fluctuations on the population average level of the L31-R data make it unclear what is exactly going on).
It is unclear why this happens, since ribosome inhibition should increase ribosomal demand and accordingly, expression \cite{You2013}.
%
We now turn to the scatter plots.
%As mentioned, we created growth-expression scatter plots for each time point within the dataset, for all of these datasets, and f
For brevity we only show the representative series of scatter plots that relates to the L31-C dataset, see figure \ref{fig:ribo:scatter1}. 
This figure shows that there is no clear change in correlation visible by eye after the switch to medium with antibiotics.
%
%As mentioned however, 
We further quantified the data by calculating both the slope (least square fit by Matlab's \texttt{polyfit} function) and correlation coefficient for each of the scatter plots.
%
The analyses for all five datasets are 
%This is 
shown in figures \ref{fig:ribo:switch1}-\ref{fig:ribo:switch5}.
%
The L31-R dataset (figure \ref{fig:ribo:switch1}) arguably shows a small increase in correlation approximately 200 minutes after the switch, but the correlation also decreases swiftly after that.
The L31-Y dataset (figure \ref{fig:ribo:switch2}) is difficult to interpret, 
likely the positive correlations before the switch are caused by chance as there are only a few cells in the microcolony at those points in time.
%
The first Prrsa-Y dataset (figure \ref{fig:ribo:switch3}) shows a change in signal right after the switch, but this is not seen in the two other Prrs-Y datasets (\ref{fig:ribo:switch4}-\ref{fig:ribo:switch5}).
%
%These figures show that there is no clear change in correlation observed after the switch.
In conclusion, it is difficult to interpret these datasets, as the different datasets show different trends.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Switch experiment figures
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%% General trend of signals
\begin{figure}
    \centering    
    \includegraphics[width=0.3\textwidth]{pdf_2012-11-15_pos5_L31-mCherry_fluorTrend.pdf}
    \includegraphics[width=0.3\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_fluorTrend.pdf} \\
    \includegraphics[width=0.3\textwidth]{pdf_2016-09-20_pos1_prrsa-mCerulean_pn25-yfp_fluorTrend} % added
    \includegraphics[width=0.3\textwidth]{pdf_2016-09-20_pos2_prrsa-mCerulean_pn25-yfp_fluorTrend}
    \includegraphics[width=0.3\textwidth]{pdf_2016-09-20_pos3_prrsa-mCerulean_pn25-yfp_fluorTrend}            
    \caption{ 
        \textbf{Fluorescence intensity for switch from clean medium to medium supplemental with antibiotics.}
        Red dots show single cell obsvervations, the black line indicates the colony average.
        (Top left) L31-R strain. 
        (Top right) L31-C strain.
        (Bottom three) All are rrsa-C strains.  
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.    
    }
    \label{fig:ribo:fluorsignals}
\end{figure}


%%%%%%%%%%%%%%%%%%%% Scatter plots (only one shown for brevity).
\begin{figure}
    \centering    
    \includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
    %\includegraphics[width=0.8\textwidth]{pdf_2012-11-15_pos5_L31-mCherry_summaryPlot.pdf}
    \caption{ 
        \textbf{Scatter plots for switch from clean medium to medium supplemental with antibiotics.}
        Growth (doublings per hour) is shown on the y axes, and ribosome concentration (a.u.) is shown on the x axis.
        Data shown here is from the L31-C ribosomal reporter strain. It was first grown in M9 medium (indicated by red scatter plots), and then in M9 medium plus 0.5 $\upmu$M tetracycline (indicated by blue scatter plots).     
        The labels additionally indicate the correlation $R$, and the time $t$ (in minutes).
        %
        %        
    }
    \label{fig:ribo:scatter1}
\end{figure}

%%%%%%%%%%%%%%%%%%%% Summary plots for shift experiments
\begin{figure}
    \begin{minipage}[c]{0.7\textwidth}
    %\centering    
    %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
    \includegraphics[width=0.99\textwidth]{pdf_2012-11-15_pos5_L31-mCherry_summaryPlot.pdf}
    \end{minipage}\hfill
    \begin{minipage}[c]{0.3\textwidth}
    \caption{ 
        \textbf{Quantification of the correlation between growth and ribosomal expression in the antibiotics switch experiment.}
        Data from the L31-R strain. The number of cells gradually increased over the course of the experiment (A), whilst growth rate decreased after the switch (B).
        The dynamics of growth and expression were quantified by the slope (C) and correlation coefficient (D) of the growth-expression scatter plots at each point in time.
        Data displayed in black is based on the whole colony, blue symbols indicate single cell data.
        %
        %        
    }
    \label{fig:ribo:switch1}
    \end{minipage}
\end{figure}

\begin{figure}
    \begin{minipage}[c]{0.7\textwidth}
        %\centering    
        %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
        \includegraphics[width=0.99\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_summaryPlot.pdf}
    \end{minipage}\hfill
    \begin{minipage}[c]{0.3\textwidth}
        \caption{ 
            \textbf{Quantification of the correlation between growth and ribosomal expression in the antibiotics switch experiment.}
            Data from the L31-C strain. The number of cells gradually increased over the course of the experiment (A), whilst growth rate decreased after the switch (B).
            The dynamics of growth and expression were quantified by the slope (C) and correlation coefficient (D) of the growth-expression scatter plots at each point in time.
            Data displayed in black is based on the whole colony, blue symbols indicate single cell data.
            %
            %        
        }
        \label{fig:ribo:switch2}
    \end{minipage}
\end{figure}

\begin{figure}
    \begin{minipage}[c]{0.7\textwidth}
        %\centering    
        %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
        \includegraphics[width=0.99\textwidth]{pdf_2016-09-20_pos1_prrsa-mCerulean_pn25-yfp_summaryPlot.pdf}
    \end{minipage}\hfill
    \begin{minipage}[c]{0.3\textwidth}
        \caption{ 
            \textbf{Quantification of the correlation between growth and ribosomal expression in the antibiotics switch experiment.}
            Data from the Prrsa-C, pn25-Y strain. The number of cells gradually increased over the course of the experiment (A), whilst growth rate decreased after the switch (B).
            The dynamics of growth and expression were quantified by the slope (C) and correlation coefficient (D) of the growth-expression scatter plots at each point in time.
            Data displayed in black is based on the whole colony, blue symbols indicate single cell data.
            %
            %        
        }
        \label{fig:ribo:switch3}
    \end{minipage}
\end{figure}
\begin{figure}
    \begin{minipage}[c]{0.7\textwidth}
        %\centering    
        %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
        \includegraphics[width=0.99\textwidth]{pdf_2016-09-20_pos2_prrsa-mCerulean_pn25-yfp_summaryPlot.pdf}
    \end{minipage}\hfill
    \begin{minipage}[c]{0.3\textwidth}
        \caption{ 
            \textbf{Quantification of the correlation between growth and ribosomal expression in the antibiotics switch experiment.}
            Data from the Prrsa-C, pn25-Y strain. The number of cells gradually increased over the course of the experiment (A), whilst growth rate decreased after the switch (B).
            The dynamics of growth and expression were quantified by the slope (C) and correlation coefficient (D) of the growth-expression scatter plots at each point in time.
            Data displayed in black is based on the whole colony, blue symbols indicate single cell data.
            %
            %        
        }
        \label{fig:ribo:switch4}
    \end{minipage}
\end{figure}
\begin{figure}
    \begin{minipage}[c]{0.7\textwidth}
        %\centering    
        %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
        \includegraphics[width=0.99\textwidth]{pdf_2016-09-20_pos2_prrsa-mCerulean_pn25-yfp_summaryPlot.pdf}
    \end{minipage}\hfill
    \begin{minipage}[c]{0.3\textwidth}
        \caption{ 
            \textbf{Quantification of the correlation between growth and ribosomal expression in the antibiotics switch experiment.}
            Data from the Prrsa-C, pn25-Y strain. The number of cells gradually increased over the course of the experiment (A), whilst growth rate decreased after the switch (B).
            The dynamics of growth and expression were quantified by the slope (C) and correlation coefficient (D) of the growth-expression scatter plots at each point in time.
            Data displayed in black is based on the whole colony, blue symbols indicate single cell data.
            %
            %        
        }
        \label{fig:ribo:switch5}
    \end{minipage}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Higher throughput and additional cross-correlations}

\subsubsection{Many experiments in one}

One reason that it is hard to analyse the data from the antibiotic switch experiments using microfluidic device 1, 
is
that the size of the microcolony starts out small, which leads to a 
high variability of the correlation during the first part of the experiment.
%
To produce a dataset with a constant large amount of cells, we performed additional antibiotic switch experiments in microfluidic device 2.
%
Conveniently, microfluidic device 2 allows for multi-day experiments, which allowed us 
%to increase the throughput of the experiments, and allowed us 
to also measure additional steady state expression-growth cross-correlation curves. % by prolonged multi-hour experiments.
%
Importantly, we performed these additional experiments with the new Prrna-C, pn25-Y and L19-C, pn25-Y strains, to investigate the effect of ribosomal fluctuations on protein production rates with the pn25-mVenus reporter.
%
%For example, table \ref{table:ribosomes:typicaldevice2experiment} shows the setup of a typical experiment.
An example of a typical 
%For example, the 
experimental design is given by the measurements we performed on the L19-R pn25-Y strain, which involved the following sequence of supplied medium: \textit{6 hours TY, 6 hours TY + TET, 2 hours TY, 2 hours TY + TET, 2 hours TY, 2 hours TY + TET, 2 hours TY, 6 hours M9, 6 hours M9 + TET, 2 hours M9, 2 hours M9 + TET, 2 hours M9, 2 hours M9 + TET}. (M9 indicates M9 minimal medium here, supplemented with lactose, uracil and tween20; see chapter \ref{chapter:filarecovery} for further description of used media). 
%
%By analyzing the last two hours of the 6hr sequences, we could investigate the steady state behavior of the population.
%The many switches would allow us to investigate the behavior in the case of switches. XXXXXX

Thus, to recapitulate, each experiment with microfluidic device 2 can potentially give (1) data on the ribosome-growth dynamics during antibiotic shifts, (2) data on ribosome-protein expression dynamics during antibiotic shifts, (3) additional CCs regarding ribosome-growth dynamics to characterize steady state relationships, and (4) CCs of previously uncharacterised ribosome-protein expression dynamics. 
Additionally, we could also analyse the response growth medium shifts and the response to medium shifts that occur at high frequency, but we will not focus on this here.


\subsubsection{Results of the experiments}

The large amount of data generated by microfluidic device 2 is both an advantage and a disadvantage.
%
We will not go into technical details here, but both the amount of data and also the nature of the growth in the wells presented the computer analysis with new challenges.
%
We therefore only analysed parts of the data that was produced by these experiments.% at this moment.
%
We chose to first analyse the data where we assumed cells had reached steady state; and where we can perform cross-correlation analyses (i.e. analyses mentioned in points 3 and 4 in the previous paragraph).
Additional data recorded at the times of the switches exists but has not been fully analysed yet.
%This selection of the data did not include the parts were switches were performed, 
%but only the data were we assumed cells had reached steady state growth.
%We therefore here only present cross-correlation analysis.
%
We also note that 
additional manual corrections 
% (on top of those already performed) 
could perhaps further refine the analyses that we show here. 
%
For example, some of the tracked lineages show unrealistic fluctuations which might be due to artefacts in the computer analysis and might be removed or corrected (see supplemental figures \ref{fig:ribo:branchesASC968andASC976} and \ref{fig:ribo:branchesASC1058}), % YYYYY
and also the analysis now only takes into account a selection of the total amount of imaged cells, this selection could be extended to obtain more data.

%\subsubsection{\red{Expression-growth CCs of constitutive reporter are consistent with dilution mode}}
%\subsubsection{Results from ribosome and constitutive dual reporter strains}
\subsubsection{Results from strains with ribosome and constitutive reporters}

At any rate, figures \ref{fig:ribo:CCsEmuYpn25}-\ref{fig:ribo:CCsPmuYpn25Ribo} show data obtained from the Prrna-C, pn25-Y and L19-C, pn25-Y strains.
%
Most of these CCs show large error bars, and often fall inside the range of the negative control.
This indicates that more data is required to draw definitive conclusions.
Furthermore, figure \ref{fig:ribo:meansPn25R} shows that the concentration of ribosomal RNA reporters is quite low in comparison to other reporters. This might be due a weak ribosomal binding site, 
and further emphasizes caution is required when interpreting this data.
%
Nevertheless, figures \ref{fig:ribo:CCsEmuYpn25} and \ref{fig:ribo:CCsPmuYpn25} show the CCs calculated for the pn25 constitutive expression-growth relationship for growth in minimal medium, TY medium, and TY medium supplemented with sub-inhibitory concentration of the antibiotic tetracycline (0.5 $\upmu$M).
%
Most of the concentration-growth curves show negative correlation values for negative delays.
The production rate-growth curves do not show clear trends.
We saw earlier (see chapter \ref{chapter:CRP}) that constitutive reporters often show dilution mode dynamics.
The CCs we observe here could be consistent with that transmission mode.
%
Furthermore, figures \ref{fig:ribo:CCsEmuYRibo} and \ref{fig:ribo:CCsPmuYRibo} show data from the same strains, but show the CCs for the ribosomal reporters (rRNA or L19).
The CCs for the labeled L19 protein (panel A both in figures \ref{fig:ribo:CCsEmuYRibo} and \ref{fig:ribo:CCsPmuYRibo}) do not show a clear trend.
The CCs for
the behaviour of the \textit{rrna} reporter for all measured conditions 
(shown in panels \ref{fig:ribo:CCsEmuYRibo}B-D and \ref{fig:ribo:CCsPmuYRibo}.B-D), %, which show 
%all 
show behaviour that also might be consistent with dilution mode transmission of noise; negative concentration-growth at negative delays correlations are seen both for M9 and TY medium, and also for the condition where antibiotic was added to the medium.
%
It is striking that there are no noticeable differences in the rRNA expression growth dynamics between different conditions.
%
One explanation might be that 
the promoter--reporter construct does not fully capture the dynamics of ribosomal RNA expression, as the construct does not directly label ribosomal RNA (which is something that can only be done for proteins).
%the rrna reporter might not capture all ribosomal RNA fluctuations, since it is by its nature not a translational fusion reporter such as the other ribosomal reporters.



The dual reporter strains with both ribosomal and pn25 constitutive reporters were constructed to allow us to not only study ribosome-growth dynamics, but also allow us to study the impact of ribosomal fluctuations and protein expression.
%
%We now turn to the question we can address with the ribosomal and constitutive reporter strains:
To understand whether ribosomes indeed have an effect on protein production, we look at the CCs between ribosomal concentration and constitutive reporter concentration, shown in figure \ref{fig:ribo:CCsEERiboPn25}, and CCs between ribosomal production rate and constitutive production rate, as shown in figure \ref{fig:ribo:CCsPPRiboPn25}.
%
The L19-C, pn25-Y concentration-concentration CC shows a very unclear pattern, 
but the Prrna-C, pn25-Y concentration-concentration CCs show a positive peak around $\tau = 0$ delay in various conditions, indicating concerted fluctuations.
%
Positive correlations are expected, since concerted fluctuations were observed earlier for a pair of two constitutive reporters \cite{Elowitz2002}, for groups of proteins \cite{Stewart-Ornstein2012} and in general is expected to some extent for any two proteins because of the existence of extrinsic noise \cite{Chalancon2012}.
%Thus, a positive correlations between expression is expected to be found for any two proteins in the cells, and also for our ribosomal and constitutive protein reporter pair.
%
However, here, we look at two proteins of which one reports for ribosomal concentration, which might have an impact on protein production itself.
This might change the dynamics.
%
Additional features in the CC on top of the positive peak at 0 delay might tell us something about the ribosome-protein expression dynamics.
For example, more pronounced correlations at positive delays could indicate transmission of fluctuations from ribosomes to protein expression.
%
However, The Prrna-C, pn25-Y concentration correlation for M9 medium (figure \ref{fig:ribo:CCsEERiboPn25}.B) seems to show the opposite: there is more positive correlation at negative delays.
This implies that ribosome concentration fluctuations correlates with past protein concentration fluctuations.
%
The two reporters might have different maturation times, which could lead to artificial correlations negative delay.
%
Maturation times are however in the order of tens of minutes \cite{Iizuka2011, Walker2016t}, and a discrepancy between the two is thus not expected to cause such a big effect as observed here.
%
%Hypothetically, % when ribosomes are not limiting to protein production,
A very speculative hypothesis is that 
fluctuations in protein abundance might positively regulate ribosomal production,
which might help the cell anticipate ribosomal demand.%;
%though very speculative, 
%this might explain the correlation at negative delays.

In TY medium (figure \ref{fig:ribo:CCsEERiboPn25}.C), the CC does not show a stronger correlation at negative delays.
%
One could speculate this is because ribosomes become more limiting here, thus shifting the balance from negative delays in M9 medium to positive delays in TY medium.
%
The CC in TY medium with antibiotics (figure \ref{fig:ribo:CCsEERiboPn25}.D) shows a strong background signal.
The second peak in correlation at negative delays might be an artefact of insufficient data.
%We do not have further interpretations for these conditions.
More data is required to draw any conclusions for this condition. 


The production-production CCs for the same conditions (figure \ref{fig:ribo:CCsPPRiboPn25}) all appear to show peaks at 0 delay, that are rather symmetrical around the y-axis, and appear to not show additional features.
This is consistent with aforementioned concerted protein expression fluctuations.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Dataset w/ pn25 reporters
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Overview figures of branches and means for all datasets
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% Means for ASC968andASC976 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_overview_means.pdf}
    \caption{ 
        \textbf{Population mean values of different parameters measured in different strains and conditions.}
        In the labels, (A) indicates the L19-C, pn25-Y strain and (B) the Prrna-C, pn25-Y strain.
        Growth media are indicated between brackets. 
        M9 indicates M9 minimal medium, TY indicates TY medium and TY+AB indicates TY medium supplemented with antibiotics.
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        %        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
        %        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        %        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        %        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        %        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %       A: giu_asc968_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        B: giu_asc1058_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        C: giu_asc1058_M9_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        D: giu_asc976_TY_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        E: giu_asc976_TY_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        F: giu_asc976_M9_steadymw_asc976_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        
    }
    \label{fig:ribo:meansPn25R}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Constitutive reporters 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_Y6_mean_cycCor_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between concentration of constitutive reporter and growth.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsEmuYpn25}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between production rate of constitutive reporter and growth.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsPmuYpn25}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Ribosomal reporters
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_C6_mean_cycCor_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between concentration of ribosomal reporter and growth.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsEmuYRibo}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_dC5_divAreaPx_cycCor_muP9_fitNew_atdC5_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between production rate of ribosomal reporter and growth.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsPmuYRibo}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Correlations between ribosomal expression and pn25 expression
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_C6_mean_cycCor_Y6_mean_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between concentration of ribosomal reporter and concentration of pn25 reporter.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsEERiboPn25}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_CCs_dC5_divAreaPx_cycCor_dY5_divAreaPx_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between production rate of ribosomal reporter and production rate of pn25 reporter.}
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %
        %        (A) giu_asc968_M9_steady						L19-mCerulean, 	 pn25-mVenus
        %        (B) giu_asc976_M9_steady mw_asc976_M9_steady	Prrna-mCerulean, pn25-mVenus
        %        (C) giu_asc976_TY_steady						Prrna-mCerulean, pn25-mVenus
        %        (D) giu_asc976_TY_steady_antibiotics			Prrna-mCerulean, pn25-mVenus
        %        
    }
    \label{fig:ribo:CCsPPRiboPn25}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Dataset with Elf dual reporter
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsubsection{Results from strains with two ribosomal reporters}

As mentioned, to get a wider view on ribosomal dynamics, we employed a strain which has labels on additional ribosomal proteins. 
%
This L9-R, S2-Y strain has a red fluorescent reporter on the L9 ribosomal protein, and a yellow reporter on the S2 ribosomal protein, see table \ref{table:ribostrains1}.
%
We performed measurements on this strain 
%that provide additional information 
to obtain additional data 
on the single cell dynamics between ribosomal expression and growth.
%
Furthermore, the presence of two ribosomal reporters in one strain allowed us to investigate the interaction between them.
%
To investigate the effect of ribosomal stress on ribosomal dynamics, 
we analysed strain L9-R, S2-Y 
% We show analyses from experiments with this L9-R, S2-Y strain 
growing in steady state both in M9 minimal medium and M9 minimal medium with sub-inhibitory doses of the translational inhibitory antibiotic tetracycline (0.5 $\upmu$M).
%
To check the validity of the reporters, 
%To see whether these reporters behaved as expected, 
we first looked at the population level response of the reporter concentration to the presence of antibiotics.
Figure \ref{fig:ribo:meansRR} shows that due to the presence of antibiotics, the population average growth rate decreased by 12\%, whilst the concentration of ribosomal reporter more than doubles.
%
Such large increases of ribosomal concentration are consistent with earlier observations of growth in the presence of antibiotics, although in earlier observations this large increase was accompanied by a larger decrease in growth rate \cite{Hui2015}.
%
Nevertheless, earlier ribosomal labels even showed a decrease in concentration with increasing growth rate (see section \ref{section:ribo:shiftexperiments}), 
so these labels show behavior that is much more consistent with expectations.
%
Figures \ref{fig:ribo:CCsEmuS2} and \ref{fig:ribo:CCsPmuS2} show CCs from the S2 r-protein expression-growth dynamics, and 
figures \ref{fig:ribo:CCsEmuL9} and \ref{fig:ribo:CCsPmuL9} show CCs from the L9 r-protein expression-growth dynamics.
Furthermore, figures \ref{fig:ribo:CCsEERiboribo} and \ref{fig:ribo:CCsPPRiboribo} show expression--expression CCs between the ribosomal labels.
%
Like the strains discussed earlier, the CCs showed high correlations in the negative control correlations.
This indicates that more experiments are required to make claims about this data.




Nevertheless, we can try to interpret the current data.
%
The S2 reporter concentration-growth CC for cells growing in M9 minimal medium (left panel in figure \ref{fig:ribo:CCsEmuS2}.A) 
shows negative correlations consistent with the dilution mode of noise transmission \cite{Kiviet2014}.
%
We ignore the right panel, 
given a later analysis of this experiment 
% given the pattern observed in later experiments
(see the right panel of figure \ref{fig:ribo:CCsEERiboribo}.A), which will be discussed later.
%
Interestingly, 
% figure \ref{fig:ribo:CCsEmuS2}.B, which shows 
the S2 reporter concentration-growth CC for cells growing in the same M9 medium supplemented with antibiotics, does not show these negative delays, see figure \ref{fig:ribo:CCsEmuS2}.B. 
This could be indicative of a situation where ribosomes are more limiting due to the antibiotic stress, and fluctuations transmit from ribosomal fluctuations to the growth rate of the cell.
%
Production rate-growth CCs of the S2 reporter, shown in figure \ref{fig:ribo:CCsPmuS2}, are harder to interpret, both for M9 medium and M9 medium with antibiotics.
They are however not inconsistent with the earlier observations and interpretation.

Next, let's see whether the other ribosomal reporter in this strain shows similar behaviour.
Firstly, the negative concentration-growth CCs of the L9 reporter in M9 medium --- where we again only consider the the left panel in figure \ref{fig:ribo:CCsEmuL9}.A --- are consistent with the negative concentration growth CCs of the S2 reporter.  
The dynamics in M9 medium plus antibiotics again seem different compared to the experiment without antibiotics, as negative correlations now manifest as positive delays (instead of at negative delays).
The negative correlations are however different from the positive correlations we saw for the S2 reporter in the same experiment.
%
This indicates that either repetitions of these experiments might yield different results, or one of the labels does not appropriately reflect ribosomal concentration, or different ribosomal proteins might have their own dynamics.
%
Production-growth CCs of the L9 reporter (figure \ref{fig:ribo:CCsPmuL9}) are again hard to interpret, although the condition with antibiotic stress shows a distinct peak around zero delay.

Aside from expression-growth dynamics, we can additionally look at the interaction between expression of the two ribosomal proteins in this strain. 
% 
Firstly, the right panel in figure \ref{fig:ribo:CCsEERiboribo}.A shows the S2-L9 concentration-concentration CC; which shows a shape that we have not observed before and is inconsistent with the other experimental data in the same condition (left panel in the same figure). 
%Correlations seem to increase for longer delays. 
Taken together, we think data from this specific experiment should be disregarded until more experiments are performed; as indicated earlier for this condition.
(This implies not taking into account all the right panels in figures \ref{fig:ribo:CCsEmuS2}.A, \ref{fig:ribo:CCsPmuS2}.A, \ref{fig:ribo:CCsEmuL9}.A and \ref{fig:ribo:CCsPmuL9}.A.)
%
In any case, for the other panels, we expect the CCs to show positive correlations that are symmetrical around zero delay, since we expect both reporters to represent the ribosomal concentration.
%
The left panel in \ref{fig:ribo:CCsEERiboribo}.A shows the S2-L9 concentration-concentration CC for growth in M9 minimal medium, which is --- against expectations --- not symmetric.
The S2-L9 concentration-concentration CC for growth in M9 minimal medium supplemented with antibiotics is different, but also not symmetric (figure \ref{fig:ribo:CCsEERiboribo}.B).
%
This asymmetry is consistent with the earlier differences in expression growth CCs between the two labels.
Again, it might be that 
repetitions of these experiments might yield different results, or one of the labels does not appropriately reflect ribosomal concentration, or different ribosomal proteins might have their own dynamics.



Furthermore, the CCs between the production rates of the S2 and L9 labels (figure \ref{fig:ribo:CCsEERiboribo}) do seem symmetric in both the M9 minimal medium and M9 minimal medium with antibiotics, which is consistent with aforementioned expectations.
We do not observe further noteworthy features in the production--production CCs.



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Elf strain mean overview figure
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_overview_means.pdf}
    \caption{ 
        \textbf{Population mean values of different parameters measured in L9-R, S2-Y strain experiments.}
        Shown are data from growth in M9 minimal medium ("M9") versus growth in M9 minimal medium supplemented with antibiotics ("M9+AB").
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        %        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
        %        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        %        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        %        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        %        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %
        %       A: giu_asc968_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        B: giu_asc1058_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        C: giu_asc1058_M9_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        D: giu_asc976_TY_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        E: giu_asc976_TY_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        F: giu_asc976_M9_steadymw_asc976_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        
    }
    \label{fig:ribo:meansRR}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% S2-Y ribosomal reporter
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_Y6_mean_cycCor_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between the concentration of the S2 ribosomal protein and growth.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %
%        (A) A: giu_asc1058_M9_steady, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
%        (B) B: giu_asc1058_M9_steady_antibiotics, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
    }
    \label{fig:ribo:CCsEmuS2}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between the production rate of the S2 ribosomal protein and growth.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
%
%        (A) A: giu_asc1058_M9_steady, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
%        (B) B: giu_asc1058_M9_steady_antibiotics, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
        %        
    }
    \label{fig:ribo:CCsPmuS2}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% L9-R ribosomal reporter
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_R6_mean_cycCor_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between the concentration of the L9 ribosomal protein and growth.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
%
%        (A) A: giu_asc1058_M9_steady, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
%        (B) B: giu_asc1058_M9_steady_antibiotics, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
        %        
    }
    \label{fig:ribo:CCsEmuL9}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between the production rate of the L9 ribosomal protein and growth.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
%
%        (A) A: giu_asc1058_M9_steady, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
%        (B) B: giu_asc1058_M9_steady_antibiotics, dR5_divAreaPx_cycCor_muP9_fitNew_atdR5_cycCor
        %        
    }
    \label{fig:ribo:CCsPmuL9}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Correlations between ribosomal expression r-protein1 and r-protein2
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_Y6_mean_cycCor_R6_mean_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between concentrations of two ribosomal proteins.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics.
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %        
    }
    \label{fig:ribo:CCsEERiboribo}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_CCs_dY5_divAreaPx_cycCor_dR5_divAreaPx_cycCor.pdf}
    \caption{ 
        \textbf{Cross-correlations between production rates of two ribosomal proteins.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics.
        "C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %        
    }
    \label{fig:ribo:CCsPPRiboribo}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\section{Discussion and conclusion}

% PART 1
%(-) summarize conclusions
%(-) compare conclusions between different ribosomal reporters
%(.) we need C-p correlations to check effect ribo flucs on production of proteins..

\subsection{Antibiotic shift experiments on device 1}

In conclusion, we have used different single cell approaches in an attempt to further understand the ribosome.
%
We first performed an experiment using our microfluidic device 1 to subject cells to a switch from growth in minimal medium to growth in minimal medium with antibiotics.
Based on literature, we expected that cells needed a higher concentration of ribosomes when they were exposed to translational antibiotics (see e.g. ref. \cite{You2013}), and thus that single cells that expressed more ribosomes relative to the population average would benefit shortly after the switch.
%
If we would be able to observe such single cell growth advantages, this would imply 
ribosomes are limiting in this scenario.
Their behaviour could then serve 
%limiting behaviour of ribosomes in this scenario and serve 
as positive control and point of reference for other experiments. 
%
To our surprise, the L31 reporter did not show the expected increased concentration during exposure to antibiotics after the switch.
%
This was also observed before \cite{Walker2016t} for this particular ribosomal protein, and it
raises the question how representative the L31 reporter is for the ribosomal concentration.
%
A second reporter we used, a ribosomal RNA promoter fused to a cyan fluorescent reporter protein, did show the expected increase of ribosomal concentration after the switch to medium supplemented with antibiotics.
%
The results from these experiments regarding single cell growth advantage were however inconclusive, they neither showed a clear absence nor a clear presence of an increased correlation after the switch.

\subsection{Single cell ribosome expression and growth}

To follow up on these experiments, we performed experiments using microfluidic device 2, which allows for measurements that are longer and thus allows for more switches, and additionally enables tracking a higher and constant number of cells. 
%
This set of experiments could have provided additional information on the ribosomal behavior during switches from medium that requires less ribosomal expression to medium that requires more ribosomal expression.%, such as medium with sub-inhibitory concentrations antibiotics.
Additionally, the long time lapse capabilities of the device would allow us to record data in steady state.
On top of this, we used newly constructed strains in these experiments that allowed us to measure the effect of ribosomal fluctuations on single cell protein expression, as the strains also contained a constitutive pn25 reporter construct.
%
However, the analysis of data in this device presented additional challenges for the analysis.
%The large amount of data, different cellular movements during growth in the wells and image features of cells in this device are slightly different in this device, presenting additional challenges during the analysis.
%
We therefore only analysed a subselection of the data 
and focused on analyses of steady state behaviour.
%that was acquired during the experiments performed with microfluidic device 2.

On top of this, the data that we did analyse showed a very high background signal, which indicated that 
more data needs to be analysed or additional experiments are required
% more experiments are needed 
to draw definitive conclusions.
%
Nevertheless, we made several observations regarding the interaction between single cell ribosomal fluctuations and single cell growth rate fluctuations.
%
Firstly, in experiments involving the ribosomal RNA reporter, we saw that cross-correlation analysis in most conditions could be consistent with dilution mode transmission of noise.
This would be consistent with the idea that ribosomal RNA does not affect growth rate, but growth rate fluctuations instead result in ribosomal fluctuations.
% > REMEMBER TO LATER POINT OUT THAT RRNA MIGHT NOT BE REPORTED FOR CORRECTLY
%
Secondly, in experiments in M9 minimal medium involving labeled ribosomal proteins (L9 and S2), we also saw cross-correlations that might be consistent with dilution mode dynamics.
However, when this strain grew on M9 minimal medium supplemented with antibiotics, cross-correlations were less consistent with dilutions mode dynamics.
Hence, contrary to what the r-RNA data indicated, this data suggested that ribosomal fluctuations could become more consequential for the cell when it experiences translational stress.

Similar experiments were described earlier by Noreen Walker \cite{Walker2016t}, 
we briefly summarized her experimental observations in section \ref{section:ribos:walker} of this chapter.
%
Observations by Walker on a strain with a ribosomal RNA reporter in medium M9 medium supplemented with acetate (our lactose-supplemented M9 sustains a higher growth rate) or defined rich medium did not show dilution mode behaviour, which could mean ribosomal RNA shows different dynamics in different media.
%
Some of the experiments involving L19 and L31 reporters that she performed showed dilution mode behavior in M9 minimal medium, similar to our observations on the labeled S2 and L9 ribosomal proteins in minimal medium, but some of her experiments did not. (This depended on the fluorescent label that was used, mCherry or mCerulean respectively.)
%
Walker also performed CC analyses of steady state growth in rich medium supplemented with antibiotics, which resulted only in subtle changes in the dynamics, both for a labelled L31 ribosomal protein and for a ribosomal RNA promoter reporter construct.
%
It is unclear how these results obtained in different media connect to our results.

\subsection{Single cell ribosome expression and protein expression}

In this chapter, we also performed experiments on strains that carried both a ribosomal expression reporter and a constitutive reporter.
The latter reporter was intended to probe the effect of ribosomal expression fluctuations on the cell's ability to produce proteins.
%
We observed positive correlations in the CCs between the concentrations of these two reporters. 
%
Such correlations are however 
%The positive correlations were however hard to interpret since 
%
%they are 
expected between the expression of any two proteins, 
and thus do not necessarily imply that there is an effect of ribosomal expression fluctuations on protein production in the cell.
%
We did observe additional features in the CCs. 
These depended on the medium in which the cells were growing.
%
In most media, these features were not consistent with an effect of ribosomal fluctuations on protein production,
except in TY.
%
In TY medium we observed additional correlations between constitutive protein expression and past ribosome expression,
which is consistent with an effect of ribosomal fluctuations on protein production rates.
%
TY medium sustains very fast bacterial growth rate, 
which could explain an increased sensitivity to ribosomal fluctuations.
%
As mentioned however, more experiments are needed to draw definitive conclusions.


To shed further light on these matters, one could additionally look at the cross-correlation between
the concentration of the ribosomal reporter and the production of the constitutive reporter.
%
This would require a rather straightforward extension of our analysis scripts.


\subsection{The ribosome is a complex structure}

% PART 2
% measuring the ribosome is not simple ... 
%(a) ribosomes are big and it is difficult to pick exactly what to label
%(b) it is difficult to label RNA properly
%(z) mention the order in which ribosomes are formed \cite{Chen2013} 
% 		<---> perhaps not required for my story to mention this? Chen2013 was cited anyways, which should be the case

We have discussed many experiments, which employed different ribosomal labels and probed dynamics in different conditions.
%
Different conditions sometimes showed different results.
%
Often however, results also depended on which ribosomal protein was tracked.
%
One explanation for that observation is that ribosomal subunits have their own dynamics.
%
Though the ribosome is often viewed as a complex that has a fixed composition, 
recent discoveries in eukaryotes show that this might actually not be the case \cite{Preiss2016, Slavov2015}.
%
Purification and mass-spectrometry of ribosomes in yeast and mouse embryonic stem cells showed that 
pools of ribosomes with different ribosomal protein composition exist, which could also be linked to functional properties \cite{Slavov2015}.
%
This raised the suggestion that the cell can modulate expression of different ribosomal proteins separately, 
to achieve specific regulatory goals.
%
It is not infeasible this is also the case in \ecoli, and this would explain why experiments tracking different ribosomal proteins show different dynamics.
%
This potential added layer of regulation makes it more difficult to understand the effect of ribosomal concentration fluctuations.
Coincidentally, for eukaryotes, it has been observed that especially ribosomal surface proteins vary.
Surface proteins are also convenient targets for labelling.
In fact, the L9, S2 and L19 proteins that we  labelled are also on the surface of the ribosome, see figure \ref{fig:ribo:labelsPicGiulia}.
Also the L31 protein that we labelled is only loosely attached to the ribosome \cite{Walker2016t}.
%
Consistent with the view that the ribosomal proteins are not expressed equally for functional reasons, 
not all operons from which the ribosomal proteins are expressed are regulated by exactly the same regulators \cite{Keseler2017}.
%
Additionally, some ribosomal subunits are observed to be essential, whilst others are not (i.e. some respective null mutants are viable, whilst others are not).
This further illustrates that different ribosomal proteins might have different functional expression profiles.
%
From the proteins that we labelled, L9 and L31 are non-essential, and S2 and L19 are essential.
%
%This suggests that it could be that 
%single cell fluctuations in different ribosomal proteins might thus have a different effects on growth rates.
This distinction might also be reflected in different relationships between expression and growth rate for the different ribosomal proteins, 
though our data might be too limited to draw conclusions about this.
%
%This is consisent with our observations that the shape of CCs sometimes depended on which ribosomal protein was labelled.
%
Concluding, understanding ribosomal dynamics based on labelling ribosomal proteins might prove a challenging task.

\begin{figure}
    \centering
    \includegraphics[width=0.70\textwidth]{riboLabelsGiuliaCombined.png}
    % \includegraphics[width=0.49\textwidth]{ribosomes_L19.png}
    \caption{ 
        \textbf{Some of the ribosomal proteins that were labeled for experiments in this chapter.}
        The structure of the ribosome is displayed twice in grey, with on the left the L19 ribosomal protein highlighted in red, and on the right the L9 and S2 ribosomal subunits highlighted in red and yellow, respectively.
        This image is based on x-ray crystallography (PDB ID: 4v4q, \cite{Schuwirth2005}). Pdb files were downloaded from \texttt{www.rcsb.org} \cite{Berman2000} 
        visualized with UCSF Chimera (version 1.11.2, build 41376, \cite{pettersen2004}). These two visualizations were made by Giulia Bergamaschi.
        %        Giulia's text with this figure:
        %         Label location on the ribosomes; the fluorescent reporters were attached to the colored
        %         proteins. Left: labelling in strain asc1058: the L9 (red) protein has been tagged with mCherry in the
        %         large subunit, while S2 (yellow) has been labelled with mVenus in the small one. Right: labelled
        %         protein L19 (mCerulean) in strain asc968 in blue (created with UCSF Chimera visualisation software,
        %         www.cgl.ucsf.edu/chimera).
        %        
    }
    \label{fig:ribo:labelsPicGiulia}
\end{figure}

\section{Outlook}

\subsection{Making ribosomes limiting}

% PART 3
%(c) when we "stress" the ribosomes a new equilibrium sets in
%    (c) point out that ribosome might adapt itself to circumstances by regulation, and so it is logical that dynamic fluctuations have similar effects in different conditions
%(d) would be nice to be able to titrate the ribosomes as we did with lac --> introduce new constructs
%    (-) either key protein
%    (-) or rip out whole r-RNA parts and fill up again

%> making limiting is desired
%> just stressing is probably not useful
%  ^>  new equilibrium
%> instead perhaps rna plasmid construct

As mentioned in the results section, it can sometimes be very useful to create an extreme situation to better understand the dynamics of a system.
%
For ribosomal dynamics, an extreme situation might be one where ribosomes are limiting, 
%Thus, creating a situation where ribosomes are limiting in single cells, 
i.e. a situation in which single cell growth rates correlate strongly with ribosomal expression.%, might allow us to better understand ribosomal dynamics.
%
This topic is also discussed in the previously mentioned thesis by Walker \cite{Walker2016t}. 
%
We think it is not straightforward to create such a situation.
%
For example, whilst it might be a sensible idea that adding sub-inhibitory concentrations of antibiotics to the cellular growth medium might cause translational stress and 
thus more limiting behaviour of ribosomes,
the addition of antibiotics to growth medium was observed to have limited consequences in experiments.
%
% This could be explained when 
One explanation for this 
might be that a population of cells reacts to an antibiotic disturbance by altering its ribosomal expression level 
in such a way that local deviations from the newly established level have similar effects as deviations from the original expression level.
%This might be explained by a scenario where the population reacts to an antibiotic disturbance by altering its ribosomal expression level to cope with the translational stress,
%but where deviations from that new expression level show similar dynamics as deviations from the original situation with the lower ribosomal expression level
% in the original situation.
This idea is further illustrated by the cartoon in 
%How this might work is depicted in 
figure \ref{fig:ribo:optimumCartoon}.
%
\begin{figure}
    \centering
    \includegraphics[width=1.0\textwidth]{optimum_figure.pdf}
    % \includegraphics[width=0.49\textwidth]{ribosomes_L19.png}
    \caption{ 
        \textbf{Cartoon of how ribosomal expression might affect growth rate.}
        Ribosomal expression in single cell deviates from the population average. 
        Both the average and single cell expression levels might not lie exactly at the optimum.
        Low expression might lead to slower growth because ribosomes become limiting.
        High expression might decrease growth rate because resources to produce these ribosomes are drawn from other essential cellular processes.
        Panel (a) shows how the expression-growth relationships in single cells might relate between cells not exposed to antibiotics (black line) and cells exposed to antibiotics (red line). Panel (b) shows that these relationships might be similar if we consider only the deviations from the population averages (which are here assumed to coincide with optimal expression levels). $R$ and $\mu$ respectively indicate ribosomal expression and growth rate, $\Delta{R}$ and $\Delta{\mu}$ indicate mean-subtracted respective values.%these respective values after the average value has been subtracted.
    }
    \label{fig:ribo:optimumCartoon}
\end{figure}
%
Different stressors instead of antibiotics might be used to put the cell under translational duress.
%
To create an alternative situation where ribosomal capacity is challenged, 
we constructed strain ASC1088 (see table \ref{table:ribo:strains2}) which carries a plasmid that expresses the mCherry protein at a high expression level.
%
Similar to the antibiotic treatment, protein over--expression might also put a large burden on ribosomal capacity, thus increasing the effect ribosomal fluctuations have on growth rate (see Walker's thesis for a longer discussion on this topic \cite{Walker2016t}).
%
However, partially based on earlier observations of the limited effect of unneeded protein expression \cite{Shachrai2010},
we suspected that experiments with over-expression might yield similar results as the antibiotic experiments, 
we abandoned experiments with this strain.

% Xx future experiments could overcome xx.
%There might be ideas for future experiments to force ribosomes to become limiting.
Future experiments might be devised in which ribosomes are forced to become limiting.
%
As also mentioned by Walker \cite{Walker2016t}, previously constructed \textit{rrs} null mutants \cite{Condon1993, Condon1995, Quan2015, Bollenbach2009, Gyorfy2015} might provide a venue towards this aim.
%
Given the role of the ribosomal RNA as backbone for the ribosome, shortages in ribosomal RNA content might have profound effects on the cellular growth rate.
%
Ribosomal RNA is expressed from multiple partially redundant operons labelled \textit{rrna} up to \textit{rrnh}, as shown in supplementary figure \ref{fig:ribo:tabledeltarrn}. 
%
% The null mutants previsouly created by 
Bollenbach et al. \cite{Bollenbach2009} created several null mutants which had more and more of these operons removed, as also shown in supplementary figure \ref{fig:ribo:tabledeltarrn}. 
%
In these null mutants, ribosomal RNA fluctuations might have a large effect on growth rate.
Ribosomal proteins on the other hand might still be expressed at normal levels, and thus would be present at plentiful levels.
Hence, the concentration of ribosomal RNA might be limiting, whilst the concentration of ribosomal proteins is not.
The expression of ribosomal RNA is however hard to track directly.
%
One way for future experiments to get around this might be the creation of a plasmid that expresses ribosomal RNA and a reporter simultaneously.
%
A fluorescent reporter sequence (and ribosome binding site) could be fused at the end of an \textit{rrn} operon sequence, which could be expressed from a plasmid in one of the deletion strains.
%
This would result in both rRNA and coding RNA being expressed simultaneously, and thus allow for a way to track whether 
fluctuations in ribosomal content can affect single cell growth rate.
%
Some research on how such a construct could look was already performed and can be found in supplementary note I (section \ref{supp:ribo:1}) to this chapter.


% FINAL CONCLUSIONS / OUTLOOK

Furthermore, single cell methods are developing rapidly, and future advances might help us to further understand the complex dynamics between the ribosome, protein production and cellular growth rate.


% DISCARDED conclusions
% (d) take the gel pad as reference and point out that it would be nice to lower negative controls to the same level
% (b) would be nice to re-run some of noreen's analyses with the script that creates the controls.

% LITERATURE CURRENTLY NOT CITED:
% Schneider2003: rRNA regulation -- Schneider, D. A., Ross, W., & Gourse, R. L. (2003). Control of rRNA expression in Escherichia coli. Current Opinion in Microbiology, 6(2), 151–156. http://doi.org/10.1016/S1369-5274(03)00038-9      

\section{Methods}

Where minimal medium is mentioned, this refers to M9 minimal medium (also described in chapter \ref{chapter:filarecovery}).
%
Minimal medium in this chapter was always supplemented with uracil (0.2 mM), lactose (0.1\% g/mL).
TY medium is described in chapter \ref{chapter:filarecovery}.
%
During time lapse experiments under the microscope, tween20 (0.001\% g/mL) was added to the medium additionally.
%
When antibiotics are mentioned, this refers to tetracycline, which was used at a concentration of 0.5 $\upmu$M.
%
All experiments were conducted as described in the methods section of chapter \ref{chapter:filarecovery}.
See also chapter \ref{chapter:methods} for further information.

\section{Acknowledgements}

%I am indebted to Noreen Walker and Giulia Bergamaschi regarding this chapter.
%
I am thankful to Noreen Walker, who fostered the ribosome project for 4 years, for her collaboration and input regarding these additional experiments.
%
Much of the data shown in this chapter were taken by Giulia Bergamaschi, master student in the Tans lab at that time. I am thankful for 
the mountains of work she performed in only three months time.
%her hard work. 

Molecular graphics and analyses 
% addition MW
of crystal structures
%
were performed with the UCSF Chimera package (production version 1.11.2, build 41376). Chimera is developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco (supported by NIGMS P41-GM103311). 




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Supplementary stuff
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\FloatBarrier
\clearpage

\section{Supplementary note I}
\label{supp:ribo:1}

\subsection{Making rRNA concentration limiting and traceable}
%\subsection{Creating a limiting rRNA concentration that can be tracked}
%\subsection{A strain in which rRNA concentration is limiting and can be tracked}
%\subsection{Creating a strain where rRNA is limiting and can be tracked}

%\subsection{Introduction}

%As mentioned in the main text, 
%ribosomal RNA components are the backbone of the ribosome. There are three of these RNA components in the ribosome, which are 16S (part of the small ribosomal subunit), 23S and 5S in size (both part of the large ribosomal subunit), respectively encoded by the rrs, rrl and rrf genes. The genes that produce these RNA components occur in multiple copies, and are organized into multiple (seven) operons that always contain the genes for these three RNA components, and additionally also copies of tRNA genes. The seven RNA operons are referred to as rrnX, with X a label ranging from A to H. See also supplementary figure \ref{fig:ribo:tabledeltarrn}.
%%
%Previously, strains with knockouts of these RNA genes have been made to investigate the properties of the operons. These knockouts had one up to six copies of the rrn operons lacking; in the cases where 5 or 6 operons were removed, additional tRNA genes expressed from plasmids were introduced to ensure viability of these strains.

In this supplementary section, we aim to show how one could create an artificially limiting situation involving ribosomal genes and also track ribosomal expression to show this. 
Or in other words, how to create a strain in which traceable ribosomal RNA fluctuations have an effect on the growth rate. 
%
%\subsection{Possible method}
%
As briefly described in the main text, the $\Delta$\textit{rrn} strains created by Bollenbach et al. \cite{Bollenbach2009} could experience ribosomal RNA limitations.
%
One could introduce a plasmid containing a ribosomal RNA operon sequence, followed by a ribosomal binding site sequence and fluorescent reporter sequence, 
to investigate correlations between additional ribosomal RNA expression (from the plasmid) and single cellular growth rate.
%
To achieve this, one could either use the pK4--16 plasmid constructed by Quan et al. \cite{Quan2013} --- which contains the \textit{rrnb} operon --- as a starting point, or clone the \textit{rrnb} operon into another vector (for which one could use primers sequences for \textit{rrnb} also published by Quan et al \cite{Quan2013}).
%
The new vector should have a low copy number and its antibiotic resistance module should not be the same as the modules used in strains the vector might be introduced to (ampicilin might be a convenient choice, but not streptavadin or chloramphenicol).
%
To track rRNA expression one could add a 
%To add the 
fluorescent reporter (e.g. mCerulean2.0) to the \textit{rrnb} operon, 
which could be inserted after the coding sequences and accompanying RNA cleavage sites, but before the operon's terminator sites.
%
The structure of the operon and the cleavage and terminator sites are described in Kaczanowska et al. \cite{Kaczanowska2007} (see also Shajani et al. \cite{Shajani2011}),
%
and terminator sites are also annotated in the Ecocyc database\footnote{In the \ecoli genome, the region with the terminators lies between nucleotide 4171756 and 4171760, and an annotated sequence can be found via \texttt{https://ecocyc.org/ECOLI/select-gen-el}.} \cite{Keseler2017}.
%
In addition to a fluorescent protein sequence itself, the reporter sequence would also have to contain a ribosome binding site, which could be synthetically made using the Salislab ribosome binding site (RBS) calculator\footnote{Using \texttt{https://salislab.net/software/} and the mCerulean2.0 sequence, an RBS sequence might look like: \newline\texttt{CTTTCAATCAATTCGTAATTTTAGATAAATAATAAGGAGGTCTCTAA}.} \cite{Salis2009, Salis2011}. 
Furthermore, to prevent interference with ribosomal RNA maturation, a linker could be introduced between the ribosomal RNA sequences and the reporter sequence.
%
The resulting plasmid could be introduced in strain $\Delta$2 (also known as CGSC 12340 and SQ40), which lacks the \textit{rrne} and \textit{rrng} operons that are both similar to \textit{rrnb}, see table \ref{fig:ribo:tabledeltarrn}.
%
The copy number of a low copy number plasmid would likely be variable in single cells, which aids towards the goal of investigating the effect of different single cell expression levels of the ribosomal RNA.

\subsection{Additional considerations}

An additional consideration might be the strength of the \textit{rrn} promoters.
These are different per operon, the \textit{rrne} is by far the strongest \cite{Maeda2015}.
The \textit{rrnb} promoter strength is two to several times less depending on the condition.
This is compatible with our aim to keep ribosomal RNA expression limited.

A challenge to this strategy might be that 
%There are several challenges that can be identified in pursuing this strategy.
%Firstly, a study by Quan et al. \cite{Quan2015} showed that 
tRNA turns out to be limiting, as suggested by Quan et al. \cite{Quan2015}.
tRNA is also expressed from the \textit{rrn} operons. To make sure we are not mistaking tRNA limitations for ribosomal limitations, additional artificial tRNA expression (for example from plasmid ptRNA67 described in Zaporojets et al. \cite{Zaporojets2003,Quan2015}) could be a tool to make sure this is not the case.



%\subsection{more stuff}
%additionally add some mrna sequence??
%
%COMPLICATIONS:
%% Maeda2015 shows different expression strengths of operons
%Quan2015 shows especially tRNA might be limiting, and also shows growth rate defect is limited
%Quan2013 already published a plasmid that expresses the rrnB operon
%Kaczanowska2007 provides a detailed map of the rrnB operon

\begin{figure}
    \centering
    \includegraphics[width=1.0\textwidth]{tableRrna.png}
    \caption{ 
        \textbf{Table showing the \textit{rrn} operons in \ecoli with cartoon overlay indicating different \textit{rrn} null mutant strains created by Bollenbach et al..}
        %Information on operon structure was obtained from ref. \cite{Keseler2017}, \textit{rrn} null mutants where published in ref. \cite{Bollenbach2009}.
        Ribosomal RNA in \ecoli is expressed from multiple partially overlapping operons. 
        Operons contain different ribosomal parts (and tRNA molecules). 
        \textit{rrs} encodes 16S rRNA, \textit{rrl} encodes 23S rRNA, and \textit{rrf} encodes 5S rRNA.
        Other abbreviations indicate genes that encode tRNA molecules.
        This is also indicated in the second line of the "ribosomal parts" column. 
        Symbols in the third column indicate equivalence between the operons.
        The last column indicates regulators of the operons, symbols in brackets indicate evidence is not definitive.
        (This information was obtained from the online database Ecocyc \cite{Keseler2017}).
        The overlayed crosses and circles indicate the null mutants that were constructed by Bollenbach et al. \cite{Bollenbach2009}.
        Each column of crosses corresponds to one strain, a cross indicates that the respective strain lacks the operon indicated. A circle indicates additional tRNA was expressed from a plasmid.
        These strains were also referred to as $\Delta$1-6, where the number corresponds to the number of operons that were removed.
    }
    \label{fig:ribo:tabledeltarrn}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% MORE SUPPLEMENTS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\FloatBarrier
\clearpage

\section{Supplemental figures and tables}
\FloatBarrier

\begin{table}
    \begin{tabular}{ l l l l }
        \centering
        r-protein & Gene name & Operon & Fluorescent label \\ 
        \hline
        L31 & rpmE & none & Cerulean, mCherry \\ 
        L19 & rplS & rpsP-rimM-trmD-rplS & Cerulean, mCherry \\ 
        L9 & rplI & rpsF-priB-rpsR-rplI & mCherry \\ 
        S2 & rpsB & ttf-rpsB-tsf & Venus\\ 
    \end{tabular}
    \caption{Labeld ribosomal proteins in the Tans lab. 
        %    Ribosomal proteins that we have labeled.
    } \label{tab:ribolabeledprots}
\end{table}

\begin{table}[h]
    \begin{tabularx}{\textwidth}{llXl}
        %
        \textbf{ASC number}	& \textbf{Shorthand} & \textbf{Description} & \textbf{Source}		\\
        \hline
        %      									
        ASC656	& L31-R & L31-mCherry-kanR (no linker).	(Kanamycin resistant.) & NW, VS \\
        ASC657	& L19-R & L19-mCherry-kanR (no linker).	(Kanamycin resistant.) & NW, VS \\
        ASC680	& L31-R, Prrn-G & L31-mCherry-kanR (no linker), Δ(cheZ)::Prrn-GFP-catR. (Kanamycin and chloramphenicol resistant.)	& NW, VS \\
        ASC779	& Prrn-G& Δ(cheZ)::Prrn-GFP, rrsa promoter. (No resistance.) &	NW, VS \\
        
        \hline										
        & L19-C & 	L19-mCerulean L19-gc-mCerulean-kanR (GC linker)	(Kanamycin resistance.)	& VS \\
        & Prrn-C & 	Δ(cheZ)::Prrn-mCerulean-kanR	(Kanamycin resistance)	& VS \\
        \hline
        ASC1088 & mCherry+
        % L19-C, pn25-Y, p-pn25-R 
        &	ASC968 + high copy plasmid w. pn25-mCherry (Resistances: kanamycin, chloramphenicol, ampicilin.) & VS \\
        \hline
    \end{tabularx}
    %Strains used in previous work and 
    \caption{\textbf{Miscellaneous strains.} NW indicates these strains were used in Noreen Walker's thesis \cite{Walker2016t}. VS indicates these strains are created by Vanda Sunderlikova.}
    \label{table:ribo:strains2}
\end{table}



% Branches for ASC968andASC976 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_ASC968andASC976_branches_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Growth of single cells of the different strain populations in the different conditions.}
%        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
%        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %"C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %
        %       A: giu_asc968_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        B: giu_asc1058_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        C: giu_asc1058_M9_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        D: giu_asc976_TY_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        E: giu_asc976_TY_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        F: giu_asc976_M9_steadymw_asc976_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        
    }
    \label{fig:ribo:branchesASC968andASC976}
\end{figure}

% Branches for ASC1058 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{pdf_riboASC1058_branches_muP9_fitNew_cycCor.pdf}
    \caption{ 
        \textbf{Growth of single cells of the different strain populations in the different conditions.}
        (A) L9-R, S2-Y strain. Grown in in M9 minimal medium.
        (B) L9-R, S2-Y strain. Grown in in M9 minimal medium supplemented with antibiotics. 
%        (A) L19-C, pn25-Y strain. Grown in M9 minimal medium.
%        (B) Prrna-C, pn25-Y strain. Grown in M9 minimal medium.
%        (C) Prrna-C, pn25-Y strain. Grown in TY medium.
%        (D) Prrna-C, pn25-Y strain. Grown in TY medium supplemented with antibiotics.
        %"C", "Y" and "R" indicate the color of the fluorescent labels: cyan, yellow and red respectively.
        %
        %       A: giu_asc968_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        B: giu_asc1058_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        C: giu_asc1058_M9_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        D: giu_asc976_TY_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        E: giu_asc976_TY_steady_antibiotics, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        F: giu_asc976_M9_steadymw_asc976_M9_steady, dY5_divAreaPx_cycCor_muP9_fitNew_atdY5_cycCor
        %        
    }
    \label{fig:ribo:branchesASC1058}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Notes to self
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%\clearpage
%
%{\color{red}
%    \section{Misc. todo's etc}
%    
%    References that might be convenient:
%    \begin{itemize}
%        \item Bosdriesz E, Molenaar, D., Teusink, B., and Bruggeman, F.J. CHAPTER 6 How fast-growing bacteria robustly tune their ribosome concentration to approximate growth rate maximisation. Manuscr. Accept. Publ. FEBS J., 119–153.
%        \cite{BosdrieszE}
%        \item Maeda, M., Shimada, T., and Ishihama, A. (2015). Strength and Regulation of Seven rRNA Promoters in Escherichia coli. PLoS One 10, 1–19. \cite{Maeda2015}
%        \item Gyorfy, Z., Draskovits, G., Vernyik, V., Blattner, F.F., Gaal, T., and Posfai, G. (2015). Engineered ribosomal RNA operon copy-number variants of E. coli reveal the evolutionary trade-offs shaping rRNA operon number. Nucleic Acids Res. 43, 1783–1794. \cite{Gyorfy2015}
%        \item \textit{(The following two articles talk about the ribosomal RNA deletion strains, also surprising interactions with antibiotics. Of main importance is the observation that mainly tRNA seems to be limiting. See also unpublished \cite{Quan2013}.)} Bollenbach, T., Quan, S., Chait, R., and Kishony, R. (2009). Nonoptimal Microbial Response to Antibiotics Underlies Suppressive Drug Interactions. Cell 139, 707–718. \cite{Bollenbach2009} AND 1. Quan, S., Skovgaard, O., McLaughlin, R.E., Buurman, E.T., and Squires, C.L. (2015). Markerless Escherichia coli rrn Deletion Strains for Genetic Determination of Ribosomal Binding Sites. G3 Genes|Genomes|Genetics 5, 2555–2557. \cite{Quan2015} \textit{Quan et al. also refer to Orelle et al. 2013; Polikanov et al. 2014b; Orelle et al. 2015.}
%        \item \textit{(This article details the rRNA regulation.)} Schneider, D.A., Ross, W., and Gourse, R.L. (2003). Control of rRNA expression in Escherichia coli. Curr. Opin. Microbiol. 6, 151–156. \cite{Schneider2003} 
%        \item \textit{(These two article with description of the structures generated from rRNA genes, also mention the cleavage of the RNA genes.)}  Kaczanowska, M., and Ryden-Aulin, M. (2007). Ribosome Biogenesis and the Translation Process in Escherichia coli. Microbiol. Mol. Biol. Rev. 71, 477–494. \cite{Kaczanowska2007} AND Shajani, Z., Sykes, M.T., and Williamson, J.R. (2011). Assembly of bacterial ribosomes. Annu. Rev. Biochem. 80, 501–26. \cite{Shajani2011}
%        NOTE that \cite{Chen2013} has also a picture of biogenesis and assembly.
%        \item \textit{(Following article measures burden on ribosome and effect on growth in a dynamic environment.)} Shachrai, I., Zaslaver, A., Alon, U., and Dekel, E. (2010). Cost of Unneeded Proteins in E. coli Is Reduced after Several Generations in Exponential Growth. Mol. Cell 38, 758–767.        
%        \cite{Shachrai2010}
%        \item Maaloe1979 and Scott2010 (see Hui2015) which claims that ribosomes operate at "saturated translational capacity".
%    \end{itemize} 
%    
%\clearpage    
%}
















