\chapter{Introduction}
\label{chapter:introduction}


%****************************************************************************
%\begin{abstract}
%no abstract for introductionchapter
%\end{abstract}

%% Start the actual chapter on a new page.
%\newpage % no newpage for introductionchapter


%****************************************************************************
%****************************************************************************

% Who should understand this? Rob, Karin, my mother to some extend...

%\section{A quantitative view on biology}
\section{The power of numbers}

%Numbers are important.
%
Describing the world around us with numbers is not just an interesting hobby for bookkeepers.
%
Quantitative descriptions of phenomena can give insights that otherwise would have been out of reach.
%
%A quantitative approach can also be very valuable to understand
%This also applies to biological processes \cite{Lazebnik2003}.
%
% Writing down mathematical descriptions 
% Despite some canonical examples, 
%
There are some canonical examples in the field of biology.
%
Like the Lotka-Volterra model from the early 1900s, which explains fluctuations in predator and pray numbers using differential equations \cite{Lotka1920,Volterra1928}.
Or the mathematical reaction-diffusion models created by Turing, which explain how large spatial inhomogeneities can spontaneously arise from molecules 
that interact by simple rules \cite{Turing1952}.
%with simple interaction rules \cite{Turing1952}.
Such principles are for example hugely important during embryogenesis, when cells need to decide which part of the body they will develop into.
%
However, perhaps partially because of reluctance by biology researchers \cite{Lazebnik2003} and partially because a lack of proper quantitative measuring tools \cite{Kitano2002, Wollman2018}, such quantitative descriptions % of biological systems 
were not commonplace in biology.
%quite rare.
%
But since the turn of the millennium, 
quantitative biology has taken a huge flight again.
%many insights in biology are acquired by taking advantage of quantitative methods.
%
I hope the work in this thesis also offers exciting examples of new insights that are acquired with this new wave of quantitative measurements.

\section{Prying in the lives of single cells}

%
%In the Tans lab, we use a microscope and a computer to make time lapse movies of living and growing bacteria.
%
%Until recently, bacterial researchers were not able perform experiments on a specimen of their organism of interest.
%Until recently
Before the onset of techniques that allowed high throughput quantitative measurements, bacterial researchers usually did not perform studies on individual specimens of their organism of interest.
%
Experiments 
%to learn about bacterial behaviour 
were conducted in test tubes on millions of individual cells at once.
%
While this gives great information about the average behaviour of bacteria, 
which filled text-books with the detailed workings of bacteria,
it does not allow one to learn everything about how a bacterium works.
%
% An example of this is given in chapter \ref{chapter:filarecovery}.
%
New techniques now allow us to better probe the life of single cells.
%
Where a century ago people had to observe tiny bacterial colonies by eye through the microscope, painstakingly draw them, and quantify division times manually \cite{Kelly1932},
we can now do the same using a computer and 
% our computerized analysis of the data allows us to 
track the lives of thousands of individual cells meticulously in a few days work, as described in chapter \cite{chapter:methods} of this thesis.
%
This can give new insights, as illustrated by chapter \ref{chapter:filarecovery}.
%
During single cell experiments in the Tans lab that probed the effect of antibiotics on \textit{Escherichia coli} bacteria \cite{RozendaalVerslagXXX} we noticed a peculiarity:
when \ecoli with elongated morphologies due to stressful conditions (referred to as filamanted bacteria) divide, 
they always divided at very defined relative positions.
%
Before this observations, it was thought that filamentous bacteria behave as if they were a string of multiple bacteria merged together.
%
But our measurement showed that instead of following rules according to that principle, 
bacteria followed another set of rules.
%
This required an unexpected and not previously observed flexibility in the placement of their potential division sites.
It also revealed that the well-known Min division regulation system has a previously unrecognised functionality when cells are in their filamentous state.
%
Aside from these two novel observations, 
the results also implied that bacterial cells have advanced mechanisms to control their morphology far beyond often observed bacillary form (the short rod-like shape).
%
Conditions that induce these forms are not often studied in the lab, 
but are very relevant in daily life where bacteria might survive antibiotic treatments or cold conditions (fridges) by adopting this filamentous form.
%for example proliferation of undesired food-borne or pathogenic bacterial is combated exactly by exposing to bacteria to harsh conditions like heat, cold, or antibiotics.
%
Concluding, chapter \ref{chapter:filarecovery} illustrates both the importance of investigating individual bacteria and the importance of investigating other conditions other than standard lab growth conditions. 

\section{Single cells are different} 

But observing bacteria does not only give insights about behaviour that is (to an extent) identical in all bacteria.
%
As observed in a pioneering study by Spudich and Koshland \cite{Spudich1976} single bacteria show different food-searching behaviour despite being genetically identical.
%
The role of stochasticity in gene expression was further 

\section{bla}

TWO CONCEPTS COME TOGETHER

the filamentous bacteria re-arranged their 
%
We showed that this is due to a novel functionality of the well-known MinC, MinD and MinE proteins.
These three proteins form a reaction diffusion system that can create patterns employing the aforementioned Turing mechanism, 
which was known to help non-filamentous cells divide at half their length. 
%

%
Since this kind of single cell division data was not available before, the specific division pattern was not measured before.
OPENS UP NEW INSIGHTS / LARGE IMPLICATIONS
%
We further used 



The division of single bacterial cells could not be tracked before 



\section{blabla}

Experiments were conducted by looking at test tubes with hundreds of millions of cells\footnote{} 

now
what used to take ages
can be done in a few days


Specifically, we are interested in the lives of single cells.
%

%
%


In the Tans lab, we work with single cells, one of the recent advances 
%
and noticed something peculiar when observing .

\section{Individuality in single cells}



\section*{stuff}
But recently, also more and more research into biology takes advantage of quantitative 


The advance of new techniques in biology has 


There are some canonical examples in the field of biology.
%
Like the Lotka-Volterra model from the early 1900s, which explains fluctuations in predator and pray numbers using differential equations \cite{Lotka1920,Volterra1928}.
Or the reaction-diffusion models described by Turing, which explain how large spatial inhomogeneities can spontaneously arise from molecules with simple interaction rules \cite{Turing1952}.
%
Nevertheless, the 

Lazebnik2003


This is also true in the domain of biology.
%
Phenomena that include the 


For example, during the development of an embryo, the multiplying clump of cells need to decide which cells eventually become the head, which cells the end of a toe, etcetera.
%
The precise concentration of certain regulatory factors (morphogens) helps to decide the eventual fate of cells.
%
Mathematical descriptions are extremely useful in this context.
%
For example, it was recently shown that 

%\begin{figure}
%    \begin{minipage}[c]{0.7\textwidth}
%        %\centering    
%        %\includegraphics[width=1.0\textwidth]{pdf_2016-02-17_pos2_L31-mCerulean_clouds.pdf}
%        \includegraphics[width=0.99\textwidth]{lizard_Manukyan2017.png}
%    \end{minipage}\hfill
%    \begin{minipage}[c]{0.3\textwidth}
%        \caption{ 
%            \textbf{Top view of an ocellated lizard.}
%            The pattern on the skin of the lizard is an example of a process that can be understood by writing down a mathematical description of the process.
%            Image taken from \cite{Manukyan2017}.
%            %
%            %        
%        }
%        \label{fig:ribo:switch1}
%    \end{minipage}
%\end{figure}




\subsection*{stuff}

 the development of an embryo is governed by interaction between molecules where 


Classically, biology has always been a very qualitative science.
%
Processes were described in terms 
Focus lay on identifying which factors (which protein or gene, for instance) were involved in certain phenotypes, and whether interaction existed between factors.
%
DNA replication can be 




%Biology has long been a crude science \cite{Lazebnik2003}.
%%
%To understand cellular decision networks, it was deemed sufficient to find out which protein interacted with which other protein or DNA sequence, or what the effect of a protein knockout was.
%%
%In recent years, the emerging field of systems biology, or quantitative biology, has introduced a new way of thinking.
%%In recent years, the emerging field of systems biology has provided us with many insights by \red{try thinking of some examples}
%%
%By careful quantitative analysis of the cell, and mathematical analyses that might only describe the interactions between a few cellular components, albeit in a minute way, 
%one could fundamentally understand and predict decision networks in the cell \cite{Alon2006}.
%
%In this thesis, I will present examples of such ....
%
%In the first chapter of this thesis however, we discuss ...
%
%A good example of a system is the Min system.




\section*{Notes}

Mention Kelly et al 1932 \cite{Kelly1932}?!

\section*{more stuff}


Wollman2018 (review)
Karlebach2008,Ideker2012,DiVentura2006 (reviews) \cite{Karlebach2008,Ideker2012,DiVentura2006}.
Davidson2008 (individuality in bacteria) \cite{Davidson2008}.



%\section{Martijn's great introduction, first section}
%
%%%%%%%%%%%%% Figure Intro Cell-to-cell Variability
%%\begin{figure}[h]
%%	\includegraphics[width=\textwidth]{XXX}
%%	\caption{\label{fig:ch1examplesvariability} \textbf{Examples of cell-to-cell variability.} \figA Genetically identical \ecoli bacteria produce different amounts of proteins (upper panel) and the protein concentration in single cells fluctuates over time (lower graph). \figB Clonal \textit{B. subtilis} bacteria can differentiate into different cell types. The differentiation is driven by stochastic protein production. Figure \figA was taken from \cite{Taniguchi2010}, \figB was taken from \cite{Suel2006}.}
%%\end{figure}
%%%%%%%%%%%%%
%
%Even in a single species of enzymes, the catalytic rate can vary from one enzyme to the next \cite{Lu1998}. (Interestingly, the average waiting time for a reaction has the same form as the Michaelis Menten equation \cite{Xie2013}.)
%
%Transcription is a bursty process \cite{Golding2005}. 
%
%Global trends in noisy protein expression have been investigated. 
%Noise scales with the amount of protein that is expressed \cite{Bar-Even2006}.
%
%\section{Random notes}
%
%In eukaryotes, it is feasible to detect transcript from the RNA \cite{Levsky2002}.


%****************************************************************************
%****************************************************************************
% avoid floats to appear after the footnote.
\FloatBarrier

%\blfootnote{write a footnote?}

