

\chapter{Size laws and septal ring dynamics in filamentous Escherichia coli cells}
\label{chapter:filarecovery}

\blfootnote{The contents of this chapter have been published as Wehrens, M., Ershov, D., et al., 2018. \textit{Size Laws and Division Ring Dynamics in Filamentous Escherichia coli cells.} Current Biology, pp.1–8. Available at: https://doi.org/10.1016/j.cub.2018.02.006. \cite{Wehrens2018a}}

\begin{figure}[t]
    \centering
    \includegraphics[width=0.8\textwidth]{Graphical_Abstract.pdf} %   width=13.97cm
    \caption*{ 
        \centering        
        \textbf{Graphical abstract.}
    }
    \label{fig:filarecovery:graphicalabstract}
\end{figure}

\section*{In brief}
Wehrens, Ershov, et al. report a new size control mechanism in E. coli cells, which have elongated due to stress. Multiple division rings are continuously rearranged in response to growth and division to control daughter cell size when divisions resume. Divisions are spatially controlled by the Min system and temporally by the adder principle.

\section*{Highlights}
\begin{itemize}[itemsep=1pt,parsep=1pt]
    \item Long filamentous E. coli cells continuously assess and adjust division ring positions.
    \item Sudden division site suppression and formation is explained by the Min system.
    \item Division timing is consistent with the adder principle.
    \item These rules enable controlled entry in and exit from the filamentous state.
\end{itemize}

\clearpage

\section*{Summary}
\textit{
Our understanding of bacterial cell size control is based mainly on stress-free growth conditions in the laboratory \cite{Marshall2012, Jorgensen2004, Robert2015, Chien2012, Turner2012, Wallden2016, Taheri-Araghi2014, Campos2014, Amir2014, Osella2014}. 
In the real-world however, bacteria are routinely faced with stresses that produce long filamentous cell morphologies \cite{Suzuki1967, Rolinson1980, Miller2004, Domadia2007, Justice2006, Moller2012, Jones2004, Linn1987, Kawarai2004, Wainwright1999, Rosenberg1967, Adler1965, Kantor1966, Radman1975, Michel2005, Jones2013, Justice2008, Pulvertaft1952}. 
\ecoli is observed to filament in response to DNA damage \cite{Adler1965, Kantor1966, Radman1975, Michel2005}, antibiotic treatment \cite{Suzuki1967, Rolinson1980, Miller2004, Domadia2007, Pulvertaft1952}, host immune systems \cite{Justice2006, Moller2012}, temperature \cite{Jones2004}, starvation \cite{Wainwright1999}, and more \cite{Kawarai2004, Linn1987, Rosenberg1967}; conditions which are relevant to clinical settings and food preservation \cite{Jones2013}. 
This shape plasticity is considered a survival strategy \cite{Justice2008}. Size control in this regime remains largely unexplored. Here we report that \ecoli cells use a dynamic size ruler to determine division locations combined with an adder-like mechanism to trigger divisions. As filamentous cells increase in size due to growth, or decrease in size due to divisions, its multiple Fts division rings abruptly reorganize to remain one characteristic cell length away from the cell pole, and two such length units away from each other. These rules can be explained by spatio-temporal oscillations of Min proteins. Upon removal of filamentation stress, the cells undergo a sequence of division events, randomly at one of the possible division sites, on average after the time required to grow one characteristic cell size. These results indicate that \ecoli cells continuously keep track of absolute length to control size, suggest a wider relevance for the adder principle beyond the control of normally sized cells, and provide a new perspective on the function of the Fts and Min systems.
}



\section{Results}
\subsection{Division site selection rules in filamentous cells}
To investigate divisions in filamented Escherichia coli cells we used a microfluidic device that allows media exchange \cite{Boulineau2013} (Fig. S1A-C). We first grew the cells for 2-3 generations at 37°C in minimal medium within the device, and then induced filamentation in one of three ways: exposure to tetracycline (TET), a temperature increase to 42°C, or overexpression of SulA. While the molecular basis in the former cases is unclear, translation inhibition antibiotics have been reported to induce filamentation \cite{Pulvertaft1952}, and SulA is a division inhibitory protein \cite{Dajkovic2008}. As a result the cells grew to approximately 10-20 times the typical length without dividing (Fig. 1A), while their width remained approximately constant (Fig S1D). Division resumed when the stressor was removed and ultimately returned to normal stress-free growth and division, unless the stress was too severe and the cells failed to recover (Fig. S1E-G). The relative location of division events throughout this recovery process was characterized by S = Ld/Lm, where Ld and Lm are the daughter and mother cell lengths respectively (Fig. 1B). 

We found that S displayed a specific pattern when plotted against Lm (Fig. 1D and S1H-K), which was for instance not the case when plotted against the time elapsed since stress removal (Fig. 1E and S1L). The same S-Lm pattern was observed for the different levels and types of stress (Fig. S1H-K). Several features were as expected. First, normally sized cells of a few microns long divided in the middle, with S = 1/2. Second, for increasing Lm, other division sites appear with S-values that remain constant within a certain length-window (Fig. 1D). These features are consistent with a longstanding model of division site locations (see Fig. 1C), which was supported by early time-lapse microscopy experiments in the 70s \cite{Donachie1970}, detection of invaginations in cell division mutants \cite{Taschner1988}, and consistent with observed roughly equidistant division rings at multiple locations \cite{Addinall1997, Arjes2014, Mileykovskaya1998} and uniform cell wall growth \cite{Typas2012}. In this model, during filamentous growth, the distance between division sites increases progressively, until this distance doubles and new sites emerge in between existing sites. Note that this scenario can be compared to normally dividing cells arranged along a line, as new division sites would then also emerge in between two previous division sites.

At the same time, other features differed substantially. Fig. 1C suggests moderately filamented cells divide at S = 1/4,  1/2, and 3/4. However, for cells with S = 1/4 and S = 3/4, the site at S = 1/2 was repressed in most of that length-window (Fig. 1D, green bars). Other sites appeared suppressed as well. For instance, the data did not show cells with two rings at S = 1/3 and 2/3, or with other odd fractions such as multiples of 1/5 (Fig. 1D). We did not detect a significant spatial preference: the different possible sites within the length window 15-21μm displayed a similar probability to divide (Fig. 1F). On the other hand, the observed sites could be captured in specific rules (Fig. 1G). The distance between two possible division sites equaled two normally-sized cells (rather than one as in Fig. 1C), though the distance between a pole and the nearest division site was one normally-sized cell. More generally, the different possible sites within one length-window was described by S = (2m-1)/2n, with m indexing the sites from 1 to the total n = round(Lm/2w), and w = 3 μm the average length of unstressed cells. These rules also indicate discontinuous changes. For instance the S values of the green bars (1/4 and 3/4) disappear in the subsequent length regime with the blue bars (1/6, 3/6, and 5/6), and so on (Fig. 1D). Thus, potential division sites appear not to be conserved upon changes in length, unlike the model indicated in figure 1C. To further probe these various unexpected findings, we analyzed the recovery process in time. 

\subsection{Time and length changes between divisions}
To study the recovery process in time, we quantified cellular lengths along lineages after stress removal. The resulting traces displayed sudden drops denoting division events, as the long cells progressively converted into normally sized cells (Fig. 2A and S2A,D). Notably, we rarely observed multiple divisions occurring at the same time within one cell – after a single division event some time elapsed (for cells over 10 μm long recovering from tetracycline exposure, 89\% of the interdivision times were 10 min. or more) before the next division took place.  Thus, in cells that were long enough to fit multiple possible division sites, division occurred in just one site at a time.

The interdivision time was also found to decrease with increasing cell length; cells that were born longer divided faster (Fig. 2B). However, the interdivision time appeared to level off at about 10 min, consistent with divisions in a minCDE null strain and cells having a limited ‘division potential’ \cite{Donachie1996}. This dependence on cell length was similar for the different filamentation triggers (Fig. 2C and S2B,E,G). The tetracycline and temperature data differed significantly in one bin only (two sample t-test, p = 0.05), while the SulA data set showed a somewhat smaller interdivision time. The latter may reflect the more downstream role and limited growth-rate effect of SulA. Interestingly, the length added between two divisions appeared constant and independent of birth size (Fig. 2D, S2C,F). This added length was similar to the length of normal newborn cells (1.6 vs. 1.8 microns respectively, Fig. S2H). Thus, strikingly, long and short new-born cells grow a similar absolute amount until the next division, while the shorter cell takes more time to produce that added length. The latter is consistent with larger cells producing cell mass faster because they contain more biosynthetic machinery (Fig. S2I). Consistently, we find that the average area under the length versus time curve is proportional to the added length divided by the growth rate (Figure S2J). 

A constant added volume or length between divisions has been reported for normal stress-free growth, when cells are not filamentous and divide mid-cell \cite{Amir2014, Campos2014, Taheri-Araghi2014, Wallden2016}. This "adder" principle can explain how cells in these conditions maintain a constant average size despite stochastic variability in birth size \cite{Amir2014, Campos2014, Taheri-Araghi2014}. Here, the adder principle does not strictly govern size, as birth sizes of filamentous cells are affected more strongly by the position of division sites. The findings suggest that the time between divisions is coupled to the growth process. The added length in stress-free cells has been reported to be proportional to the number of origins of replication \cite{Ho2015, Zheng2016}. We found nuclei to continue to multiply during filamentation in accordance with length increases, as observed by fluorescent labeling (Fig. S2K), which suggests that the origins continue to increase as well. Note that the unstressed interdivision time here is about 60 mins (Fig. 2C), which is associated with non-nested replication. Thus, while the number of nuclei differs several fold between normal and filamented cells, the added size between divisions remains approximately the same, suggesting that the number of origins does not set the added size in this stressed regime. We find that the cell volume to nucleoid ratio was approximately constant for cells of different length within the filamentous regime (Fig. S2L). The ratio of cell volume to the number of replication origins was studied previously for non-filamentous cells \cite{Si2017, Cooper1968}.

\subsection{Dynamic reorganization of division rings}
The data revealed that the reductive divisions during recovery occur in concert with substantial cellular growth (Fig. 2A and S2M). To illuminate how continued growth affected division site positions, we imaged the division machinery by fluorescently labeling the essential cell division protein FtsA with sYFP2 \cite{Erickson2010, Lutkenhaus2012}. As expected, we observed a number of bands of fluorescence intensity along the cell axis indicating division rings \cite{Addinall1997, Arjes2014, Mileykovskaya1998}  (Fig. 3A). When tracked in time however, these division rings displayed an unexpectedly dynamic behavior, both during growth of the filament and upon division events. For instance, upon a division, a new ring appeared at a new location (Fig. 3B, top arrow), within the smaller daughter. At the same time, one of the other rings that had been present in the mother cell, disappeared in the other daughter at the same moment (Fig. 3B, bottom arrow). Indeed, most division events were immediately followed by the rapid reorganization of division rings (Fig. 3B). Independently of division events, growing filamentous cells also showed sudden reorganization of ring positions (Fig. 3C, S3). For instance, all four rings in one cell abruptly disappeared at the same time, while five rings appear in the next frame at different positions (Fig. 3C, grey arrow).

Once formed, division rings often remained fixed at the same position for over 50 mins (Fig. 3B). The sudden reorganization dynamics of these mature division rings indicated that such positional memory can be erased rapidly. In early stages of ring formation, FtsZ clusters have been reported to display positional dynamics in non-filamentous cells \cite{Tsukanov2011}. The observed ring dynamics also explained the observed ‘suppression’ of division events at certain sites: it for instance allows a central ring at S = 1/2 in a normally sized cell to disassemble when filamentous growth causes entry into the second length window with rings at S = 1/4 and 3/4. Similarly, it explains when rings are added. Cells do not add multiple rings when the distances between existing rings has doubled, but rather add a single ring when the total cell length has become 2w longer, and correspondingly the entire pattern of rings changes. Indeed, more generally, the observed division ring positions (Fig. 3D) followed the same spatial rules as the division event positions (Fig. 1D,G). Note that one does not observe division rings at all the sites denoted by the spatial rules (Fig. 1D,G), such as the middle region of the cell depicted in Fig. 3B. Furthermore, as expected, we did not see division events unless a fluorescent band was observed first (see Fig. 3B). In cells with multiple rings, division typically occurred at one site at a time (Fig. 3B, see also Fig. 2A-B). The fluorescence intensity of the bands did not correlate significantly with division probability (Fig. S3B). 

Thus, the dynamics of the division rings is important to obeying the spatial division rules. Growth and division events can ‘move’ lineages from one length-window to another, and hence require changes in the pattern of possible division sites. Note that for cellular growth within a length window, rings do not change their relative positions. Thus, the cells continuously assess the locations of their division machinery, and repositions the potential division locations, resulting in daughter cells of specific size. 

\subsection{Min oscillations can explain division site rules}
The spatial position rules indicate absolute bounds for the distance between putative sites. Specifically, this distance is on average between 1.5 and 2.5w, with w being approximately 3μm. These observations indicate a mechanism able to measure absolute lengths. We surmised that this function is served in \ecoli by proteins encoded at the min locus: MinC, MinD, and MinE. MinD can bind to the cell membrane, and the MinD and MinE reaction-diffusion system results in pole to pole oscillations of MinD along the cell membrane. In non-filamentous cells, the time-averaged concentration profile of MinD shows a minimum value localized at S = 1/2. Because MinC binds MinD, but also inhibits ring formation, non-filamentous cells form division rings mid-cell \cite{Loose2011, Raskin1999}. 

To test whether MinD oscillations could contribute to the division rules in filamentous cells, we extended the reaction-diffusion model of Meinhardt and De Boer \cite{Meinhardt2001} (Materials and Methods). Strikingly, the simulations recapitulated all key features of the spatial position rules (Figs 1D,G and 4A): distinct length windows of equal size, discontinuous changes in all minima positions between adjacent windows, and absolute bounds for the distance between minima. A similar MinD pattern was obtained using model of Huang et al. \cite{Huang2003} as implemented by Fange et al. \cite{Hattne2005, Fange2006} (Fig. S4A). To obtain further support for a role of the Min system in the position rules, we used a YFP-MinD fusion to assess the MinCD distribution. The resulting time-averaged fluorescence profiles validated the simulation results (Fig. 4B). 

Next, we characterized filamentation and recovery in minCDE null cells. Divisions no longer obeyed the division rules (Fig. 4C). Division sites were more uniformly distributed, and for instance no longer peaked at S = 1/4 and 3/4, nor showed the previously observed suppression at S =1/2 or other S-values. The positions were not completely arbitrary, with S=1/2 remaining frequent, suggesting the involvement of other mechanisms such as nucleoid occlusion. The interdivision time was similar as observed for wild-type cells, and consistent with earlier observations \cite{Donachie1996} (Fig. S2N-P). While the length added between divisions remained independent of cell length, it was now more broadly distributed (Fig. S2O). Moreover, the recovery produced numerous non-viable mini-cells \cite{Donachie1996, Adler1967} that were arrested in their cell cycle and stopped growing.  These findings further support the notion that the Min system is central to the spatial division rules in filamentous cells.

\section{Conclusions}
Cell size control of \ecoli is actively studied in continuous laboratory cultures, where cells grow in the bacillary form (see e.g. \cite{Marshall2012, Jorgensen2004, Robert2015, Chien2012, Turner2012, Wallden2016, Taheri-Araghi2014, Campos2014, Osella2014}). Here, size-control typically concerns maintaining a constant cell size in the presence of stochastic variability, for instance in birth size and growth rate. \ecoli is also known to adopt the longer filamentous form in response to diverse stressors. Here we surmised that size regulation mechanisms could also be relevant to enter into, maintain, and exit from filamentous states. We find that within growing filamentous cells, multiple non-constricting division rings remain at specific fixed relative positions when the cells remain within a specific length range, while all the rings abruptly change in number and position when exceeding this range, which can be caused by growth and division events (Fig. 4D). These spatial rules are explained by the Min system, which is found to produce a pattern that is strikingly similar to sound standing waves, with minima at matching positions that change discontinuously when an additional minimum fits along the cell length. Hence, the Min system can be thought of as a ruler mechanism that measures absolute size, with division rings as tick marks that have a spacing corresponding to not one but two unstressed cells. Upon disappearance of the stressor, divisions occur in sequence, at just one of these multiple putative division sites at a time, with the interdivision time decreasing with cell length. The length added in between divisions is rather invariant with cell length, in a manner that is reminiscent of the on average constant added length for non-filamentous cells that experience small variations in birth size during normal growth, suggesting that this adder principle is more broadly relevant beyond normal growth. How these long cells divide just once and then add a fixed length remains an intriguing question. One may consider additional suppression of division at certain sites by nucleoid occlusion mechanisms \cite{Wu2011, Mulder1989}, though this would not naturally explain the suppression of division for a certain amount of growth. Another possibility is a limiting septal protein that must increase in number after division \cite{Bi1990}, though the multiple chromosomes and FtzA rings were observed. Overall, it is puzzling how cells that strongly vary in size, would be limited similarly to just one division. Taken together, the findings reveal a system of size sensing and division control in filamentous \ecoli cells, and bring a new perspective to the functional role of Min and Fts dynamics.

\section*{Acknowledgements}
Work in the group of S.J.T. is supported by the Netherlands Organization for Scientific Research (NWO). P.A.L. is funded in part by  National Institutes of Health grant GM64671 and a grant from the Fulbright U.S. Scholar Program. R.K is funded by the European Research Council FP7 ERC grant 281891 and the NIH Grant GM081617. We thank Svetlana Alexeeva and Tanneke den Blaauwen, Alexander Dajkovic, Cees Dekker and Suckjoon Jun for kindly sharing their plasmid constructs, Daan J. Kiviet for kindly providing a microfluidic device mold, and Nick de Lange for performing the sulA recovery experiment. 

%Author contributions
%Conceptualization, MW, DE, RR, NW, PAL, SJT; Software, MW, DE, RR, NW; Investigation, MW, DE, RR, NW; Writing  Original Draft, MW, PAL, SJT; Writing, Review and Editing, MW, PAL, SJT; Supervision, DS, RK, PAL, SJT.

%Declaration of Interests
%The authors declare no competing interests.

\begin{figure}
    \centering
    \includegraphics[width=1.0\textwidth]{Figure1.pdf}
    \caption{ 
        \textbf{Division site rules in filamentous E. coli.} 
%        Figure 1. 
(A) Colony of filamentous cells. (B) Relative division site S, daughter length Ld, and mother length Lm. (C) Possible division sites in classical scenario, with a minimal equidistant spacing, and hence more sites for longer cells. (D) Division sites (see panel B) observed in filamentous cells when switching from stress medium (1 μM tetracycline) to non-stress medium. Each point corresponds to one division (N = 4108). Data is symmetric around S = 1/2. Colored lines denote inferred approximate division location (see panel G). Dashed lines correspond to divisions producing daughter cells of 2 and 4 μm. (E) As panel (D) but as a function of the time of the division event. (F) The distribution of relative division locations for cells with a maternal size between 15-21μm, suggesting a lack of site preference. Dashed black lines indicate bin boundaries. (G) Division site rules inferred from the data in panel D. n is the number of possible division sites, m indexes the possible sites in one cell, w is a characteristic length (about 3μm). See also Figure S1.        
    }
    \label{fig:filarecovery:fig1}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1.0\textwidth]{Figure2.pdf}
    \caption{ 
         \textbf{Time and length changes between divisions.}
         (A) Measured cell lengths over time after switching from stress medium (1 μM tetracycline) to non-stress medium. Grey traces correspond to single cell lengths. Colored lines are lineages following the longest daughter, black line the shortest daughter. Black squares indicate end of measurement. Data is from five 1μM tetracycline recovery experiments (N = 4134 cells). (B) Interdivision time against mother birth size, for recovery from 1μM tetracycline. Black dots are averages, bars are standard deviations (N = 4108 division events). (C) Average interdivision time against mother birth size, for recovery from 1μM tetracycline (blue), 42°C heat shock (red), and overexpression of the division-inhibitor SulA (green) (N=4108, N=404, and  N=494 division events, respectively). Error bars show SEM. (D) Absolute length added between two divisions against mother birth size. Black dots are averages (N = 4108 division events.), error bars show SEM. See also Figure S2.
    }
    \label{fig:filarecovery:fig2}
\end{figure}
 
\begin{figure}
    \centering
    \includegraphics[width=1.0\textwidth]{Figure3.pdf}
    \caption{ 
        \textbf{Dynamic reorganization of division rings.}           
         (A) Fluorescence image of sYFP2-FtsA signal in cells filamented by heat shock at 42 °C. FtsA is an essential component of the division apparatus. Blue lines indicate cell edge determined from phase contrast imaging. (B) Kymograph of sYFP2-FtsA intensity profiles along the long cellular axis, for cells recovering from 1μM tetracycline. Red indicates division events, blue the resulting separation between daughters. See main text for arrow. (C) Similar kymograph for filamentous cells during heat shock at 42°C. (D) Relative locations of sYFP2-FtsA peaks along the cellular axis, during heat shock at 42°C and subsequent growth at 37°C. Colored bars correspond to inferred division site rules (Fig. 1D-E). Peaks were identified using a Matlab peak finder algorithm (N = 1572 cell images, taken from 400 cell cycles). See also Figure S3.        
    }
    \label{fig:filarecovery:fig3}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1.0\textwidth]{Figure4.pdf}
    \caption{ 
        \textbf{Min oscillations can explain division site rules.}           
         (A) Computed time-averaged MinD concentration profile along the longitudinal cellular axis for cells of different lengths, using the Meinhardt and De Boer \cite{Meinhardt2001} model. Green corresponds to high MinD concentrations. Grey dots are observed divisions (Fig. 1D). Simulation results are scaled linearly along x-axis for best correspondence with the experimental data. (B) Experimentally obtained time-averaged intensity profiles of YFP-MinD fusion proteins for cells of different sizes. Top to bottom indicated 1, 3 and 5 minima. Low values at cell poles are artefacts due decreasing cellular width. (C) Division sites in minCDE null strains filamented by 2 μM tetracycline. Each point corresponds to one division (N = 1260). Data is symmetric around S = 1/2. Colored dotted lines denote division rules in wild type cells (Fig. 1D). Minicells were observed (inset), but were not included in the analysis (grey shaded region).  (D) Cartoon illustrating the observed division site plasticity. Changes in cell length, due to continued cell growth (top) or division events (bottom) produce discontinuous changes in the MinD profile, and corresponding reorganization of the pattern of division rings, and ultimately locations of division events. See also Figure S4, Table S1 and S2.        
    }
    \label{fig:filarecovery:fig4}
\end{figure}




\section{STAR Methods}

\subsection{CONTACT FOR REAGENT AND RESOURCE SHARING}
Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Sander Tans (Tans@amolf.nl).

\subsection{EXPERIMENTAL MODEL AND SUBJECT DETAILS}
For tetracycline stress experiments, wild type strain MG1655 (rph-1 ilvG- rfb-50) was used (ASC555, Key Resource Table). For experiments involving SulA we used wild type strain W3110 (λ- IN(rrnD-rrnE)1 rph-1) with pACT3 plasmid containing Plac-sulA, a kind gift of Alexander Dajkovic \cite{Dajkovic2008} (ASC884, Key Resource Table). For experiments involving temperature recovery and sYFP2-FtsA, we used wild type strain MG1655  with plasmid pSA018 containing PTRCdown-sYFP2-ftsA (ASC777, Key Resource Table), a kind gift from Svetlana Alexeeva and Tanneke den Blaauwen (University of Amsterdam). For experiments involving MinD dynamics, we used strain MG1655 with plasmid pFX40, containing Plac::yfp-minD minE (AmpR) (ASC784, Key Resource Table), a kind gift from the Cees Dekker Lab (Delft University of Technology). For experiments with minCDE null mutants we used strain MG1655 with the minCDE gene deleted  (ASC1035, Key Resource Table), a kind gift from Piet de Boer \cite{DeBoer1989}. For experiments with labeled nucleoids \cite{Wery2001}, we used an MG1655 strain with  hupA::[hupA::mCherry FRT kan] (ASC1106, Key Resource Table), a kind gift from the Suckjoon Jun lab (University California, San Diego).
All cell lines were stored at -80°C in freeze mix stocks. Before experiments, they were inoculated in TY medium and grown at 37°C into exponential phase, then transferred to culture tubes with M9 media, grown overnight to reach exponential phase next day, and transferred to microfluidic setup or gel pad the next morning where they were supplied with M9 minimal medium. The microfluidic setup or gel pad was then imaged under the microscope. Freeze mix contained 0,7\% g/ML Peptone (Bacto™, BD Biosciences) and 24\% V/V glycerol (Merck) dissolved in sterile dH2O. TY medium contained 1\% gr/mL Tryptone (Bacto™, BD Biosciences), 0,5\% gr/mL Yeast Extract (Bacto™, BD Biosciences) and 0,5\% gr/mL NaCl (Merck) dissolved in sterile dH2O. M9 minimal medium contained 47.7 mM Na2HPO4, 25 mM KH2PO4, 9.3 mM NaCl, 17.1 mM NH4Cl, 2.0 mM MgSO4, 0.1 mM CaCl2; all the chemicals were provided by Merck. M9 medium was supplemented with 0.2 mM uracil and 0.1\% g/mL lactose (tetracycline, minCDE null mutant and labeled nucleoids experiments) or 0.1\% g/mL maltose (temperature, SulA and MinD experiments); all provided by Merck. Tetracycline stock solutions (1mM in ethanol) were made from tetracycline powder (Merck) and stored at -20°C for not more than 4 weeks, final concentrations were 1μM, 2μM and 10μM for tetracycline experiments and 2μM for minCDE null mutants and labeled nucleoids experiments. IPTG (Merck) stock solutions (1mM in water) were stored at -20°C.

\subsection{METHOD DETAILS}

\subsubsection{Experiments with microfluidic device 1}
For experiments with wild type cells (ASC555) exposed to and recovering from tetracycline, microfluidic device 1 was used, see also Figure S1B. This device \cite{Boulineau2013} consisted of cover slip, a polyacrylamide gel membrane (thickness, 500 μm) and a polydimethylsiloxane (PDMS) flow cell whose channel (3 cm * 3 mm * 91 μm) contained evenly spaced square pillars (400 μm, spaced by 600 μm) to ensure a uniform pressure on the membrane. 
The polyacrylamide gel membrane was formed by mixing 1.25 mL 40\% acrylamide (Bio-Rad), 3.7 mL deionized sterile water, 50 μL 10\% ammonium persulfate (Sigma) and 5 μL TEMED (Bio-Rad). 450 μL of the mixture was poured in a mold and the solution was left to polymerize for about 1.5 h. 
After polymerization, the gel was cut in a piece of 4 x 1.5 cm and stored in a flask with sterile water. The master PDMS mold consisted of one layer patterned by negative phototransparency masks on a silicon wafer. This layer was deposited using SU-8 (MicroChem). 
The PDMS flow cell was fabricated by molding silicone elastomer (Sylgard 184, Dow Corning) to this master mold. PDMS was mixed in a 1:10 (v/v) ratio of catalyst and resin, poured into the master mold, degassed for 1 h in a desiccator and cured in an oven at 75°C for 1 h. 
To perform experiments 1uL (OD $\approx$ 0.005) of the desired culture was pipetted on a cover slip and covered by the polyacrylamide membrane and then the microfluidic device. The device was then connected to two syringe pumps (ProSense, NE-1000 and NE-300) by polyethylene tubing of 0.58 mm internal diameter (Smiths medical International Ltd.). The flow was controlled by a manual valve (Hamilton, HV 4-4). 
The culture medium flow rate during the experiments was 60 μL/min.

\subsubsection{Experiments with microfluidic device 2}
Another PDMS device was used for experiments involving the minCDE null mutant (ASC-1035) and labeled nucleoids strains (ASC1106). The device was developed by Daan J. Kiviet in the Martin Ackermann lab. It is similar to the device described in ref. \cite{Taheri-Araghi2014}, but it has wider microcolony wells. It contains a 200 μm wide main flow channel, splitting into two 100 μm wide flow channels (both 23.5 μm high). Perpendicular to these flow channels are 5 times repeated 0.75 μm high cavities (also known as "wells", where microcolonies of cells grow during the experiments) with widths of 1x 80μm, 1x 60μm, 2x 40μm, 3x 20μm, 3x 10μm + 3x 5μm, and depths of 60 μm, 30μm, 50μm and 40 μm. The PDMS devices were made by casting them into an epoxy mold, a gift from Daan J. Kiviet and the Martin Ackermann lab.
To produce the PDMS device, polymer and curing agent (Sylgard 184 elastomer, Dow Corning Corp.) were mixed in ratio of 1 mL of curing agent to 7.7 g of polymer (we found this deviation from the recommended 1:10 ratio provided a better rigidity of the PDMS). This mixture was cast into the epoxy mold. Air bubbles were removed from the mixture either by putting the mold and casting in a desiccator for 30 minutes, or by leaving the mixture for several hours before casting. The mold and casting were then put in an 80°C oven for 1-12 hours. Subsequently, the mold was removed from the casting, and holes were punched for the liquid in- and outlets. The casting was cut into a smaller size using a scalpel to remove rough, raised or uneven edges. Then the PDMS casting was covalently bound to a clean glass cover slip by treating the PDMS and glass surface with a portable laboratory corona device (model BD-20ACV, Electro-Technic Products, Inc.) (5-10 sweeps of approx. 5 seconds for each surface from approx. 5-10mm distance). The casting was gently tapped using a gloved finger to improve contact between the PDMS and glass surfaces. Consecutively, the device was baked for another 1-12 hrs and stored  for a couple of weeks before the experiment was started. 
To perform an experiment, 2 mL culture of \ecoli was grown to high OD (>1) in a 10ml Falcon culture tube on a rotator at 37°C. The concentration of bacteria was further increased 30x by spinning down 1ml of the sample in an Eppendorf tube at 2300 RCF, removal of supernatant and resuspension. To inoculate the device, first, 1 μL of sterile 0.01\% Tween (dH2O) solution was slowly pipetted into the PDMS device, after which 1 μL of the concentrated culture was introduced in the device. When bacteria had penetrated the growth wells, the device was connected to polyethylene tubing, pumps and a valve controller similar to the other microfluidic device. Superfluous bacteria in the flow channels were removed by the culture medium flow. The flow rate during these experiments was 16 μL/min.

\subsubsection{Experiments with gel pads}
Experiments with the SulA strain (ASC884) and the sYFP2 labeled FtsA strain (ASC777) were performed on polyacrylamide gel pads. To produce polyacrylamide pads \cite{Kiviet2014}, a mold was created by placing two 25 mm x 76 mm x 1 mm silanized microscopy glass slides (Thermo Scientific) on top of each other. The top glass slide contained a 18 mm x 52 mm rectangular hole, and the two slides were sealed together with high vacuum grease (Dow Corning). 
Polyacrylamide mix (1.25 mL 40\% acrylamide, 3.7 mL deionized sterile water, 50 μL 10\% ammonium persulfate, 5 μL TEMED) was poured into the cavity and covered by a silanized coverslip. 
The mix was placed at room temperature for half an hour to allow polymerization, and then cut into gel pads of approx. 5 mm x 5 mm x 1 mm which were stored in sterile dH2O. 
To perform experiments, a gel pad was soaked in the desired medium, placed in the cavity of a clean two-glass slide setup identical to the mold (except that glass slides were not silanized), inoculated on top with 1 uL (OD $\approx$ 0.005) of the desired bacterial culture, covered with a glass cover slip, and mechanically sealed with a metal clamp to avoid drying of the sample (see also Figure S1C).

\subsubsection{Singe cell microscopy}
In tetracycline experiments, strain ASC555 (see Key Resource Table) was grown in the microfluidic device first in clean M9 medium, then M9 medium with 1μM tetracycline (or 2μM or 10μM tetracycline for supplemental datasets), and then clean M9 medium again (see experimental model and subject details for detailed information on growth media). For temperature experiments, strain ASC777 was grown on M9 medium soaked polyacrylamide gel pads subsequently at 37°C, 42°C and 37°C; sYFP2-FtsA expression was induced with 3.5μM IPTG. For SulA experiments, SulA expression was induced in strain ASC884 with 200μM IPTG during O/N growth in culture tubes with M9 medium, and cells were transferred to polyacrylamide pads soaked in clean M9 medium next day. For YFP-MinD experiments, ASC784 cells were grown in culture tubes with M9 medium at 37°C. Filamentation was induced by 1 μM tetracycline and expression of YFP-MinD was induced by addition of IPTG (20μM final concentration) half an hour before imaging. 2 μL of culture was then imaged under the microscope between a glass slide and a cover slip. For minCDE null mutant experiments, cells were grown and filamented in M9 media with 2μM tetracycline, while divisions still occurred. For nucleoid labeling experiments, cells were grown and filamented in M9 media with 2μM tetracycline. For all time lapse experiments, phase contrast images were acquired at 1-2 minute intervals. Additionally, during the temperature experiment, fluorescent pictures were taken every 4 minutes, with an exposure time of 200ms. For nucleoid visualization, additional fluorescent images were taken every 5 minutes, with an exposure time of 25 ms. For the YFP-MinD experiments, only fluorescent images were taken at maximum acquisition rate with a 500 ms exposure time (i.e. with a rate of approximately 2 frames per second).

\subsubsection{Imaging and image analysis}
Cells were imaged with an inverted microscope (Nikon, TE2000), equipped with 100X oil immersion objective (Nikon, Plan Fluor NA 1.3), cooled CMOS camera (Hamamatsu, Orca Flash4.0), xenon lamp with liquid light guide (Sutter, Lambda LS), GFP, mCherry, CFP and YFP filter set (Chroma, 41017, 49008, 49001 and 49003), computer controlled shutters (Sutter, Lambda 10‐3 with SmartShutter), automated stage (Märzhäuser, SCAN IM 120 x 100) and an incubation chamber (Solent) allowing precise 37°C temperature control. An additional 1.5X lens was used, resulting in images with pixel size of 0.041 $\upmu$m. The microscope was controlled by MetaMorph software. Series of phase contrast images were analyzed with a custom Matlab (Mathworks) program originally based on Schnitzcells software \cite{Young2012}, which allows for automated segmentation of cells growing in a colony. The number of segmented and analyzed colonies was: 5 (1 μM tetracycline), 3 (2 μM tetracycline), 3 (10 μM tetracycline), 5 (SulA), 2 (temperature), 1 (nucleoid), 5 (Min deletion). In the Min deletion experiments, mini-cells were observed (inset Fig. 4C) but not segmented because of their abnormal size and dynamics. For all experiments, some cell cycles could not be monitored completely because the cells grew outside the field of view or because the experiment stopped, and were hence excluded from the analysis. To follow the cells over time, the images were manually corrected where necessary, and tracked to create a lineage branch-like structure. Each cell’s length (polynomial fitted to a cell’s curved segmentation region, or the segmented region's skeleton length for the temperature and sulA datasets) is computed for each frame. To determine relative division locations S, daughter cell lengths were divided by the maternal cell length (defined as the summed length of the two daughters). Growth rates were determined by fitting an exponential function to recorded cell lengths over multiple frames. For the experiments involving labeled FtsA or nucleoids, the fluorescent intensity along the cellular axis was determined using 1 pixel wide slices perpendicular to the morphologically computed skeleton. Peaks were then identified using the function peakfinder.m written by Nathanael C. Yoder; only peaks with a signal above a threshold level of 400 A.U. were considered (against an estimated background signal of approximately 50-100 A.U.). In Figure 3A, the signal intensity outside cells was decreased for visualization purposes.

\subsubsection{Simulations with the Meinhardt and De Boer model}
We simulated the Min system behavior using the differential equations and rate constants described in Table S1, which were developed by Meinhardt and De Boer \cite{Meinhardt2001}. We extended the range of bacterial lengths simulated. This simulation numerically solves the differential equations, with stochastically fluctuating reaction constants, to calculate the protein numbers in each length element of a cell for multiple reaction species. We ran simulations for bacterial lengths of 15 to 100 a.u. for 106 iterations and recorded the system state every 1000 iterations. We then calculated the time-averaged MinD protein number profiles for each bacterial length. 

\subsubsection{Simulations with the KC Huang model using MesoRD}
We simulated Min system behavior using the software MesoRD \cite{Hattne2005}, "a tool for stochastic and deterministic simulation of chemical reactions and diffusion in 3D and planar 2D spaces". 
The model of the protein interactions is described in Table S2, and based on a stochastic adaptation \cite{Fange2006} of the K.C. Huang model \cite{Huang2003}. Ref.  \cite{Fange2006} also provides the reaction scheme in Systems Biology Markup Language (SBML). 
We used a diffusion constant of 8.2 µm2/s as measured from cytoplasmic diffusion of GFP, which has a mass similar to MinD (26.9 kDa and 29.4 kDa respectively) \cite{Elowitz1999}, the other parameters indicated in table S2 are taken from \cite{Fange2006}. 
We ran 100s-300s simulations for a range of cellular sizes (with a compartment size of $5\cdot10^{-8}$, and recording the system state every second) and determined time-averaged concentration profiles.

\subsection{QUANTIFICATION AND STATISTICAL ANALYSIS}
The number of data points for each experiment can be found in the figure captions. To compare interdivision times, two sample t-tests were used (p = 0.05), as described in the main text and caption of Figure S2. Error bars always show SEM.

\subsection{DATA AND SOFTWARE AVAILABILITY}
Analysis and plotting was performed using custom Matlab scripts, which can be found at: \\
\texttt{https://github.com/TansLab/Tans\_Schnitzcells}, \\ 
\texttt{https://github.com/TansLab/Tans\_filamentation}, and \\
\texttt{https://github.com/TansLab/Common\_libraries}.
