


\hyphenation{met-a-bol-ic}



% Notes
% 
% For format of Scientific Reports, see:
% https://www.nature.com/srep/publish/guidelines#format-manuscripts
% - Title length 20 words
% - Main text 4500 words ("guide")
% - Abstract: 200 words (no refs)
% 	> "General introduction to the topic and as a brief, non-technical summary of the main results and their implications"
% - Note that figure captions cannot be more than 350 words
% - Preferably, method section limited to 1500 words
% - results may have subheadings
%
% For format of PLOS ONE, see:
% http://journals.plos.org/plosone/s/submission-guidelines
% - 2 titles, long and short: 250 characters, short title: 100 characters
% - abstract 300 words
% - ..?
% 
% Art of presenting science

% Notes on literature:
% Also check CRP page on ecocyc: https://biocyc.org/gene?orgid=ECOLI&id=CPLX0-226.


%\chapter{Negative metabolic feedback suppresses cell wide fluctuations}
\chapter{CRP responds dynamically to internal noise}
\label{chapter:CRP}

\textit{Martijn Wehrens, Laurens H.J. Krah, Benjamin D. Towbin, Rutger Hermsen, Sander J. Tans}

\section*{Abstract}

Some regulatory interactions are known to help the cell deal with changes in the environment, but it is unclear how these regulatory interactions respond to concentration changes due to stochastic fluctuations.
%
In this chapter, we address this open question by looking at the cAMP receptor protein (CRP), of which the 

% Chapter clear first page
\thispagestyle{empty}
\clearpage

\section*{Introduction}


%%% ------------------------
% Let's first define the problem (or knowledge gap)
%%% ------------------------

\subsection*{Environmental and stochastic input to regulatory networks}

The world is unpredictable.
%
A bacterial cell observes both its extracellular environment and its intracellular environment,
and adjusts its gene expression to 
optimize its chances of survival in this world.
%
%To survive bacteria need to respond to unforeseen situations and express the right gene at the right time. 
% Expressing the right gene at the right time is vital for bacterial survival. 
%Cells rely on biochemical networks to control gene expression.
%(Biochemical network is a general term that refers to the chemical interactions between proteins and small molecular mechanisms.)
To control gene expression a bacterium --- like any other cell --- relies on chemical interactions between proteins and small molecules \cite{Bray1995, Alon2006, Alon2007, Tyson2010}.
%A bacterium, like any other cell, relies on chemical interactions between proteins and small molecules to control gene expression \cite{Bray1995, Alon2006, Alon2007, Tyson2010}.
%These interactions are also referred to as "biochemical networks".
These interactions, also referred to as "biochemical networks", 
%
%Biochemical networks 
%allow cells to deal with changes that can come both from the extracellular and the intracellular environment.
allow cells to deal with 
% two types of 
unpredictable situations
both in the intracellular and extracellular environment.
%dynamics.
%
%On one hand, the extracellular environment might change, and a different gene expression profile is required.
Changes in the extracellular environment might require the cell to adjust gene expression accordingly.
%
On the other hand,  even without environmental changes, the stochastic nature of chemical reactions can lead to fluctuations of proteins and metabolites over time within the cell \cite{Elowitz2002,Kiviet2014}.
The architecture of the biochemical network might enable the cell to deal with these fluctuations, or even use them to its advantage (see also chapter \ref{chapter:literaturereview}).
%
%Studies often focus on how the biochemical network is designed to deal with either one or the other of these two types of dynamics.
%Studies often investigate network architecture in only one of these contexts.
%Studies often investigate network architecture in only an "environmental" context or only in a "stochastic fluctuation" context.
Studies often investigate network architecture only in the context of "environmental" inputs, or only in the context of "stochastic fluctuation" inputs.
%
However, any regulatory interaction in the cell is faced with both these inputs.
%However, any regulatory interaction in the cell is faced with both an environmental and  contexts.
%
%It is unclear to what extend regulatory interactions that are known to help the cell deal with a changing environment are affected by stochastic concentration fluctuations
This might have large implications for the optimal network architecture.
%
%In this work, we address the open question that connects these two contexts:
In this work, we investigate the link between environmental and stochastic regulation for the first time. % by asking
%
%are 
Specifically, we ask:
are regulatory interactions that are known for their role in adaptation to environmental changes also 
%affected 
activated 
by concentration fluctuations in the intracellular environment due to noise?


%%% ------------------------
% Explaining what is known about the CRP system
%%% ------------------------

%\subsection*{The metabolic regulator CRP recieves both stochastic and environmental input}
\subsection*{Stochastic and environmental input in a model system: CRP}

%To investigate this question, we take a closer look at the dynamics of regulation by the cAMP receptor protein (CRP; previously called catabolite gene activator protein, or CAP). 
To investigate this question we looked at a master regulator of metabolic enzyme expression in \textit{Escherichia coli}.
%
Metabolic enzymes convert large carbohydrate molecules into smaller metabolites, and generate energy for the cell during this process in the form of ATP \cite{Nelson2005}.
%
It has been suggested that the cAMP receptor protein (CRP)\footnote{CRP was previously called catabolite gene activator protein, or CAP} 
controls the expression level of all catabolic genes in concert \cite{You2013, Hui2015, Kochanowski2017};
%
CRP controls 378 promoters, among which 70 transcription factors \cite{Green2014, Shimada2011}.
%
Amidst these targets are also all enzymes in the TCA cycle, see also figure \ref{fig:CRP:figOverviewTCARegulation}.
%
CRP is allosterically activated by the small signaling molecule cyclic adenosine monophosphate (cAMP), 
which is produced from ATP by the enzyme adenylate cyclase (CyaA).
Adenylate cyclase is thought to be inhibited by $\upalpha$-ketoacids, such as oxaloacetate (OAA), $\upalpha$-ketoglutarate ($\upalpha$-KG) and pyruvate (PYR) \cite{You2013}.
(Previously, also the phosphorylated enzyme IIA of the phosphotransferase system was thought to activate adenylate cyclase \cite{Keseler2017, Deutscher2008, Gorke2008}, but it has now been suggested that the role of $\upalpha$-ketoacids feedback is bigger \cite{You2013}.)
%
Thus, the CRP-cAMP regulation is wired such that metabolites from the TCA cycle and glycolysis pathway provide negative feedback on metabolic enzyme expression, see also figure \ref{fig:CRP:fig0}.A. 
It is thought that this negative feedback in the CRP regulatory system
is responsible for adjusting the concentration of metabolic enzymes in the cell to handle the carbohydrate nutrient source available to the cell appropriately \cite{Towbin2017}.
%
%More specifically, to maximize growth rate, the cell needs to properly allocate its protein pool among different generic activities such as catabolism, anabolism, protein production, and replication of DNA \cite{Hui2015}.
%The negative feedback loop is thought to enable the cell to estimate what the best concentration of metabolic enzymes is under most conditions
%and set enzyme expression accordingly \cite{Towbin2017}, see also figure \ref{fig:CRP:fig0}.A.
%
However, when the cell is growing steadily on a single carbon source for an extended period of time, 
there is still a large heterogeneity observed in growth rates and metabolic enzyme expression levels \cite{Kiviet2014} (see also chapter \ref{chapter:literaturereview}).
%
%Observations by Kiviet et al. suggest that single cell stochastic fluctuations in enzyme levels result in changes in metabolite fluxes, and eventually fluctuations in growth rate \cite{Kiviet2014}.
Observations by Kiviet et al. suggest that single cell enzyme concentrations fluctuate stochastically over time, which in turn results in fluctuations in metabolite fluxes and eventually in growth rate \cite{Kiviet2014}.
%
Thus, single cell metabolite concentrations might fluctuate over time due to stochastic fluctuations .
%This observation also implies that there might be large fluctuations in the pool of metabolites available to the cell.
That would imply that also the CRP system receives a large variation of inputs in single cells, even when there are no changes in the extracellular environment.
%
%In other words: though the mean CRP activity might be set by the external environment, single cells might have wholly different CRP activities due to stochastic fluctuations.
%
We here set out to investigate whether the CRP system indeed experiences and acts on these different stochastic inputs.

\subsection*{CRP with and without processing stochastic input}

To find out whether the CRP system responds to stochastic input we need to decouple the responses to stochastic and environmental inputs.
%To address this question we need to be able to decouple the responses to these two different inputs.
%
To this end, we employed two strains that have the appropriate mean CRP activity for the sugar source they're grown on, 
but where in one strain the feedback loop cannot respond to stochastic input it might receive.
%
The two strains we used are a wild type MG1655, and a \textit{cyaA}, \textit{cpda} null mutant \cite{Towbin2017}.
(\textit{cyaA} produces cAMP and \textit{cpda} degrades cAMP.)
% note $\Delta$ is upright by default
%
The latter strain is unable to modify its cAMP levels, and thus has a crippled feedback loop that cannot respond to both environmental and stochastic input signals.
%
%This means these cells cannot use their feedback regulation to determine the correct mean CRP activity they should have in their environment. 
%
To repair this strain's ability to express the right amount of metabolic enzymes with regard to their environment, 
we provide cAMP extracellularly in the cells' growth medium.
%
This bypasses the feedback loop and directly sets the correct CRP activity for the sugar they are growing on \cite{Towbin2017},
but leaves the feedback loop unable to respond to stochastic input signals.
%
This experimental design allows us to compare a cell that has the correct mean CRP activity and \textit{can} respond to stochastic input signals, 
with a cell that has the correct mean CRP activity but \textit{cannot} respond to stochastic input signals.
%
We then use single cell time lapse microscopy and fluorescent labeling to determine single cell growth and CRP dynamics. 
%
We first use these single cell technique to see whether the CRP system can respond on timescales at which the stochastic fluctuations take place.
%
Secondly, we use cross-correlation analyses to quantify the dynamical growth and CRP behavior of the two strains described above.
%
This shows that the CRP system can indeed operate on timescales that are comparable to time scales of stochastic fluctuations, and moreover that
the dynamics of metabolism and growth change remarkably when its regulation is not able to respond to stochastic inputs.
%
These observations suggest that regulatory interactions also respond to stochastic inputs.
%
This implies in turn that our notion of steady state behavior should be updated, 
as the cellular state might be constantly changing in a way that is facilitated by regulatory interactions.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% figure with constructs etc.
\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{CRP-fig0.pdf}
	\caption{ 
		(A) 
	}
	\label{fig:CRP:fig0}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
% See file "CRP_literalgraveyard.tex" for more previous versions and ideas.
% 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Results}

\subsection*{The CRP system is able to respond to dynamic signals}

Our hypothesis is that stochastic fluctuations in metabolic enzyme concentration result in metabolite concentration changes, which in turn lead to a response by the CRP system.
%
To test this hypothesis, we first subjected the CRP system to artificial cAMP fluctuations to see whether it can respond to such quick fluctuations.
%
%However, the time scales at which these stochastic fluctuations occur might be too fast 
%for the CRP regulatory system to respond.
%To test the response time of the CRP system, we subjected cells to quickly alternating low and high cAMP concentrations in our flow cell.
%
In this paragraph we show that the CRP system indeed responds considerably at rapid timescales to changes in the cAMP signal it receives.


To probe the response of the CRP system to a quickly changing signal, 
%we subjected 
we used
our \textit{cyaA}, \textit{cpda} null mutant strain that does not respond to intracellular metabolic feedback (we'll refer to this strain as $\Delta$cAMP from hereon for convenience).
Instead of the intracellular signal, we provided it with an artificial cAMP signal by providing high (2100$\upmu$m) or low (43$\upmu$m) concentrations of cAMP in the cellular growth medium.
%
Additionally, 
to assess the response of the CRP system we equipped these cells with a chromosomally inserted reporter construct that reads out the expression level of metabolic enzymes \cite{Towbin2017}.
% with a chromosomally inserted copy of this reporter construct
%which carried a chromosomal insertion with this reporter, 
%to a regime of alternating high and low cAMP concentrations.
%under the microscope whilst alternating between high and low cAMP concentrations.
%
%To be able to assess the response of the CRP system, we needed to measure its output.
%To be able to do this, we used a reporter construct that reads out the expression level of metabolic enzymes \cite{Towbin2017}.
%
%
This reporter construct uses the lac operon promoter from which the lacI binding site has been removed, see figure \ref{fig:CRP:fig0}.\red{X}.
The promoter is fused to a fluorescent reporter protein (either gfpmut2 or mVenus, depending on the experiment) which allowed us to
%estimate 
assess
the expression level of metabolic enzymes as induced by CRP.
%
%
%
%
We grew these cells in our microfluidic device (see methods section) under the microscope, and 
first alternated between high and low concentrations of cAMP at 1hr intervals, and then alternated between the same concentrations at 5hr intervals.
%
Figure \ref{fig:CRP:fig1}.A shows the how the concentration of metabolic reporter responds to the quickly changing cAMP system. 
%
During fast switches (1 hours) the signal went up and down substantially, consistent with respectively the high and low cAMP signal supplied, with a minimum of 215 a.u. and a maximum of 258 a.u..
%
During slow switches (5 hours) it becomes apparent that the signal did not reach an equilibrium value during the fast switching behavior, as the metabolic reporter reaches a lower and higher minimum and maximum respectively at 144 a.u. and 407 a.u.. 
%
Importantly, the behavior in the regime with fast switches suggested that the concentrations of metabolic enzymes under the control of a CRP promoter can respond quickly and substantially in response to a varying input signal. 
 


This led to our next question: what is the effect of these changes in metabolic enzyme concentration on the cell? 
%
To gauge this, we looked at how cellular growth rate responded to the fluctuations in cAMP signal. 
%
Figure \ref{fig:CRP:fig1}.B shows that also the growth rate varies substantially, again suggesting the cell is susceptible to fluctuations in regulatory molecules. 
%
However, the growth rate behavior shows some puzzling features. 
%
Firstly, it appear that in the fast switch regime the high cAMP concentration immediately results in a sustained faster growth rate.
This is unexpected, as too much metabolic enzyme expression is expected to come with the cost of superfluous protein production, and might even lead to a detrimental unbalanced ratio of fluxes in the cell \cite{Ray2016}.
%
However, when looking at the slower switch regime we see that this signal also goes down after while. 
%
We can explain this seemingly erratic behavior by plotting a time trace of the growth rate against the concentration of metabolic reporter.
%
This trace is shown in figure \ref{fig:CRP:fig1}.C. 
%
The most left green square in this figure indicates a switch to a low cAMP concentration, and the part of the time trace that starts at this square shows that as the metabolic reporter concentration goes down, the growth rate first goes up, and then comes down again. 
This is consistent with the idea that metabolic enzyme concentration has an optimal value above and below which the cell grows worse \cite{Jensen1993, Dekel2005, Berkhout2013, Ray2016, Towbin2017}.
%earlier observations which show that the relationship between growth rate and metabolic enzyme concentration shows an optimum concentration above and below which the cell grows worse.
%
The idea of an optimum metabolic enzyme concentration is also confirmed when we plot growth rate against the metabolic reporter concentration, as shown in supplemental figure \ref{fig:CRP:scatterspulsing}. 
%
Conversely, looking at the part of the trace that starts at the leftmost red square, the reporter concentration appears not to be affected, but the growth rate does go up.
%
The rest of the trace is then consistent with traversing on the right part of an optimum curve.
%
An explanation for this part of the trace being inconsistent with the earlier discussed part of the trace, and the quick growth rate response without concentration change, 
could be that cAMP also has some post-translational roles in the cell. 
This would explain the quick responses both in the fast-switching regime and the quick response to an upshift of cAMP.


If there were a post-translational role for cAMP, then the concentration of cAMP should show a more consistent relation with growth rate than the concentration of metabolic reporter.
%
%The concentration of CRP.cAMP should be proportional with the cAMP concentration, and so should the production of our metabolic reporter.
The production rate of our metabolic reporter should be proportional to the cAMP concentration, 
so the metabolic reporter production rate could serve as a proxy for cAMP concentration.
%
However, this production rate also depends on the growth rate of the cell, and might do so in a non-trivial manner.
Indeed, it is difficult to find a straightforward explanation for the relationship between production and growth over time, as shown in supplemental figure \ref{fig:CRP:productiontimeevolutions}.
Only when the metabolic production is normalized against a second reporter with a constitutive promoter (see figure \ref{fig:CRP:fig0}.B) as a reference, do we see that most traces overlap.
%
(Such a normalization could also be done for the concentration, which appears to make the traces slightly more reproducible, but not substantially different.)
%
The larger overlap in the normalized production vs growth time trace (compared to the concentration of metabolic reporter vs. growth) offers some support for a post-translational role of cAMP.
%
In \ecoli however, cAMP is usually described as an activator of the translational regulator CRP protein only \cite{gorke2008, keseler2017}.

Nevertheless, the observations made in this pulsing experiment show that the CRP system can respond to quickly fluctuating input signals, as also might occur due to stochastic fluctuations.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% figure with time evolution of cAMP pulsing experiment
\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{CRP-fig1_.pdf}
	\clearpage % insert a page break
	\label{fig:CRP:fig1}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{The CRP system can respond to quickly changing input signals.}
	The concentration of the metabolic reporter (A) and growth rate (B) over time during the cAMP pulsing experiment. Panel (C) shows how the relation between the growth rate and metabolic reporter evolves over time, where the color of the trace and the arrows indicates the progression of time.
	Red circles or squares indicate the points in time at which a switch was made to growth medium with a cAMP concentration of 2100 $upmu$M, green symbols indicate the timepoints at which a swich was made to growth medium with a cAMP concentration of 43 $\upmu$M.
	Blue dots correspond to single cell observations, and the black line is the population average. In panel C, only the population average is shown (see supplemental figure 	\ref{fig:CRP:timevolutionCRPgrowthsinglecell} for a few examples of single cell traces).
}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%\subsection*{Probing CRP dynamics in a constant environment}
\subsection*{The effect of stochastic fluctuations on CRP regulation}

\subsubsection*{The feedback loop affects population dynamics}

Thus, given that the CRP system seems sensitive to quickly changing input, we wanted to further probe how cells growing in a constant environment respond to intracellular stochastic fluctuations.
%
%Since we suspected that CRP regulation might both respond to environmental and intracellular fluctuations, 
%
We wanted to isolate and/or manipulate the regulatory response to these fluctuations.
%In order to study the response of the regulatory system to stochastic fluctuations, we needed to isolate the regulatory response to those fluctuations.
%
%Indeed, 
We managed to remove any potential response of the regulatory system to stochastic fluctuations, while maintaining the mean expression level 
required in the cellular environment.
%required as response to the environment.
%
This was done by growing our $\Delta$cAMP strain on minimum medium supplemented with lactose and 800 $\upmu$M cAMP.
%
This concentration of cAMP resulted in growth rates that are comparable to those observed in wild type cells growing on lactose (see also supplementary figure \ref{fig:CRP:overviewsummaryparams}).
On the other hand, the cAMP levels in these cells were fixed and thus unable to respond to intracellular metabolite concentrations.
%
Using our metabolic reporter construct, we were able to assess the CRP dynamics in these cells.
We compared CRP expression dynamics in these cells without feedback against wild type cells (which also carried the metabolic reporter and were also growing on lactose).
%
This thus gave us a way to decouple the regulatory response to stochastic intracellular changes from the regulatory response to environmental changes,
and study whether there is indeed regulatory activity in response to stochastic changes.

%We first investigated the metabolic dynamics in the wild type setting.
%
The left panel in figure \ref{fig:CRP:fig2}.A shows a scatter plot that relates the concentration of metabolic reporter to the growth rate for a single cell experiment with wild type cells.
%
%As expected, individual cells that have a higher metabolic enzyme expression show a lower growth rate.
%
%However, d
The scatter cloud appears to lay on a straight line with a negative slope.
%
Conversely, the green cloud in the right panel in figure \ref{fig:CRP:fig2}.A shows
% the relationship between metabolic regulation and growth observed in cells 
the growth-expression relationship observed in cells 
that lack the ability to respond to stochastically fluctuating metabolite concentrations, ie. our $\Delta$cAMP cells growing in 800 $\upmu$M cAMP lacking negative feedback regulation.
%
Remarkably, the cells without feedback do not show a similarly negatively sloped relationship between metabolic expression and growth rate.
%
%This indicates that the dynamics of wild type cells is influenced by stochastic influences from inside the cell.
This indicates that the CRP regulatory interactions play a role in shaping stochastic fluctuations in the cell. 
%
To make sure  
%this behavior 
the changes 
were not due to generic changes in cellular dynamics, we made use of a second reporter construct.
%
This construct carried a similar promoter to the metabolic promoter, but it contained mutations that removed the CRP binding site and introduced a $\upsigma$70 consensus binding site instead \cite{Towbin2017} (see also methods).
%
%The idea is that the resulting constitutive promoter fused to a fluorescent mCerulean reporter shows the generic dynamics that exist in protein production in the cell.
This resulted in a constitutive promoter, which was then fused to a fluorescent mCerulean reporter.
%
The aim of this construct was to report specifically for the generic dynamics of protein production due to cellular state changes (i.e. intrinsic noise)
which might also be experienced by the CRP reporter ---
%
%which result from stochastic cellular state changes,
%and growth rate fluctuations, 
%usually referred to as extrinsic noise
% --- but not for due to CRP regulation.
but to exclude the effects of CRP regulation.
%
Figure \ref{fig:CRP:fig2}.B shows that the expression-growth relationship of this constitutive reporter does not change between cells with or without feedback (blue and green clouds).
This confirms the attribution of the change in dynamics observed in \ref{fig:CRP:fig2}.A to the CRP regulation responding to stochastically fluctuating metabolite concentrations.

\subsubsection*{The not so average cell}

Our "feedbackless" $\Delta$cAMP cells allowed us to investigate a second question:
how do the stochastic deviations \textit{from} mean behavior relate to the mean behavior itself?
%
To further probe the behavior of cellular populations, we exposed our $\Delta$cAMP cells to two additional conditions: 
%
%To further investigate the behavior of cells , we exposed our $\Delta$cAMP cells to two additional constant conditions:
respectively 80 or 5000 $\upmu$M cAMP (further conditions were equal to the 800 $\upmu$M cAMP condition).
%
The behavior of cells on the level of population averages is reasonably straightforward to understand.
%
As we show in supplementary note I at the end of this chapter, the steady state relationships between the protein expression and growth are determined by three ingredients:
(1) the interaction between protein production and dilution, (2) the concave relationship between metabolic enzyme expression and growth which suggests there is an optimal level of metabolic enzyme expression and (3) the observation that an increase in metabolic protein expression leads to a mandatory equal decrease in constitutive protein expression and vice versa.
%
However, neither the wild type (blue) or $\Delta$cAMP (green) scatter clouds in figure \ref{fig:CRP:fig2} seem to adhere to the concave relationship:
%
the wild type cells simply show a negative relationship between CRP concentration and growth rate, whilst the feedbackless cells lay on a line of constant growth rate.
%
The population averages of the $\Delta$cAMP cells obtained from the additional experiments at  80 or 5000 $\upmu$M cAMP follow the previously observed concave curve \cite{Towbin2017}, 
but the single cell datapoints (visualized by the red and orange clouds respectively in figure \ref{fig:CRP:fig2}.A) do not seem to lay on a continuous concave curve.
%showed population average values that follow the same concave curve as observed earlier (supplementary figure \ref{fig:CRP:scatterspulsing} and \cite{Towbin2017}).
%
%The clouds roughly follow this behavior, but seem to deviate.
%
This deviation from the average behavior is much more pronounced in the concentration-growth relationship of the constitutive reporter.
The population average behavior of the constitutive reporter growth rates is expected to show a concave relationship with reporter concentration (supplementary notes) similar to the metabolic reporter,
but the single cell data does not follow this relationship at all, see figure \ref{fig:CRP:fig2}.B. 
%
% Despite the trend observed in the average growth-expression relationship, which showed a concave cAMP-growth curve with an optimum metabolic expression (supplementary figure \ref{fig:CRP:scatterspulsing} and \cite{Towbin2017}), we found that the shape of this cloud appears to follow a straight line with a negative slope.
%On one hand, the average concentration seems consistent with ideas about cellular resource allocation \cite{You2013}: cells that are forced to produce more metabolic proteins, automatically produce less other proteins such as the constitutive reporter. This relationship between the average concentrations of the metabolic and constitutive reporters is also directly visualized in the top right panel of supplemental figure \ref{fig:CRP:fig3scatters_CC_pp}.
%
%However, the clouds in figure \ref{fig:CRP:fig2}.B on the other hand, which show the deviations of single cells from the average cell, do not follow this average behavior at all.

\subsubsection*{Average relations do not predict dynamic relations and CRP regulation responds to stochastic inputs}
%\subsubsection*{Deviations from average behavior and CRP regulation responds to stochastic inputs}

Taken together, these observations suggest that the dynamical behavior of single cells
%that deviate from the average cell, 
does not necessarily follow relationships that exist for average quantities over different conditions. More importantly, the fact that cells that are unable to respond to internal metabolite fluctuations show substantially different behavior from those that can, indicates that the CRP regulation network is actively responding to inputs when the cell resides in a constant environment.
%
To further understand the shape of the single cell data point clouds, we turn to cross-correlation analyses in the next paragraph.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{CRP-fig2.pdf}
	\caption{ 
		\textbf{Artificial removal of negative feedback regulation changes the growth-metabolism relationship and reveals its dynamic role.}		
		% \textbf{Growth-concentration relationships for metabolic and constitutive reporters.}
		(A) Colored dots show single cell growth rate values plotted against respective single cell concentrations of metabolic reporter, which is a proxy for the concentration of metabolic enzymes. 
		The left panel shows that wild type cells which deviate from the average behavior (circle) show a negative correlation between metabolic reporter and growth rate.
		The green dots in the right panel shows that this behavior changes in $\Delta$cAMP cells which are not able to respond to internal metabolite levels. 
		Whereas the green cloud corresponds to externally supplied cAMP levels that lead to wild type growth rates (800 $\upmu$M cAMP), the 
		red and orange dots correspond to cAMP concentrations that lead to diminished growth rates (80 and 5000 $\upmu$M respectively).		
		Note that the black lines show the average growth rate for cells that are binned according to concentration, and the black isolines reflect kernel density estimates of the probability distribution (using the matlab function \texttt{kde2d} \cite{Botev2010}).
		(B) As panel A, except that this panel shows the relationship between growth and the concentration of a constitutive reporter. 
		As opposed to the metabolic behavior, single cells that deviate from the average expression level for the respective condition do not show  different behavior.
		This shows that the change in metabolic behavior shown in A is not a generic effect on protein expression, but specific to the (regulation of the) metabolic reporter.
		Additionally, these panels also illustrate the difference between single cell behavior from the trend set out by the average data points.
	}
	\label{fig:CRP:fig2}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\subsection*{Understanding the CRP response to stochastic fluctuations}

We want to understand the effect of fluctuations in cellular parameters on regulatory interactions and other parameters (such as growth) at the single cell level.
%
To understand these dynamic relationships time is an important component.
%To understand how parameters dynamically relate to each other in the stochastic cellular environment, time is an important component.
%
%It has for example been shown by using cross-correlations that certain enzyme's single cell production rates have a delayed effect on single cell growth rates \cite{Kiviet2014}.
%
In this paragraph we dissect the time dynamics with cross-correlations
%use cross-correlations 
to show that metabolic fluctuations would transfer to cellular growth rate if it were not for the existence of the negative feedback CRP regulation.

\subsubsection*{Cross-correlations reveal rich dynamics between metabolism and growth}

%We here also employ cross-correlations to investigate the delayed correlations between our metabolic, constitutive and growth parameters.
%
We start by investigating the effect that single cell metabolic concentration fluctuations have on growth rates.
%
The cross-correlation $R_{M,\mu}(\tau)$ will show us the relation between the metabolic reporter concentration and past and future growth rates.
%
More precisely, as explained in chapter \ref{chapter:methods}, the cross-correlation $R_{f,g}(\tau)$ calculates the average correlation between a parameter $f$ at time $t$ with parameter $g$ at time $t+\tau$.
%
The colored lines in figure \ref{fig:CRP:fig3}.A-B show the cross-correlation $R_{M,\mu}(\tau)$ for wild type (green) and feedbackless $\Delta$cAMP cells (blue).
%between the metabolic reporter concentration $M$ and growth rate $\mu$ based on single cell observations.
%
%As explained in \red{XXADDREFXX}, the cross-correlation $R_{x,y}(\tau)$ calculates the correlation between a parameter $x$ at time $t$ with parameter $y$ at time $t+\tau$; thus 
%correlations between $x$ and future or past values of $y$ become apparent.
%
From these curves it is immediately apparent that the dynamic metabolic concentration-growth relationship is very different between cells 
that possess the endogenous CRP feedback and those that do not.
%
Not only is the difference between correlations in wild type and $\Delta$cAMP cells that was also observed earlier in the scatter plots apparent from the values at delay $\tau=0$,
but it also becomes apparent that the concentration-growth relationship shows a strong negative correlation at negative delays, 
whilst the $\Delta$cAMP cells show a relationship that is positive for most delays and that is equal for positive versus negative delays, or even slightly more present at positive delays.
%
%Not only is the correlation that was observed earlier in the scatter plots apparent from the value at delay $\tau=0$, the concentration-growth relationship also shows a strong negative correlation at negative delays, whilst for the .
%
Previous studies showed that investigating the relationship between production rates and growth rates can also give further insights in the dynamics of metabolism and growth \cite{Kiviet2014}.
%
Hence, we here also tracked the production rate of our reporters.
%
The scatter plots for these relationships are shown in supplemental figure \ref{fig:CRP:fig2sup}.
%
Note that it is good to place these scatter plots in the context of the expected population average relationships (figure \ref{fig:CRP:averagerelations2}), 
since these reveal that a specific production rates can map to two growth rate values.
% on the algebraic curves.


%
To further understand these dynamics, we also tracked the production rates of the metabolic and constitutive reporters.
%



This also highlights that the dynamic interactions between cellular parameters go beyond instantaneous relationships that are captured by scatter plots, but instead show richer interactions that act over delays. 



\subsubsection*{To write down}

reveals more <richly> differences in dynamics


supplementary note benjamin (to write) shows that excursions from mean concentrations can take arbitrary form
>> so it is convenient to generalize them to a linearized model




stronger with future growth

To further investigate the observation that stochastically different cells do not follow the mean trend ....


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{CRP-fig3.pdf}
	\caption{ 
		\textbf{Without feedback, metabolic dynamics change and transmit to growth.}
		(A-B) Cross-correlations $R_{M,\mu}(\tau)$ between single cell metabolic reporter concentration $M$ and growth rate values $\mu$ (colored lines) and 
		cross-correlations $R_{p_M,\mu}(\tau)$ between metabolic production rate $p_M$ and growth rate values $\mu$ (black lines). 
		Correlations for wild type cells are shown in A, while correlations for feedbackless $\Delta$cAMP cells are shown in B.
		%
		(C-D) Cross-correlations $R_{Q,\mu}(\tau)$ between single cell constitutive reporter concentration $Q$ and growth rate values $\mu$ (colored lines) and 
		cross-correlations $R_{p_Q,\mu}(\tau)$ between constitutive production rate $p_Q$ and growth rate values $\mu$ (black lines). 
		Correlations for wild type cells are shown in C, while correlations for feedbackless $\Delta$cAMP cells are shown in D.
		%
		For all panels, the correlation (which is normalized) is plotted on the y-axis, and reflects the average correlation between two parameters between time points $t$ and $t+\tau$, 
		the delay $\tau$ is plotted on the x-axis (in hours). The faded lines indicate cross-correlations from different microcolonies, the darker lines their averages. The cross correlations are calculated from cell lineages as described in chapter \ref{chapter:methods}.
		%(A) Cross-correlation for the wild type cell's metabolic reporter expression and growth rate.
		%(B) Cross-correlation for the feedbackless $\Delta$cAMP cell's metabolic reporter expression and growth rate.
		%(C) Cross-correlation for the wild type cell's constitutive reporter expression and growth rate.
		%(D) Cross-correlation for the feedbackless $\Delta$cAMP cell's constitutive reporter expression and growth rate.
	}
	\label{fig:CRP:fig3}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%







%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{CRP-fig4.pdf}
	\clearpage % insert a page break
	\label{fig:CRP:fig4}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{A simple model explains how feedback filters out noise transmission.}
	A simple model predicts correlation inversion.
	A.
	B.
	C.
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{Discussion}

CONCLUSION: THE RULES ARE THE SAME BUT THE THINGS THAT CHANGE DUE FLUCTUATIONS OR IN CONDITIONS ARE DIFFERENT

\textbf{Talking points.} Evolution doesn't separate dynamic and steady state functionality, and can potentially optimize both.

> Perhaps the feedback ensures that the cells also exhibit "c-line" behavior as observed by you2013.
- Figure \ref{fig:CRP:fig2}.A is also consistent with You2013, i.e. the C-line. However not entirely clear how this would mechanistically work.
> Why are not all cells growing faster?
> Also mention Chalancon2012, which talks specifically about interaction between regulation fucntion and noise in a network.
> Mention Rosenfeld2005, idem.
> Check out review bruggeman2018.
> Talk about CV
> Overproduction metabolic -> too little production generic proteins -> that's why growth decrease.. (this is a bit nuanced difference from simply overproduction costly)

> Additional strains made and conditions measured.

For example, how cellular composition changes in response to different food sources has been a topic of study for a long time \cite{Schaechter1958}.
%
Regulation of metabolism has recently been shown to fit into a larger picture, in which the cell co-regulates large groups of genes together.
Each of these groups (also called sectors), relate to a major 
%
% Recently, it has been shown that large groups of genes are co-regulated, these groups are also called sectors, each of which relate to a major category of cellular activity such as 
category of cellular activity such as 
metabolism, anabolism, protein synthesis, replication, etc 
%catabolism, metabolism, anabolism, , protein synthesis, replication, etc. 
\cite{Klumpp2009, You2013, Scott2014, Hui2015, Hermsen2015, Erickson2017}.
%
% Note that Erickson is not really growth laws itself, but more about how a switch between two environments occurs
%
An important player in regulating the size of the metabolic sector is the cAMP receptor protein (CRP) \cite{Keseler2017, Grainger2005, Robinson1998, Zheng2004, Gorke2008, Fic2009, Green2014}.
%
CRP is activated cyclic adenosine monophosphate (cAMP) and it controls 378 promoters, among which 70 transcription factors \cite{Green2014, Shimada2011}.
%
The CRP.cAMP regulation 
%
Recently, it has been shown that the CRP.cAMP regulation 

%So-called growth laws describe how the ratio between these sectors is controlled by the cells to adapt to different environments.
%
%For some sectors, it is known that their expression level can be controlled by a single molecule or protein.
%For example, guanosine tetraphosphate (ppGpp) controls ribosomal expression \cite{Cashel1969, Potrykus2008, Ross2013, Hui2015}, and the 
%cAMP receptor protein (CRP) controls expression of metabolic enzymes.
% 



\section*{Methods}



\begin{table}[h]
	\begin{tabularx}{\textwidth}{llXl}

	\textbf{ASC number}	& \textbf{Shorthand} & \textbf{Description}	\\
	\hline

	ASC839	& 				& Wild type MG1655 strain btained from Benjamin Towbin, Uri Alon lab (also known as strain bBT12 and CGSC number 8003). Known mutations: $\uplambda$-, 	$\Delta$fnr-267, rph-1. \cite{Towbin2017} \\
	ASC839	& 				& \textit{cyaA}, \textit{cpda} null mutant. Obtained from Benjamin Towbin, Uri Alon lab (also known as strain bBT80). Based on ASC838. \cite{Towbin2017} \\
	
	
	ASC0990  &  			& Wild type strain, except for $\Delta$(galk)::s70-mCerulean-kanR and $\Delta$(intc)::rcrp-mVenus-cmR. Kanamycin and chloramphenicol resistant. \\
	ASC1004  & $\Delta$cAMP & Strain based on ASC839 ($\Delta$cyaA $\Delta$cpda), introduced $\Delta$(galk)::s70-mCerulean-kanR and $\Delta$(intc)::rcrp-mVenus-cmR. Kanamycin and chloramphenicol resistant. \\
	
	\hline
	\end{tabularx}
	\caption{\textbf{Strains used in this work.} ASC stands for AMOLF strain collection.}
\end{table}

\begin{table}[h]
	\begin{tabularx}{\textwidth}{llXl}
		
		\textbf{ASC number}	& \textbf{Shorthand} & \textbf{Description}	\\
		\hline
				
		\red{XXX} & 				\red{XXX} & 				\red{XXX} \\
				
		\hline
	\end{tabularx}
	\caption{\textbf{Additional strains produced for this work, but not used in experiments presented here.} ASC stands for AMOLF strain collection.}
\end{table}

\textbf{Strains} See table \ref{table:CRP:strains}.

\textbf{Pulsing experiment.} Strain asc1004 was grown O/N at 30 $\degree$C and 10X concentrated by spinning the cells down at \red{5K XX}, removing supernatant and resuspension in a table top centrifuge.
Cells were introduced into microfluidic device \red{type 2} (see chapter \ref{chapter:filarecovery}) with a syringe, after which the device was placed under the microscope in a 37 $\degree$C temperature chamber and we supplied TY medium (flow rate 8 $\upmu$l/min) for a few hours, whereafter we switched to M9 minimal medium plus 0.2 mM uracil, 0.1 $\%$ lactose, 0.01 $\%$ tween and 300 $\upmu$M cAMP (flow rate 7 $\upmu$l/min).
After this cells were grown in the same medium but supplemented with 0.001 $\%$ tween and sequentially 1hr of 2100 $\upmu$M cAMP ("high") and 1hr of 43 $\upmu$M cAMP ("low"), which was repeated 5 times (totaling 10hrs), and then 5 hrs low, 5 hrs high, and 5 hrs low (all at a flow rate of 8 $\upmu$l/min). 
Times at which the valve switches were recorded and in the analysis corrected by adding the arrival delay of 58 minutes (in this particular experiment that delay was not yet optimized).
Fluorescent images were taken every 20 minutes, using a CFP and YFP filter set (chroma models 49001 and 49003 respectively), both with exposure times of 150ms.
A selection of this sequence was analyzed and displayed.  

\textbf{More detailed information on the computer analyses.} 
For a description of the computer analyses and more information see chapters \ref{chapter:methods} and \ref{chapter:filarecovery}.
%Extended information can be found in chapters \ref{chapter:methods} and \ref{chapter:filarecovery}. 
However, some details are also worth mentioning here. 
Since concentrations, production rates and growth rates play a large role in this chapter, it is good to indicate how they are calculated.
Concentration (a.u./px) is defined as the mean fluorescence signal (a.u.) in a central bar of pixels along the cell's long axis \cite{Kiviet2010}.
%
To calculate the production rate (a.u. px$^{-1}$ min$^{-1}$), first the sum of the fluorescence signal (a.u.) over all pixels that make up a cell is calculated. 
For each frame $n$ where a fluorescence image was taken, the production rate is determined as the slope of a linear fit through three points $n-\delta{n}$, $n$, and $n+\delta{n}$, where $\delta{n}$ is the interval at which fluorescence pictures are taken. This production rate is then subsequently divided by the total number of pixels of the cell in frame $n$.
The growth rate (with units dbl/hr or sometimes with units /min) is determined for each frame $n$ by fitting an exponential curve through frames $n-\delta{n}/2$ until $n+\delta{n}/2$ (or sometimes a smaller range).
Note that to determine scatter plots and correlations, only frames where fluorescence images were taken are considered.
This means that the choice for a growth rate fitting window of width $\delta{n}+1$ ensures that all data is used, but no point is used twice

\subsection*{Todo}

\begin{itemize}
	\item Add description of the lac reporter construct made by benjamin.
	\item Check maternal strain that Vanda used for the double reporter constructs.
	\item Comment on which datasets we are showing and which ones we are not, and where.
\end{itemize}

% For promoter sequences, see: M_2016_06_26_CRP_s70_promoter_sequences.docx

\section*{Acknowledgements}

I thank Pieter Rein ten Wolde and Harmen Wierenga for useful discussions.

%\section{Author contributions}

\section*{Things to keep in mind}

See notes sent to me by Pieter Rein!

***

\cite{You2013}
You, C., Okano, H., Hui, S., Zhang, Z., Kim, M., Gunderson, C.W., Wang, Y.-P., Lenz, P., Yan, D., and Hwa, T. (2013). Coordination of bacterial proteome with metabolism by cyclic AMP signalling. Nature 500, 301–6. Available at: http://www.ncbi.nlm.nih.gov/pubmed/23925119 [Accessed January 20, 2014].
(This is simply the Hwa article re. proteome partitioning, but should check because they also talk about cAMP)
According to chubukov2014 this is also article that shows akg feedback to CRP.

**

\cite{Somavanshi2016}
Somavanshi, R., Ghosh, B., and Sourjik, V. (2016). Sugar Influx Sensing by the Phosphotransferase System of Escherichia coli. 1–19.

Also check out other papers in (physical) yellow folder in cabinet labeled CRP.

**

\cite{Flamholz2013}
Flamholz, A., Noor, E., Bar-Even, A., Liebermeister, W., and Milo, R. (2013). Glycolytic strategy as a tradeoff between energy yield and protein cost. Proc. Natl. Acad. Sci. 110, 10039–10044.

Fig. S2 is already worth it because it gives nice overview between how glycolysis convert glucose to pyruvate and then either ferments it or aerobically burns it to CO2 (+acetate).

**

\cite{chubukov2014} has a few nice graphs about different "activities" that need to happen in the cell (ie. categories of metabolites and how they are produced.) Also explains how CRP is controlled! Point to YOu2013 for a-kg inhibition feedback loop.

**

Stewart-Ornstein 2017 \cite{Stewart-Ornstein2017} was sent by Sander, mentions CRP, so quickly check whether might be interesting.

**

Perhaps it is interesting to check out the network topology validation by Michael Stumpf (see Heidelberg Quant conference 2017). 

**


Check also:
Van Heerden et al. \cite{VanHeerden2017}
and
Nordholt et al. \cite{Nordholt2017}.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\clearpage
\FloatBarrier
\section*{Supplementary notes}

\subsection*{I. Understanding steady state values of metabolic and constitutive production rates, concentration, and growth rates}

\subsubsection{Motivation}
The purpose of this chapter is to understand 
single cell stochastic deviations away from steady state. 
%the dynamical behavior of metabolism and growth, 
%it is rather unsatisfactory to not understand how 
%steady state values are established in different conditions.
%population average values are established.
%
%Moreover, 
However, understanding the steady state relationships themselves between metabolic and constitutive protein production, concentration and the cellular growth rates helps us place the stochastic fluctuations away from the steady state values in the right context.
%
We explain our understanding fo this matter in this supplementary. % note we explain our understanding of this matter.

\subsubsection{Model and fitting of model to data}

To better understand steady state relationships it is convenient to write down the differential equations that describe the (supposed) time evolution of the system, 
and consider the steady state condition where the time derivative is zero.
%
Let us here start with an equation for the metabolic reporter.
%
%Towbin et al. \cite{Towbin2017} suggest that the relationship between the concentration, production and growth rate is as follows.
Generally, the change in metabolic protein concentration $M$ over time is set by a production and a dilution term:
\begin{align}
	\label{eq:CRP:odeM}
	\dot{M} = p_M - M \lambda = \phi_M(\text{cAMP}) \lambda -  M \lambda
	,
\end{align}
% 
where $p_M$ is the production rate ($\text{a.u}/(\text{px min})$) of $M$ ($\text{a.u}/(\text{px})$) and $\lambda$ ($/\text{min}$) the growth rate of the cell.
%Our single cell experiments allow the independent measurement of all these three parameters.
We can determine all these three parameters from our single cell experiments.
Following Towbin et al. \cite{Towbin2017}, we additionally propose $p_M = \phi_M(\text{cAMP}) \lambda$, where $\phi_M$ ($\text{a.u}/(\text{px})$) is a function of the concentration of cAMP and scales proportionally to it; the fractional production $\phi_M$ sets the production rate $p_M$ as a function of $\lambda$.
%
To understand the interaction between the metabolic reporter expression and the constitutive reporter expression we were inspired by the linear regulation model posed by You et al. \cite{You2013}, and pose
\begin{align}
	\label{eq:CRP:relationMQ}
	\phi_Q = T - \phi_M
	,
\end{align}
where $\phi_Q$ is the fractional production of the constitutive reporter, and $T$ the total fractional production of both $M$ and $Q$, which we consider a constant independent of the cAMP concentration. 
(Note that this ignores interaction with other cellular protein sectors such as ribosomes.) 
%
This implies that
\begin{align}
	\label{eq:CRP:odeQ}
	\dot{Q} = p_Q - Q \lambda = \phi_Q \lambda -  M \lambda = (T - \phi_M(\text{cAMP})) \lambda -  M \lambda
	,
\end{align}
where $Q$ is the concentration of the constitutive reporter, $p$ its production rate and $\phi_Q$ its fractional production rate.
%
Based on equations \ref{eq:CRP:odeM}-\ref{eq:CRP:odeQ} and the steady state assumption $\dot{M}=\dot{Q}=0$ we can make some predictions. Firstly, this suggests that the production rates (a quantity that we determine directly from our experiments) divided by the growth rates should be equal to the concentration for both the reporters. The top two panels in figure \ref{fig:CRP:averagerelations1} show that this is indeed the case. Secondly, it implies that the sum of the concentration of the two reporters should be equal, since $\phi_M(\text{cAMP}) = M$ and $\phi_Q(\text{cAMP}) = Q$ and thus 
\begin{align}
	\label{eq:CRP:sumconstant}	
	T = M + Q
	.
\end{align}
This relationship is confirmed in the bottom left panel of figure \ref{fig:CRP:averagerelations1}. 
One might have expected that ribosomal expression also factors into the relationship between $M$ and $Q$ --- e.g. $T=Q+M+R$ with $R$ the ribosomal concentration.
Surprisingly, equation \ref{eq:CRP:sumconstant} implies that this is not the case here; $R$ either remains constant or $R$ does change with $M$ but does not affect $Q$.
% See also notes 23-3, p.12 for this.
%
(Note that the observations in \cite{You2013} are in wild type cells; so the observations here do not necessarily contradict their observations.)
%
In any case, the steady state condition implies that $\phi_M(\text{cAMP}) = M$ (equation \ref{eq:CRP:odeM}), and thus that we can set the concentration of $M$ directly by adjusting the supplied cAMP concentration.
In other words: $M \propto \text{cAMP}$.
Furthermore, Towbin et al. observed a concave relationship between the CRP concentration and growth rate $\lambda$, which they call the optimality curve, or "O-curve" \cite{Towbin2017}.
We indeed observe a similar relationship, as shown in figure \ref{fig:CRP:averagerelations1}. 
Towbin et al. derive a functional form for this relationship, but for simplicity we have fitted a 3rd degree polynomial to this O-curve, i.e.
\begin{align}
	\label{eq:CRP:muwithCRP}	
	\lambda = f_\lambda(M) = a M^2 + b M + c
	,
\end{align}
where $a$, $b$ and $c$ are fitting parameters.
%
Using the relationships we have seen so far (figure \ref{fig:CRP:averagerelations1}), 
we can predict the relationships between the parameter pairs 
$\lambda\text{-}Q$, $p_M\text{-}p_Q$, $p_M\text{-}\lambda$, and $p_Q\text{-}\lambda$.
%$\lambda$-$Q$, $p_M$-$p_Q$, $p_M$-$\lambda$, and $p_Q$-$\lambda$.
%
Firstly, the relationship between $\lambda$ and $Q$ is defined by: 
\begin{align}
	\label{eq:CRP:muwithQ}	
	\lambda = f_\lambda(T-Q)
	,
\end{align}
which is indeed supported by the top left panel in figure \ref{fig:CRP:averagerelations2}.
%
Secondly, the relationships between $p$ and $\lambda$ can be parameterized by $M$ and $Q$: 
\begin{align}
	\label{eq:CRP:pmu}	
	\left(p_M, \lambda\right) & = \left(M f_\lambda(M), f_\lambda(M)\right) \nonumber \\ 
	\left(p_Q, \lambda\right) & = \left(Q f_\lambda(T-Q), f_\lambda(T-Q)\right)
	;
\end{align}
which can be found using the steady state condition and \ref{eq:CRP:odeM}. These equalities in equation \ref{eq:CRP:pmu} are corroborated by the bottom left and bottom right panels in figure \ref{fig:CRP:averagerelations2}.
%
Thirdly, we find that $p_M$ and $p_Q$ can be related through parameterizing $(p_M, p_Q)$ by $M$:
%
\begin{align}
	\label{eq:CRP:pMpQ}	
	\left(p_M, p_Q\right) = 
	\left(M f_\lambda(M), (T-M) f_\lambda(M)\right)
	.
\end{align}
This relationship is indeed confirmed in the top right panel of figure \ref{fig:CRP:averagerelations2}.
%
Note that no additional fitting was performed in figure \ref{fig:CRP:averagerelations2}.

\subsubsection*{Conclusion}

In conclusion, three ingredients determine the steady state relationships in the chapter: 
(1) The interaction between protein production and dilution that set the concentration, 
(2) the concave relationship ("O-curve") between metabolism and growth (figure \ref{fig:CRP:averagerelations1} bottom right panel) and
(3) the fact that the total concentration metabolic and constitutive expression remains constant.

\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% SCATTERS f(CRP,MU) and f(const, mu)
	\centering
	\includegraphics[width=0.39\textwidth]{pdf_relationsAverages_Y.pdf}
	\includegraphics[width=0.39\textwidth]{pdf_relationsAverages_C.pdf}
	%
	\includegraphics[width=0.39\textwidth]{pdf_relationsAverages_sum_YC.pdf}
	\includegraphics[width=0.39\textwidth]{pdf_relationsAverages_Y_mu.pdf}		
	\caption{ 
		\textbf{Fitted relationships between concentration, production rate and growth rate.}
		Circles correspond to steady state observations in wild type cells (blue) and $\Delta$cAMP cells supplied with 80, 800 and 2000 $\upmu$M cAMP (red, green and yellow respectively).
		(Top) Concentration is predicted to equal production rate divided by growth rate. Though there is a minor offset to the line x=y (solid black line) as shown by the fits (dashed and dotted lines), this prediction seems approximately correct. (The dotted line only is based on the average offset, whereas the dashed line is a polynomial fit.)
		(Bottom left) This panel shows that the sum of the metabolic and constitutive reporter concentrations remains approximately equal. The mean sum is 445 a.u./px.
		(Bottom right) This panel shows the concave relationship between growth and the metabolic reporter concentration. The dashed line shows a third degree polynomial fit to the data points. The fitted parameters (equation \ref{eq:CRP:muwithCRP}) are $a=-4.38\cdot10^{-7}$, $b=2.55\cdot10^{-4}$ and $c=-0.0276$.
	}
	\label{fig:CRP:averagerelations1}
\end{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% SCATTERS f(CRP,MU) and f(const, mu)
	\centering
	\includegraphics[width=0.39\textwidth]{pdf_relationsAverages_C_mu.pdf}
	\includegraphics[width=0.39\textwidth]{pdf_relationsAverages_prodC_prodY.pdf}	
	%
	\includegraphics[width=0.39\textwidth]{pdf_relationsAverages_pY_mu.pdf}
	\includegraphics[width=0.39\textwidth]{pdf_relationsAverages_pC_mu.pdf}	
	\caption{ 
		\textbf{Observed and predicted (not fitted) relationships.}
		Circles correspond to steady state observations in wild type cells (blue) and $\Delta$cAMP cells supplied with 80, 800 and 2000 $\upmu$M cAMP (red, green and yellow respectively).
		(Top left) The observed and predicted (equation \ref{eq:CRP:muwithQ}) relationship between the constitutive reporter production and growth rate.
		(Top right) The observed and predicted (equation \ref{eq:CRP:pmu}) relationship between the constitutive and metabolic production rates.
		(Bottom left) The observed and predicted (equation \ref{eq:CRP:pmu}) relationship between the metabolic production rate and growth rate.
		(Bottom right) The observed and predicted (equation \ref{eq:CRP:pMpQ}) relationship between the constitutive production rate and growth rate.
		%
	}
	\label{fig:CRP:averagerelations2}
\end{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_benjamin_ODE_relations.pdf}
	\caption{ 
		\textbf{Using models to understand the dynamic behavior of stochastic metabolic and growth fluctuations.}
		(Left) Illustrations of the contributions from the positive (production) and negative (dilution) terms to the differential equation that describes the time evolution of the concentration of metabolic enzymes as a function of that concentration $C$, according to the Towbin et al. model \cite{Towbin2017}. The different lines correspond to different values of the metabolite concentration $x$. 
		(Right)	Similar as on the left, except that the terms are production and consumption of metabolites as a function of metabolite concentration $x$ and that the different lines correspond to different values of the metabolic enzyme concentration $C$.
	}
	\label{fig:CRP:benjamin_ODEs}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% End of suppl. notes
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\clearpage
\FloatBarrier
\section*{Supplementary figures}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Sup belonging to figure 1 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	\centering
	\includegraphics[width=1.0\textwidth]{CRP-figXsup_overviewRegulationTCA.pdf}
	\clearpage % insert a page break
	\label{fig:CRP:figOverviewTCARegulation}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{Regulation of the TCA cycle and ED pathway.} 
	Some regulatory proteins control the expression of many tricarboxylic acid (TCA) cycle and/or enzymes glycolysis pathway simultaneously.
	This diagram shows the regulatory effects of CRP, but also of the Catabolite repressor activator (Cra) and the Anoxic redox control A (ArcA) protein.
	Cra controls the direction of the metabolite flux through metabolic pathways \cite{Keseler2017, Ramseier1995}. Cra is activated by the metabolite fructose-1,6-bisphosphate \cite{Kochanowski2013a}.
	ArcA is part of a two component system (ArcAB) which controls gene expression in response to aerobic versus anaerobic conditions \cite{Keseler2017, Alvarez2010}.
	Metabolites are displayed in larger font, enzymes that catalyze reactions in smaller fonts.
	In the colored boxes, regulation by CRP is displayed by an oval symbol marked "CRP".
	Similarly, when an enzyme is regulated by Cra or ArcA this is displayed next to the boxes; a plus or minus symbol indicates positive or negative regulation.
	Additionaly, in the boxes this diagram displays the concentration of particular enzymes in ppm, as annotated in PaxDb (retrieved in 2015) \cite{Wang2015}.
	The boxes are also color-coded according to abundance of the enzymes.
	The diagram is based on EcoCyc \cite{Keseler2017}. 
	From top to bottom, clockwise, enzymes abbreviations stand for glucose 6-phosphate, fructose 6-phosphate, fructose 1,6-biphosphate, glyceraldehyde 3-phosphate, 1,3-biphospho-D-glycerate, 3-phospho-glycerate, 2-phospho-glycerate, phosphoenolpyruvate, pyruvate, acetyl coenzyme A, citrate, cis-aconitate, d-threo-isocitrate, (succinate, glyoxalate, acetyl-CoA), 2-oxo-glutarate, succinyl-CoA, succinate, fumarate, malate, oxaloacetate.
	% See also: https://biocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&object=TCA
	% And: https://biocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&object=GLYCOLYSIS
}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Pulsing dynamics

\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_averagetimetraces_4.pdf}
	\includegraphics[width=1.0\textwidth]{pdf_averagetimetraces_2.pdf}
	\includegraphics[width=1.0\textwidth]{pdf_averagetimetraces_5.pdf}
	\clearpage % insert a page break
	\label{fig:XXX:XXX}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{blabla}
}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	\centering
	\includegraphics[width=0.94\textwidth]{pdf_timeevolution_2.pdf}	
	\includegraphics[width=0.94\textwidth]{pdf_timeevolution_3.pdf}
	\includegraphics[width=0.94\textwidth]{pdf_timeevolution_4.pdf}	
	\clearpage % insert a page break
	\label{fig:CRP:productiontimeevolutions}
\end{figure}

\clearpage

\captionof{figure}{    
		\textbf{Time evolution of production versus growth rate during block pulses of high and low cAMP concentrations.}
		(Top) The production rate of the CRP reporter plotted against growth rate, color coded for time. Production is determined here by the difference in absolute signal between two timepoints, divided by the amount of time inbetween these two points. Note that the production rate also depends on the growth rate, as does the final concentration of the protein. This makes this data not trivial to interpret.
		(Middle) Similar to top panel, but production rate is divided by growth rate, since production rates might be more faitfully represented as fractions of the growth rate.
		(Bottom) Similar to top panel, but production rate is divided by the production rate of the constitutively expressed label, which serves a reference for the total production rate capacity of the cell.
}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% SCATTERS f(CRP,MU) and f(const, mu)
	\centering
	\includegraphics[width=0.94\textwidth]{pdf_timeevolution_5.pdf}
	\caption{ 
		\textbf{Time evolution of metabolic reporter concentration normalized by constitutive reporter concentration.}
		This figure is similar to figure \ref{fig:CRP:productiontimeevolutions}, but shows the concentration of the metabolic reporter divided by the constitutive reporter on the x-axis, instead of production rates.
	}
	\label{fig:CRP:normalizedconcentrationtimeevolutions}
\end{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


% pdf_scattersoftraces_1

\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% SCATTERS f(CRP,MU) and f(const, mu)
	\centering
	\includegraphics[width=0.49\textwidth]{pdf_scattersoftraces_1.pdf}
	\includegraphics[width=0.49\textwidth]{pdf_scattersoftraces_2.pdf}	
	\caption{ 
		\textbf{Scatter plots of growth rate versus metabolic and constitutive reporter from single cell measurements.}
		Every point in these plots corresponds to a single cell observation on growth rate and fluorescent label concentration.
		On the left, growth rate plotted against the concentration of the metabolic reporter.
		On the right, growth rate plotted against the concentration of a constitutive reporter based on the same lac metabolic reporter but with the CRP binding site replaced by a $\upsigma$70 consensus binding site.
		The black lines indicate the average signal for corresponding concentrations and standard deviations.
		The red lines are interpolated 2nd degree polynomial fits of the displayed mean values.
		The red square shows the maximum value of the interpolated values at a metabolic reporter concentration of 225 a.u. (corresponding to a growth rate of 0.63 doublings/hr).
	}
	\label{fig:CRP:scatterspulsing}
\end{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_timetracesinglecell13.pdf}
	\includegraphics[width=1.0\textwidth]{pdf_timetracesinglecell49.pdf}
	\includegraphics[width=1.0\textwidth]{pdf_timetracesinglecell144.pdf}
	\clearpage % insert a page break
	\label{fig:CRP:timevolutionCRPgrowthsinglecell}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{Time evolution of growth rate and metabolic reporter concentration for single cells.}
	Each panel is identical to main figure \ref{fig:CRP:fig1}.C, except that each panel corresponds to the behavior of a lineage of single cells.
}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Overview (grey bars) with summary parameters from different conditions
% Figure with floating caption

\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_chromo1prime_overview_means.pdf}
	\clearpage % insert a page break
	\label{fig:CRP:overviewsummaryparams}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{Comparison of summary parameters in different experimental conditions.} 
	This figure shows growth rates (doublings per hour), production rates (a.u./(px min)), concentrations (a.u./px) and coefficients of variations (CV). 
	C is the abbreviation used for the constitutive reporter (which uses a cyan mCerulean label), and Y is the abbreviation used for the metabolic reporter (which uses a yellow mVenus label).
	WT stands for wild type, indicating wild type cells except the addition of our metabolic and constitutive reporter constructs.
	No FB stands for no feedback, indicating the $\Delta$cAMP cells that also carry the metabolic and constitutive reporter constructs.
	Low, med and high correspond to extracellularly provided cAMP concentrations of 80, 800 and 5000 $\upmu$M cAMP respectively.
	Bars are averages over multiple experiments, and each open circle corresponds to a value observed in one experiment.
}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Sup belonging to figure 2 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Scatters of production
\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{CRP-fig2sup.pdf}
	\caption{ 
	\textbf{Scatter plots of production rates against growth rate.}
	(A) Colored dots show single cell growth rate values plotted against respective single cell production rates of metabolic reporter, which is a proxy for the production rate of metabolic enzymes. 	
	Dots correspond to data points from WT cells with endogenous negative metabolic feedback, and $\Delta$cAMP cells which have been grown at concentrations of 80, 800 and 5000 $\upmu$M cAMP 
	(colored blue, and red, green and orange, respectively).
	The black lines show the average growth rate for cells that are binned according to production rates, and the black isolines reflect kernel density estimates of the probability distribution (using the matlab function \texttt{kde2d} \cite{Botev2010}).
	The circles with thick black edges show population average values per experiment.
	Note that the trends in population average values are explained in Supplemental note I at the end of this chapter.
	(B) As panel A, except that this panel shows the relationship between growth and the production rate of a constitutive reporter. 	
	}
	\label{fig:CRP:fig2sup}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Scatters of conc-conc and prod-prod
\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_chromo1prime_overview_custom_scatter_concentrationVsConcentrationYC.pdf}
	\includegraphics[width=1.0\textwidth]{pdf_chromo1prime_overview_custom_scatter_rateVsRateYC.pdf}	
	\caption{ 
		\textbf{Concentration-concentration and production-production relationships between metabolic and constitutive reporters.}
		This figure is similar to Figure \ref{fig:CRP:fig2sup}, except that the relationships plotted are different.
		Panels show: (Top) scatter plots for concentration of metabolic (CRP) reporter against the concentration of constitutive (s70) reporter; 
		(Bottom) scatter plots for production rate of metabolic (CRP) reporter against the production rate of constitutive (s70) reporter.
		Blue, red, green and orange dots again correspond respectively to data points from WT cells with endogenous negative metabolic feedback, and $\Delta$cAMP cells which have been grown at concentrations of 80, 800 and 5000 $\upmu$M cAMP.
	}
	\label{fig:CRP:fig3scatters_CC_pp}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Sup belonging to figure 3 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{CRP-fig3sup.pdf}
	\caption{ 
		(A) 
	}
	\label{fig:CRP:fig3sup}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Sup belonging to figure 4 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Extra sup not belonging but anyways shown %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\FloatBarrier
\clearpage
\section*{Additional supplemental figures}

This section contains additional supplemental figures that were not referenced in the chapter.

% Dynamic behavior against itself.
\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	\centering
	\includegraphics[width=1.00\textwidth]{pdf_highlowplots_case1.pdf}
	\caption{ 
		(A) 
	}
	\label{fig:CRP:XXX}
\end{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	\centering
	\includegraphics[width=1.00\textwidth]{pdf_highlowplots_case2.pdf}
	\caption{ 
		(A) 
	}
	\label{fig:CRP:XXX}
\end{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	\centering
	\includegraphics[width=1.00\textwidth]{pdf_highlowplots_case3.pdf}
	\caption{ 
		(A) 
	}
	\label{fig:CRP:XXX}
\end{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	\centering
	\includegraphics[width=1.00\textwidth]{pdf_highlowplots_case4.pdf}
	\caption{ 
		(A) 
	}
	\label{fig:CRP:XXX}
\end{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	\centering
	\includegraphics[width=1.00\textwidth]{pdf_highlowplots_case5.pdf}
	\caption{ 
		(A) 
	}
	\label{fig:CRP:XXX}
\end{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	\centering
	\includegraphics[width=1.00\textwidth]{pdf_highlowplots_case6.pdf}
	\caption{ 
		(A) 
	}
	\label{fig:CRP:XXX}
\end{figure}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% CRP CCs 1
% Figure with floating caption

\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_chromo1_CCs_Y6_mean_cycCor_muP9_fitNew_cycCor}
	\clearpage % insert a page break
	\label{fig:XXX:XXX}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{blabla}
}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2
% Figure with floating caption

\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_chromo1_CCs_dY5_cycCor_muP9_fitNew_atdY5_cycCor}
	\clearpage % insert a page break
	\label{fig:XXX:XXX}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{blabla}
}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Consti CCs 1
% Figure with floating caption

\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_chromo1_CCs_C6_mean_cycCor_muP9_fitNew_cycCor}
	\clearpage % insert a page break
	\label{fig:XXX:XXX}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{blabla}
}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2
% Figure with floating caption

\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_chromo1_CCs_dC5_cycCor_muP9_fitNew_atdC5_cycCor}
	\clearpage % insert a page break
	\label{fig:XXX:XXX}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{blabla}
}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Branches mu
% Figure with floating caption

\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_chromo1_branches_muP9_fitNew_cycCor}
	\clearpage % insert a page break
	\label{fig:XXX:XXX}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{blabla}
}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Branches Y (CRP)
% Figure with floating caption

\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_chromo1_branches_Y6_mean_cycCor}
	\clearpage % insert a page break
	\label{fig:XXX:XXX}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{blabla}
}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Branches dY (CRP)
% Figure with floating caption

\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_chromo1_branches_dY5_cycCor}
	\clearpage % insert a page break
	\label{fig:XXX:XXX}
\end{figure}	

\clearpage

\captionof{figure}{    
	\textbf{blabla}
}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%





%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}
	\centering
	\includegraphics[width=1.0\textwidth]{pdf_plasmids2_overview_correlations_CmuPmu_G.pdf}
	\caption{ 
		\textbf{Without feedback, metabolic fluctuations transmit to growth.}
		The interaction of metabolic and growth fluctuations was measured by plasmid constructs.
		(A) 
	}
	\label{fig:CRP:plasmidCCs}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%



